// phosphor-react-native/lib/module/bold/BluetoothConnected.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const BluetoothConnected = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "bluetooth-connected-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-bluetooth-connected-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M199.2 166.4 148 128l51.2-38.4a12 12 0 0 0 0-19.2l-64-48A12 12 0 0 0 116 32v72L71.2 70.4a12 12 0 0 0-14.4 19.2L108 128l-51.2 38.4a12 12 0 1 0 14.4 19.2L116 152v72a12 12 0 0 0 19.2 9.6l64-48a12 12 0 0 0 0-19.2ZM140 56l32 24-32 24Zm0 144v-48l32 24Zm-84-56a16 16 0 1 1 16-16 16 16 0 0 1-16 16Zm168-16a16 16 0 1 1-16-16 16 16 0 0 1 16 16Z"
  })]
});
export default BluetoothConnected;
