// phosphor-react-native/lib/module/bold/ChartScatter.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const ChartScatter = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "chart-scatter-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-chart-scatter-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M236 208a12 12 0 0 1-12 12H32a12 12 0 0 1-12-12V48a12 12 0 0 1 24 0v148h180a12 12 0 0 1 12 12Zm-120-60a16 16 0 1 0 16-16 16 16 0 0 0-16 16Zm-8-40a16 16 0 1 0-16-16 16 16 0 0 0 16 16Zm-32 72a16 16 0 1 0-16-16 16 16 0 0 0 16 16Zm96-48a16 16 0 1 0-16-16 16 16 0 0 0 16 16Zm24-40a16 16 0 1 0-16-16 16 16 0 0 0 16 16Zm-24 72a16 16 0 1 0 16-16 16 16 0 0 0-16 16Z"
  })]
});
export default ChartScatter;
