// phosphor-react-native/lib/module/bold/ClosedCaptioning.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const ClosedCaptioning = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "closed-captioning-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-closed-captioning-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M224 44H32a20 20 0 0 0-20 20v128a20 20 0 0 0 20 20h192a20 20 0 0 0 20-20V64a20 20 0 0 0-20-20Zm-4 144H36V68h184ZM52 128a44 44 0 0 1 66-38.11 12 12 0 0 1-12 20.78 20 20 0 1 0 0 34.66 12 12 0 1 1 12 20.77A44 44 0 0 1 52 128Zm80 0a44 44 0 0 1 66-38.11 12 12 0 0 1-12 20.78 20 20 0 1 0 0 34.66 12 12 0 1 1 12 20.77 44 44 0 0 1-66-38.1Z"
  })]
});
export default ClosedCaptioning;
