// phosphor-react-native/lib/module/bold/ColumnsPlusLeft.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const ColumnsPlusLeft = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "columns-plus-left-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-columns-plus-left-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M124 28h-24a20 20 0 0 0-20 20v160a20 20 0 0 0 20 20h24a20 20 0 0 0 20-20V48a20 20 0 0 0-20-20Zm-4 176h-16V52h16Zm80-176h-24a20 20 0 0 0-20 20v160a20 20 0 0 0 20 20h24a20 20 0 0 0 20-20V48a20 20 0 0 0-20-20Zm-4 176h-16V52h16ZM68 128a12 12 0 0 1-12 12h-8v8a12 12 0 0 1-24 0v-8h-8a12 12 0 0 1 0-24h8v-8a12 12 0 0 1 24 0v8h8a12 12 0 0 1 12 12Z"
  })]
});
export default ColumnsPlusLeft;
