// phosphor-react-native/lib/module/bold/DeviceMobileSlash.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const DeviceMobileSlash = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "device-mobile-slash-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-device-mobile-slash-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "m216.88 207.93-160-176a12 12 0 1 0-17.76 16.14L52 62.24V216a28 28 0 0 0 28 28h96a28 28 0 0 0 26-17.61 12 12 0 0 0 14.88-18.46ZM176 220H80a4 4 0 0 1-4-4V88.64L180 203v13a4 4 0 0 1-4 4ZM75.51 24a12 12 0 0 1 12-12H176a28 28 0 0 1 28 28v98.94a12 12 0 0 1-24 0V40a4 4 0 0 0-4-4H87.51a12 12 0 0 1-12-12Z"
  })]
});
export default DeviceMobileSlash;
