// phosphor-react-native/lib/module/bold/DiceFour.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const DiceFour = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "dice-four-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-dice-four-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M192 28H64a36 36 0 0 0-36 36v128a36 36 0 0 0 36 36h128a36 36 0 0 0 36-36V64a36 36 0 0 0-36-36Zm12 164a12 12 0 0 1-12 12H64a12 12 0 0 1-12-12V64a12 12 0 0 1 12-12h128a12 12 0 0 1 12 12Zm-92-96a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm64 0a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm-64 64a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm64 0a16 16 0 1 1-16-16 16 16 0 0 1 16 16Z"
  })]
});
export default DiceFour;
