// phosphor-react-native/lib/module/bold/Equalizer.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Equalizer = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "equalizer-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-equalizer-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M80 108a12 12 0 0 1-12 12H28a12 12 0 0 1 0-24h40a12 12 0 0 1 12 12Zm-12 28H28a12 12 0 0 0 0 24h40a12 12 0 0 0 0-24Zm0 40H28a12 12 0 0 0 0 24h40a12 12 0 0 0 0-24Zm80-40h-40a12 12 0 0 0 0 24h40a12 12 0 0 0 0-24Zm0 40h-40a12 12 0 0 0 0 24h40a12 12 0 0 0 0-24Zm40-96h40a12 12 0 0 0 0-24h-40a12 12 0 0 0 0 24Zm40 16h-40a12 12 0 0 0 0 24h40a12 12 0 0 0 0-24Zm0 40h-40a12 12 0 0 0 0 24h40a12 12 0 0 0 0-24Zm0 40h-40a12 12 0 0 0 0 24h40a12 12 0 0 0 0-24Z"
  })]
});
export default Equalizer;
