// phosphor-react-native/lib/module/bold/FigmaLogo.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const FigmaLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "figma-logo-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-figma-logo-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M198.15 96A44 44 0 0 0 168 20H96a44 44 0 0 0-30.15 76 43.9 43.9 0 0 0 1.23 65.12A48 48 0 1 0 148 196v-28.83A44 44 0 0 0 198.15 96ZM188 64a20 20 0 0 1-20 20h-20V44h20a20 20 0 0 1 20 20ZM76 64a20 20 0 0 1 20-20h28v40H96a20 20 0 0 1-20-20Zm20 84a20 20 0 0 1 0-40h28v40H96Zm28 48a24 24 0 1 1-24-24h24Zm44-48a20 20 0 1 1 20-20 20 20 0 0 1-20 20Z"
  })]
});
export default FigmaLogo;
