// phosphor-react-native/lib/module/bold/FunnelSimpleX.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const FunnelSimpleX = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "funnel-simple-x-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-funnel-simple-x-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M192 148H64a12 12 0 0 1 0-24h128a12 12 0 0 1 0 24Zm40-72H24a12 12 0 0 0 0 24h208a12 12 0 0 0 0-24Zm-104 96h-24a12 12 0 0 0 0 24h24a12 12 0 0 0 0-24Zm96.49-4.49a12 12 0 0 0-17 0L192 183l-15.51-15.52a12 12 0 0 0-17 17L175 200l-15.52 15.51a12 12 0 0 0 17 17L192 217l15.51 15.52a12 12 0 0 0 17-17L209 200l15.52-15.51a12 12 0 0 0-.03-16.98Z"
  })]
});
export default FunnelSimpleX;
