// phosphor-react-native/lib/module/bold/ListMagnifyingGlass.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const ListMagnifyingGlass = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "list-magnifying-glass-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-list-magnifying-glass-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M28 64a12 12 0 0 1 12-12h176a12 12 0 0 1 0 24H40a12 12 0 0 1-12-12Zm12 76h64a12 12 0 0 0 0-24H40a12 12 0 0 0 0 24Zm80 40H40a12 12 0 0 0 0 24h80a12 12 0 0 0 0-24Zm120.49 20.49a12 12 0 0 1-17 0l-18.08-18.08a44 44 0 1 1 17-17l18.08 18.07a12 12 0 0 1 0 17.01ZM184 164a20 20 0 1 0-20-20 20 20 0 0 0 20 20Z"
  })]
});
export default ListMagnifyingGlass;
