// phosphor-react-native/lib/module/bold/LockKeyOpen.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const LockKeyOpen = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "lock-key-open-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-lock-key-open-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M208 76H100V56a28 28 0 0 1 28-28c13.51 0 25.65 9.62 28.24 22.39a12 12 0 1 0 23.52-4.78C174.87 21.5 153.1 4 128 4a52.06 52.06 0 0 0-52 52v20H48a20 20 0 0 0-20 20v112a20 20 0 0 0 20 20h160a20 20 0 0 0 20-20V96a20 20 0 0 0-20-20Zm-4 128H52V100h152Zm-88-30.34V180a12 12 0 0 0 24 0v-6.34a32 32 0 1 0-24 0ZM128 136a8 8 0 1 1-8 8 8 8 0 0 1 8-8Z"
  })]
});
export default LockKeyOpen;
