// phosphor-react-native/lib/module/bold/MouseMiddleClick.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const MouseMiddleClick = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "mouse-middle-click-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-mouse-middle-click-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M144 12h-32a68.07 68.07 0 0 0-68 68v96a68.07 68.07 0 0 0 68 68h32a68.07 68.07 0 0 0 68-68V80a68.07 68.07 0 0 0-68-68Zm44 68v20h-28V88a20 20 0 0 0-20-20V36h4a44.05 44.05 0 0 1 44 44Zm-68 12h16v40h-16Zm-8-56h4v32a20 20 0 0 0-20 20v12H68V80a44.05 44.05 0 0 1 44-44Zm32 184h-32a44.05 44.05 0 0 1-44-44v-52h28v12a20 20 0 0 0 20 20h24a20 20 0 0 0 20-20v-12h28v52a44.05 44.05 0 0 1-44 44Z"
  })]
});
export default MouseMiddleClick;
