// phosphor-react-native/lib/module/bold/MusicNotesPlus.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const MusicNotesPlus = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "music-notes-plus-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-music-notes-plus-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M236 48a12 12 0 0 1-12 12h-12v12a12 12 0 0 1-24 0V60h-12a12 12 0 0 1 0-24h12V24a12 12 0 0 1 24 0v12h12a12 12 0 0 1 12 12Zm-16 64v52a40 40 0 1 1-24-36.65V112a12 12 0 0 1 24 0Zm-24 52a16 16 0 1 0-16 16 16 16 0 0 0 16-16ZM92 113.37V196a40 40 0 1 1-24-36.65V56a12 12 0 0 1 9.09-11.64l56-14a12 12 0 0 1 5.82 23.28L92 65.37v23.26l65.09-16.27a12 12 0 0 1 5.82 23.28ZM68 196a16 16 0 1 0-16 16 16 16 0 0 0 16-16Z"
  })]
});
export default MusicNotesPlus;
