// phosphor-react-native/lib/module/bold/Numpad.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Numpad = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "numpad-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-numpad-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M84 48a20 20 0 1 1-20-20 20 20 0 0 1 20 20Zm44-20a20 20 0 1 0 20 20 20 20 0 0 0-20-20Zm64 40a20 20 0 1 0-20-20 20 20 0 0 0 20 20ZM64 84a20 20 0 1 0 20 20 20 20 0 0 0-20-20Zm64 0a20 20 0 1 0 20 20 20 20 0 0 0-20-20Zm64 0a20 20 0 1 0 20 20 20 20 0 0 0-20-20ZM64 140a20 20 0 1 0 20 20 20 20 0 0 0-20-20Zm64 0a20 20 0 1 0 20 20 20 20 0 0 0-20-20Zm0 56a20 20 0 1 0 20 20 20 20 0 0 0-20-20Zm64-56a20 20 0 1 0 20 20 20 20 0 0 0-20-20Z"
  })]
});
export default Numpad;
