// phosphor-react-native/lib/module/bold/Oven.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Oven = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "oven-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-oven-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M208 28H48a20 20 0 0 0-20 20v160a20 20 0 0 0 20 20h160a20 20 0 0 0 20-20V48a20 20 0 0 0-20-20Zm-4 176H52V52h152ZM68 80a16 16 0 1 1 16 16 16 16 0 0 1-16-16Zm44 0a16 16 0 1 1 16 16 16 16 0 0 1-16-16Zm44 0a16 16 0 1 1 16 16 16 16 0 0 1-16-16ZM76 192h104a12 12 0 0 0 12-12v-60a12 12 0 0 0-12-12H76a12 12 0 0 0-12 12v60a12 12 0 0 0 12 12Zm12-60h80v36H88Z"
  })]
});
export default Oven;
