// phosphor-react-native/lib/module/bold/PersonSimpleCircle.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const PersonSimpleCircle = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "person-simple-circle-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-person-simple-circle-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M128 20a108 108 0 1 0 108 108A108.12 108.12 0 0 0 128 20Zm0 192a84 84 0 1 1 84-84 84.09 84.09 0 0 1-84 84ZM108 80a20 20 0 1 1 20 20 20 20 0 0 1-20-20Zm76 40a12 12 0 0 1-12 12h-32v.51l26.12 41.05a12 12 0 0 1-3.68 16.56A11.83 11.83 0 0 1 156 192a12 12 0 0 1-10.13-5.56L128 158.35l-17.88 28.09a12 12 0 0 1-20.24-12.88L116 132.51V132H84a12 12 0 0 1 0-24h88a12 12 0 0 1 12 12Z"
  })]
});
export default PersonSimpleCircle;
