// phosphor-react-native/lib/module/bold/Screencast.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Screencast = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "screencast-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-screencast-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M232 60v136a20 20 0 0 1-20 20h-40a12 12 0 0 1 0-24h36V64H48v4a12 12 0 0 1-24 0v-8a20 20 0 0 1 20-20h168a20 20 0 0 1 20 20ZM36 176a12 12 0 0 0 0 24 4 4 0 0 1 4 4 12 12 0 0 0 24 0 28 28 0 0 0-28-28Zm0-40a12 12 0 0 0 0 24 44.05 44.05 0 0 1 44 44 12 12 0 0 0 24 0 68.07 68.07 0 0 0-68-68Zm0-40a12 12 0 0 0 0 24 84.09 84.09 0 0 1 84 84 12 12 0 0 0 24 0A108.12 108.12 0 0 0 36 96Z"
  })]
});
export default Screencast;
