// phosphor-react-native/lib/module/bold/VinylRecord.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const VinylRecord = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "vinyl-record-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-vinyl-record-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M128 20a108 108 0 1 0 108 108A108.12 108.12 0 0 0 128 20Zm0 192a84 84 0 1 1 84-84 84.09 84.09 0 0 1-84 84Zm0-128a44.05 44.05 0 0 0-44 44 12 12 0 0 1-24 0 68.07 68.07 0 0 1 68-68 12 12 0 0 1 0 24Zm68 44a68.07 68.07 0 0 1-68 68 12 12 0 0 1 0-24 44.05 44.05 0 0 0 44-44 12 12 0 0 1 24 0Zm-68 28a28 28 0 1 1 28-28 28 28 0 0 1-28 28Z"
  })]
});
export default VinylRecord;
