// phosphor-react-native/lib/commonjs/thin/Alien.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Alien = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "alien-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-alien-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 20a92.1 92.1 0 0 0-92 92c0 23.19 12.26 53.33 32.8 80.63C84.58 213.6 108.41 236 128 236s43.42-22.4 59.2-43.37c20.54-27.3 32.8-57.44 32.8-80.63a92.1 92.1 0 0 0-92-92Zm52.8 167.82C162.44 212.23 141.71 228 128 228s-34.44-15.77-52.8-40.18C56 162.24 44 133.19 44 112a84 84 0 0 1 168 0c0 21.19-11.95 50.24-31.2 75.82ZM116 136a36 36 0 0 0-36-36 12 12 0 0 0-12 12 36 36 0 0 0 36 36 12 12 0 0 0 12-12Zm-12 4a28 28 0 0 1-28-28 4 4 0 0 1 4-4 28 28 0 0 1 28 28 4 4 0 0 1-4 4Zm72-40a36 36 0 0 0-36 36 12 12 0 0 0 12 12 36 36 0 0 0 36-36 12 12 0 0 0-12-12Zm-24 40a4 4 0 0 1-4-4 28 28 0 0 1 28-28 4 4 0 0 1 4 4 28 28 0 0 1-28 28Zm-4 44a4 4 0 0 1-4 4h-32a4 4 0 0 1 0-8h32a4 4 0 0 1 4 4Z"
  })]
});
var _default = exports.default = Alien;
