// phosphor-react-native/lib/commonjs/thin/Asclepius.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Asclepius = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "asclepius-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-asclepius-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M212 79v1a36 36 0 0 1-36 36h-44v88h12a20 20 0 0 0 13.33-34.91 4 4 0 1 1 5.34-6A28 28 0 0 1 144 212h-12v20a4 4 0 0 1-8 0v-20H96a4 4 0 0 1 0-8h28v-88H96a20 20 0 0 0 0 40 4 4 0 0 1 0 8 28 28 0 0 1 0-56h28V24a4 4 0 0 1 8 0v84h44a28 28 0 0 0 28-28v-1a27 27 0 0 0-27-27h-17a4 4 0 0 1 0-8h17a35 35 0 0 1 35 35ZM56 92H32a4 4 0 0 1-4-4v-8a36 36 0 0 1 36-36h32a4 4 0 0 1 0 8h-4v4a36 36 0 0 1-36 36Zm28-36v-4H64a28 28 0 0 0-28 28v4h20a28 28 0 0 0 28-28Z"
  })]
});
var _default = exports.default = Asclepius;
