// phosphor-react-native/lib/commonjs/thin/BabyCarriage.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const BabyCarriage = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "baby-carriage-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-baby-carriage-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M160 36h-8a12 12 0 0 0-12 12v60H51.77A36 36 0 0 0 16 76a4 4 0 0 0 0 8 28 28 0 0 1 28 28 76.08 76.08 0 0 0 76 76h40a76 76 0 0 0 0-152Zm67.87 72H155.4l53.71-43a67.76 67.76 0 0 1 18.76 43ZM152 44h8a67.72 67.72 0 0 1 43.2 15.52L148 103.68V48a4 4 0 0 1 4-4Zm8 136h-40a68.08 68.08 0 0 1-67.88-64h175.76A68.08 68.08 0 0 1 160 180Zm-60 44a12 12 0 1 1-12-12 12 12 0 0 1 12 12Zm104 0a12 12 0 1 1-12-12 12 12 0 0 1 12 12Z"
  })]
});
var _default = exports.default = BabyCarriage;
