// phosphor-react-native/lib/commonjs/thin/Boot.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Boot = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "boot-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-boot-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M192 116h-36V56a12 12 0 0 0-12-12H32a4 4 0 0 0-3.73 5.43c9.29 24.23 11.46 77.22-.11 117.46A3.82 3.82 0 0 0 28 168v32a12 12 0 0 0 12 12h26.11a12.08 12.08 0 0 0 5.37-1.27l12.62-6.31a4.09 4.09 0 0 1 1.79-.42h16.22a4.09 4.09 0 0 1 1.79.42l12.62 6.31a12.08 12.08 0 0 0 5.37 1.27h28.22a12.08 12.08 0 0 0 5.37-1.27l12.62-6.31a4.09 4.09 0 0 1 1.79-.42h16.22a4.09 4.09 0 0 1 1.79.42l12.62 6.31a12.08 12.08 0 0 0 5.37 1.27H232a12 12 0 0 0 12-12v-32a52.06 52.06 0 0 0-52-52ZM37.6 52H144a4 4 0 0 1 4 4v28h-36a4 4 0 0 0 0 8h36v24h-36a4 4 0 0 0 0 8h80a44.06 44.06 0 0 1 43.81 40H37.2c8.04-32.19 9.8-81.35.4-112ZM236 200a4 4 0 0 1-4 4h-26.11a4.09 4.09 0 0 1-1.79-.42l-12.62-6.31a12.08 12.08 0 0 0-5.37-1.27h-16.22a12.08 12.08 0 0 0-5.37 1.27l-12.62 6.31a4.09 4.09 0 0 1-1.79.42h-28.22a4.09 4.09 0 0 1-1.79-.42l-12.62-6.31a12.08 12.08 0 0 0-5.37-1.27H85.89a12.08 12.08 0 0 0-5.37 1.27l-12.62 6.31a4.09 4.09 0 0 1-1.79.42H40a4 4 0 0 1-4-4v-28h200Z"
  })]
});
var _default = exports.default = Boot;
