// phosphor-react-native/lib/commonjs/thin/BowlSteam.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const BowlSteam = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "bowl-steam-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-bowl-steam-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M224 116H32a4 4 0 0 0-4 4 100.32 100.32 0 0 0 56 89.82V216a12 12 0 0 0 12 12h64a12 12 0 0 0 12-12v-6.18A100.32 100.32 0 0 0 228 120a4 4 0 0 0-4-4Zm-57.67 87.66a4 4 0 0 0-2.33 3.64v8.7a4 4 0 0 1-4 4H96a4 4 0 0 1-4-4v-8.7a4 4 0 0 0-2.33-3.64A92.31 92.31 0 0 1 36.09 124h183.82a92.31 92.31 0 0 1-53.58 79.66ZM164.88 57.5c5.56-6.92 8-12.89 6.83-16.81-1-3.54-4.9-4.87-4.94-4.88a4 4 0 0 1 2.42-7.63c.32.1 7.84 2.52 10.16 10.11 2.05 6.72-.72 14.87-8.23 24.22-5.56 6.91-8 12.88-6.83 16.8 1 3.54 4.9 4.87 4.94 4.88A4 4 0 0 1 168 92a4.09 4.09 0 0 1-1.19-.18c-.32-.1-7.84-2.52-10.16-10.11-2.05-6.71.72-14.87 8.23-24.21Zm-40 0c5.56-6.92 8-12.89 6.83-16.81-1-3.54-4.9-4.87-4.94-4.88a4 4 0 0 1 2.42-7.63c.32.1 7.84 2.52 10.16 10.11 2.05 6.72-.72 14.87-8.23 24.22-5.56 6.91-8 12.88-6.83 16.8 1 3.54 4.9 4.87 4.94 4.88A4 4 0 0 1 128 92a4.09 4.09 0 0 1-1.19-.18c-.32-.1-7.84-2.52-10.16-10.11-2.05-6.71.72-14.87 8.23-24.21Zm-40 0c5.56-6.92 8-12.89 6.83-16.81-1-3.54-4.9-4.87-4.94-4.88a4 4 0 0 1 2.42-7.63c.32.1 7.84 2.52 10.16 10.11 2.05 6.72-.72 14.87-8.23 24.22-5.56 6.91-8 12.88-6.83 16.8 1 3.54 4.9 4.87 4.94 4.88A4 4 0 0 1 88 92a4.09 4.09 0 0 1-1.19-.18c-.32-.1-7.84-2.52-10.16-10.11-2.05-6.71.72-14.87 8.23-24.21Z"
  })]
});
var _default = exports.default = BowlSteam;
