// phosphor-react-native/lib/commonjs/thin/Cactus.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Cactus = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "cactus-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-cactus-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M216 212h-52v-32h8a64.07 64.07 0 0 0 64-64 24 24 0 0 0-48 0 16 16 0 0 1-16 16h-8V56a36 36 0 0 0-72 0v36h-8a16 16 0 0 1-16-16 24 24 0 0 0-48 0 64.07 64.07 0 0 0 64 64h8v72H40a4 4 0 0 0 0 8h176a4 4 0 0 0 0-8ZM96 132H84a56.06 56.06 0 0 1-56-56 16 16 0 0 1 32 0 24 24 0 0 0 24 24h12a4 4 0 0 0 4-4V56a28 28 0 0 1 56 0v80a4 4 0 0 0 4 4h12a24 24 0 0 0 24-24 16 16 0 0 1 32 0 56.06 56.06 0 0 1-56 56h-12a4 4 0 0 0-4 4v36h-56v-76a4 4 0 0 0-4-4Z"
  })]
});
var _default = exports.default = Cactus;
