// phosphor-react-native/lib/commonjs/thin/CellTower.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const CellTower = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "cell-tower-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-cell-tower-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M131.58 86.21a4 4 0 0 0-7.16 0l-72 144a4 4 0 0 0 7.16 3.58L74.47 204h107.06l14.89 29.79A4 4 0 0 0 200 236a4.12 4.12 0 0 0 1.79-.42 4 4 0 0 0 1.79-5.37ZM128 96.94 161.53 164H94.47ZM78.47 196l12-24h75.06l12 24Zm84.95-101.55A35.57 35.57 0 0 0 164 88a36 36 0 0 0-72 0 35.57 35.57 0 0 0 .58 6.45 4 4 0 0 1-7.88 1.43A45.18 45.18 0 0 1 84 88a44 44 0 0 1 88 0 45.18 45.18 0 0 1-.7 7.88 4 4 0 0 1-3.94 3.29 4.38 4.38 0 0 1-.71-.07 4 4 0 0 1-3.23-4.65Zm-96.08 39.34a76 76 0 1 1 121.32 0 4 4 0 1 1-6.38-4.82 68 68 0 1 0-108.56 0 4 4 0 1 1-6.38 4.82Z"
  })]
});
var _default = exports.default = CellTower;
