// phosphor-react-native/lib/commonjs/thin/Cheese.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Cheese = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "cheese-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-cheese-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M184 36a4.05 4.05 0 0 0-1.15.17l-160 48A4 4 0 0 0 20 88v24a4 4 0 0 0 4 4h8a20.11 20.11 0 0 1 20 19.61 19.59 19.59 0 0 1-5.72 14.14A20.92 20.92 0 0 1 31.46 156H24a4 4 0 0 0-4 4v32a4 4 0 0 0 4 4h200a12 12 0 0 0 12-12V88a52.06 52.06 0 0 0-52-52Zm.57 8a44.06 44.06 0 0 1 43.24 40H51.25ZM196 104a28 28 0 1 1-53.29-12h50.58a28 28 0 0 1 2.71 12Zm-56 84H84v-4a28 28 0 0 1 56 0Zm88-4a4 4 0 0 1-4 4h-76v-4a36 36 0 0 0-72 0v4H28v-24h3.46A29 29 0 0 0 52 155.35a27.52 27.52 0 0 0 8-19.89A28.14 28.14 0 0 0 32 108h-4V92h106.07a36 36 0 1 0 67.86 0H228Z"
  })]
});
var _default = exports.default = Cheese;
