// phosphor-react-native/lib/commonjs/thin/CircleDashed.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const CircleDashed = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "circle-dashed-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-circle-dashed-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M155.87 36.06a4 4 0 0 1-3.87 3 4.29 4.29 0 0 1-1-.13 92 92 0 0 0-46 0 4 4 0 0 1-2-7.74 100.09 100.09 0 0 1 50 0 4 4 0 0 1 2.87 4.87ZM56.65 57.94a100.18 100.18 0 0 0-25 43.29 4 4 0 0 0 7.71 2.14 92.06 92.06 0 0 1 23-39.82 4 4 0 1 0-5.7-5.61Zm-17.29 94.68a4 4 0 0 0-7.71 2.14 100.08 100.08 0 0 0 25 43.31 4 4 0 1 0 5.71-5.61 91.91 91.91 0 0 1-23-39.84ZM151 217.09a92 92 0 0 1-46 0 4 4 0 0 0-2 7.75 100 100 0 0 0 50 0 4 4 0 1 0-2-7.74Zm70.58-67.25a4 4 0 0 0-4.92 2.79 92.12 92.12 0 0 1-23 39.82 4 4 0 1 0 5.7 5.61 100.18 100.18 0 0 0 25-43.29 4 4 0 0 0-2.78-4.93Zm-4.94-46.46a4 4 0 0 0 7.71-2.14 100.08 100.08 0 0 0-25-43.31 4 4 0 1 0-5.71 5.61 91.91 91.91 0 0 1 23 39.84Z"
  })]
});
var _default = exports.default = CircleDashed;
