// phosphor-react-native/lib/commonjs/thin/ContactlessPayment.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const ContactlessPayment = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "contactless-payment-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-contactless-payment-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M93.54 102.13a55.39 55.39 0 0 1 0 51.74A4 4 0 0 1 90 156a4.07 4.07 0 0 1-1.87-.46 4 4 0 0 1-1.67-5.41 46.73 46.73 0 0 0 0-44.26 4 4 0 1 1 7.08-3.74Zm50.58-33.66a4 4 0 0 0-1.65 5.41 114.67 114.67 0 0 1 0 108.24 4 4 0 1 0 7.06 3.76 122.65 122.65 0 0 0 0-115.76 4 4 0 0 0-5.41-1.65Zm-28 16a4 4 0 0 0-1.65 5.41 81 81 0 0 1 0 76.24 4 4 0 1 0 7.06 3.76 89 89 0 0 0 0-83.76 4 4 0 0 0-5.41-1.65ZM228 128A100 100 0 1 1 128 28a100.11 100.11 0 0 1 100 100Zm-8 0a92 92 0 1 0-92 92 92.1 92.1 0 0 0 92-92Z"
  })]
});
var _default = exports.default = ContactlessPayment;
