// phosphor-react-native/lib/commonjs/thin/Cookie.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Cookie = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "cookie-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-cookie-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M161.66 166.34a8 8 0 1 1-11.32 0 8 8 0 0 1 11.32 0Zm-75.32-8a8 8 0 1 0 11.32 0 8 8 0 0 0-11.32 0Zm3.32-56a8 8 0 1 0 0 11.32 8 8 0 0 0 0-11.32Zm36.68 16a8 8 0 1 0 11.32 0 8 8 0 0 0-11.32 0ZM228 128A100 100 0 1 1 128 28a4 4 0 0 1 4 4 44.05 44.05 0 0 0 44 44 4 4 0 0 1 4 4 44.05 44.05 0 0 0 44 44 4 4 0 0 1 4 4Zm-8.08 3.84a52.08 52.08 0 0 1-47.78-48 52.08 52.08 0 0 1-48-47.78 92 92 0 1 0 95.76 95.76Z"
  })]
});
var _default = exports.default = Cookie;
