// phosphor-react-native/lib/commonjs/thin/Cricket.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Cricket = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "cricket-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-cricket-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M240.49 84.2 187.8 31.51a12 12 0 0 0-17 0L63.52 138.83a12 12 0 0 0 0 17L87 179.31l-49.83 49.86a4 4 0 0 0 5.66 5.66L92.69 185l23.51 23.51a12 12 0 0 0 17 0l107.29-107.34a12 12 0 0 0 0-16.97Zm-113 118.63a4 4 0 0 1-5.66 0l-23.49-23.52 32.49-32.48a4 4 0 0 0-5.66-5.66l-32.48 32.49-23.52-23.52a4 4 0 0 1 0-5.66L105.66 108H164v58.34ZM234.83 95.51 172 158.34V104a4 4 0 0 0-4-4h-54.34l62.83-62.83a4 4 0 0 1 5.66 0l52.68 52.68a4 4 0 0 1 0 5.66ZM60 84a24 24 0 1 0-24-24 24 24 0 0 0 24 24Zm0-40a16 16 0 1 1-16 16 16 16 0 0 1 16-16Z"
  })]
});
var _default = exports.default = Cricket;
