// phosphor-react-native/lib/commonjs/thin/Cursor.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Cursor = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "cursor-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-cursor-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M165.17 135.51a4 4 0 0 1 1.17-6.46l46.31-17.79.16-.06a12 12 0 0 0-1.05-22.33L51.67 36.6A12 12 0 0 0 36.6 51.67l52.27 160.09a11.86 11.86 0 0 0 10.8 8.24h.58a11.86 11.86 0 0 0 11-7.19l.06-.16 17.79-46.31a4 4 0 0 1 6.47-1.17l51.31 51.32a12 12 0 0 0 17 0l12.69-12.69a12 12 0 0 0 0-17Zm45.66 62.63-12.69 12.69a4 4 0 0 1-5.66 0l-51.31-51.31a12 12 0 0 0-8.48-3.52 12.13 12.13 0 0 0-2.24.21 12 12 0 0 0-8.77 7l-.06.16-17.79 46.31a4 4 0 0 1-7.36-.42L44.2 49.19a4 4 0 0 1 5-5l160.07 52.28a4 4 0 0 1 .42 7.36l-46.31 17.78-.16.07a12 12 0 0 0-3.71 19.49l51.32 51.31a4 4 0 0 1 0 5.66Z"
  })]
});
var _default = exports.default = Cursor;
