// phosphor-react-native/lib/commonjs/thin/Equalizer.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Equalizer = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "equalizer-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-equalizer-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M76 96a4 4 0 0 1-4 4H24a4 4 0 0 1 0-8h48a4 4 0 0 1 4 4Zm-4 28H24a4 4 0 0 0 0 8h48a4 4 0 0 0 0-8Zm0 32H24a4 4 0 0 0 0 8h48a4 4 0 0 0 0-8Zm0 32H24a4 4 0 0 0 0 8h48a4 4 0 0 0 0-8Zm80-64h-48a4 4 0 0 0 0 8h48a4 4 0 0 0 0-8Zm0 32h-48a4 4 0 0 0 0 8h48a4 4 0 0 0 0-8Zm0 32h-48a4 4 0 0 0 0 8h48a4 4 0 0 0 0-8Zm80-96h-48a4 4 0 0 0 0 8h48a4 4 0 0 0 0-8Zm-48-24h48a4 4 0 0 0 0-8h-48a4 4 0 0 0 0 8Zm48 56h-48a4 4 0 0 0 0 8h48a4 4 0 0 0 0-8Zm0 32h-48a4 4 0 0 0 0 8h48a4 4 0 0 0 0-8Zm0 32h-48a4 4 0 0 0 0 8h48a4 4 0 0 0 0-8Z"
  })]
});
var _default = exports.default = Equalizer;
