// phosphor-react-native/lib/commonjs/thin/FourK.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const FourK = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "four-k-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-four-k-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M28 48a4 4 0 0 1 4-4h192a4 4 0 0 1 0 8H32a4 4 0 0 1-4-4Zm196 156H32a4 4 0 0 0 0 8h192a4 4 0 0 0 0-8ZM144 76a4 4 0 0 0-4 4v96a4 4 0 0 0 8 0v-30.5l17.75-20.28L196.54 178a4 4 0 1 0 6.92-4l-32.16-55.13L203 82.63a4 4 0 1 0-6-5.26l-49 56V80a4 4 0 0 0-4-4ZM92 176v-20H40a4 4 0 0 1-3.16-6.46l56-72A4 4 0 0 1 100 80v68h12a4 4 0 0 1 0 8h-12v20a4 4 0 0 1-8 0Zm0-28V91.66L48.18 148Z"
  })]
});
var _default = exports.default = FourK;
