// phosphor-react-native/lib/commonjs/thin/GraphicsCard.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const GraphicsCard = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "graphics-card-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-graphics-card-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M232 52H16a4 4 0 0 0-4 4v152a4 4 0 0 0 8 0v-20h24v20a4 4 0 0 0 8 0v-20h24v20a4 4 0 0 0 8 0v-20h24v20a4 4 0 0 0 8 0v-20h116a12 12 0 0 0 12-12V64a12 12 0 0 0-12-12Zm4 124a4 4 0 0 1-4 4H20V60h212a4 4 0 0 1 4 4Zm-60-92a36 36 0 1 0 36 36 36 36 0 0 0-36-36Zm-28 36a27.89 27.89 0 0 1 5.58-16.76l39.18 39.18A28 28 0 0 1 148 120Zm50.42 16.76-39.18-39.18a28 28 0 0 1 39.18 39.18ZM80 84a36 36 0 1 0 36 36 36 36 0 0 0-36-36Zm-28 36a27.89 27.89 0 0 1 5.58-16.76l39.18 39.18A28 28 0 0 1 52 120Zm50.42 16.76L63.24 97.58a28 28 0 0 1 39.18 39.18Z"
  })]
});
var _default = exports.default = GraphicsCard;
