// phosphor-react-native/lib/commonjs/thin/Hamburger.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Hamburger = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "hamburger-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-hamburger-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M48.07 100h159.86a12 12 0 0 0 11.8-14.55c-3.39-16-14.56-30.68-31.46-41.23S150.11 28 128 28s-43.51 5.76-60.27 16.22-28.07 25.19-31.46 41.23A12 12 0 0 0 48.07 100Zm-4-12.89C50.26 58 86.33 36 128 36s77.74 22 83.91 51.11a4 4 0 0 1-.82 3.39 4 4 0 0 1-3.16 1.5H48.07a4 4 0 0 1-3.16-1.5 4 4 0 0 1-.82-3.39Zm186.54 69.13-42.56 15.48-38.58-15.43a4 4 0 0 0-3 0L108 171.69l-38.51-15.4a4 4 0 0 0-2.86 0l-44 16A4 4 0 0 0 24 180a4.08 4.08 0 0 0 1.37-.24L44 173v11a36 36 0 0 0 36 36h96a36 36 0 0 0 36-36v-12.47l21.37-7.77a4 4 0 0 0-2.74-7.52ZM204 184a28 28 0 0 1-28 28H80a28 28 0 0 1-28-28v-13.93l15.93-5.79 38.58 15.43a4 4 0 0 0 3 0l38.49-15.4 38.51 15.4a4 4 0 0 0 2.86 0l14.63-5.27ZM20 128a4 4 0 0 1 4-4h208a4 4 0 0 1 0 8H24a4 4 0 0 1-4-4Z"
  })]
});
var _default = exports.default = Hamburger;
