// phosphor-react-native/lib/commonjs/thin/HandWaving.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const HandWaving = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "hand-waving-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-hand-waving-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M216.71 102 199.4 72a24 24 0 0 0-43.57 4.52L134.74 40a24 24 0 0 0-43.69 4.9A24 24 0 0 0 51.61 72l5.31 9.19a24 24 0 0 0-25.71 35.47l40 69.32a83.4 83.4 0 0 0 51 39.14 84.45 84.45 0 0 0 21.88 2.9 84 84 0 0 0 72.6-126Zm.67 61.67A76 76 0 0 1 78.16 182l-40-69.32a16 16 0 0 1 27.71-16L85.09 130a4 4 0 0 0 6.91-4L58.54 68a16 16 0 1 1 27.71-16l31.17 54a4 4 0 1 0 6.93-4L100.1 60a16 16 0 0 1 27.72-16l35 60.63a44 44 0 0 0-7.28 57.61 4 4 0 1 0 6.62-4.49 36 36 0 0 1 8.22-49 4 4 0 0 0 1.07-5.2L164.76 92a16 16 0 1 1 27.71-16l17.31 30a75.52 75.52 0 0 1 7.6 57.65ZM188.12 32.74A4 4 0 0 1 193 29.9 55.65 55.65 0 0 1 227.11 56l.33.58A4 4 0 0 1 226 62a4 4 0 0 1-5.47-1.46l-.33-.57A47.62 47.62 0 0 0 191 37.63a4 4 0 0 1-2.88-4.89ZM77.75 234.48a4 4 0 0 1-3.14 1.52 4 4 0 0 1-2.47-.86A115.55 115.55 0 0 1 43.53 202a4 4 0 1 1 6.92-4 107.72 107.72 0 0 0 26.64 30.86 4 4 0 0 1 .66 5.62Z"
  })]
});
var _default = exports.default = HandWaving;
