// phosphor-react-native/lib/commonjs/thin/Jeep.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Jeep = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "jeep-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-jeep-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M240 92h-12.77l-10-46.51a12.07 12.07 0 0 0-11.7-9.49H50.47a12.07 12.07 0 0 0-11.74 9.49L28.77 92H16a4 4 0 0 0 0 8h12v100a12 12 0 0 0 12 12h24a12 12 0 0 0 12-12v-28h104v28a12 12 0 0 0 12 12h24a12 12 0 0 0 12-12V100h12a4 4 0 0 0 0-8ZM46.56 47.16A4 4 0 0 1 50.47 44h155.06a4 4 0 0 1 3.91 3.16L219.05 92H37ZM68 200a4 4 0 0 1-4 4H40a4 4 0 0 1-4-4v-28h32Zm148 4h-24a4 4 0 0 1-4-4v-28h32v28a4 4 0 0 1-4 4Zm4-40h-72v-36a4 4 0 0 0-8 0v36h-24v-36a4 4 0 0 0-8 0v36H36v-64h184ZM60 132a8 8 0 1 1 8 8 8 8 0 0 1-8-8Zm120 0a8 8 0 1 1 8 8 8 8 0 0 1-8-8Z"
  })]
});
var _default = exports.default = Jeep;
