// phosphor-react-native/lib/commonjs/thin/Lighthouse.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Lighthouse = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "lighthouse-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-lighthouse-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M208 84a4 4 0 0 0-4 4v20h-18.77L180 55.6a4 4 0 0 0-1.34-2.6l-42.88-38.13-.1-.09a12 12 0 0 0-15.36 0l-.1.09L77.34 53A4 4 0 0 0 76 55.6L70.77 108H52V88a4 4 0 0 0-8 0v24a4 4 0 0 0 4 4h22l-9.92 98.86A12 12 0 0 0 72 228h112a12 12 0 0 0 11.94-13.16L186 116h22a4 4 0 0 0 4-4V88a4 4 0 0 0-4-4Zm-82.52-63.11a4 4 0 0 1 5 0l35 31.11h-75ZM83.62 60h88.76l4.81 48H132V88a4 4 0 0 0-8 0v20H78.81ZM187 218.69a3.94 3.94 0 0 1-3 1.31H72a4 4 0 0 1-4-4.36L72.39 172h111.22l4.39 43.62a3.93 3.93 0 0 1-1 3.07ZM182.81 164H73.19L78 116h100Z"
  })]
});
var _default = exports.default = Lighthouse;
