// phosphor-react-native/lib/commonjs/thin/ListStar.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const ListStar = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "list-star-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-list-star-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M36 64a4 4 0 0 1 4-4h176a4 4 0 0 1 0 8H40a4 4 0 0 1-4-4Zm4 68h56a4 4 0 0 0 0-8H40a4 4 0 0 0 0 8Zm72 56H40a4 4 0 0 0 0 8h72a4 4 0 0 0 0-8Zm122.55-47.3-24.46 20.19 7.46 30.11a4 4 0 0 1-1.49 4.17 4.06 4.06 0 0 1-2.4.79 4 4 0 0 1-2-.55L184 179.18l-27.64 16.27a4 4 0 0 1-5.91-4.41l7.46-30.15-24.46-20.19a4 4 0 0 1 2.24-7.08l32.24-2.49 12.4-28.71a4 4 0 0 1 7.34 0l12.4 28.71 32.24 2.49a4 4 0 0 1 2.24 7.08Zm-12.74.14L197 138.92a4 4 0 0 1-3.36-2.4L184 114.1l-9.68 22.42a4 4 0 0 1-3.36 2.4l-24.77 1.92L165 156.33a4 4 0 0 1 1.33 4l-5.78 23.36 21.45-12.6a4 4 0 0 1 4.06 0l21.47 12.64-5.78-23.36a4 4 0 0 1 1.33-4Z"
  })]
});
var _default = exports.default = ListStar;
