// phosphor-react-native/lib/commonjs/thin/MapPinSimpleArea.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const MapPinSimpleArea = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "map-pin-simple-area-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-map-pin-simple-area-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M124 99.77V176a4 4 0 0 0 8 0V99.77a36 36 0 1 0-8 0ZM128 36a28 28 0 1 1-28 28 28 28 0 0 1 28-28Zm108 140c0 12.46-11.73 23.83-33 32-20.09 7.73-46.72 12-75 12s-54.89-4.25-75-12c-21.29-8.19-33-19.56-33-32 0-18.55 25.81-34.22 67.37-40.88a4 4 0 1 1 1.26 7.88C52.93 148.74 28 162.3 28 176c0 17.39 40.18 36 100 36s100-18.61 100-36c0-13.7-24.93-27.26-60.63-33a4 4 0 1 1 1.26-7.89C210.19 141.78 236 157.45 236 176Z"
  })]
});
var _default = exports.default = MapPinSimpleArea;
