// phosphor-react-native/lib/commonjs/thin/NetworkSlash.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const NetworkSlash = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "network-slash-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-network-slash-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M100 54V40a12 12 0 0 1 12-12h32a12 12 0 0 1 12 12v32a12 12 0 0 1-12 12h-16.39a4 4 0 0 1 0-8H144a4 4 0 0 0 4-4V40a4 4 0 0 0-4-4h-32a4 4 0 0 0-4 4v14a4 4 0 0 1-8 0Zm111 159.31a4 4 0 1 1-5.92 5.38L119 124H68v40h12a12 12 0 0 1 12 12v32a12 12 0 0 1-12 12H48a12 12 0 0 1-12-12v-32a12 12 0 0 1 12-12h12v-40H24a4 4 0 0 1 0-8h87.68L45 42.69a4 4 0 0 1 6-5.38ZM80 172H48a4 4 0 0 0-4 4v32a4 4 0 0 0 4 4h32a4 4 0 0 0 4-4v-32a4 4 0 0 0-4-4Zm152-56h-68a4 4 0 0 0 0 8h24v26.83a4 4 0 1 0 8 0V124h36a4 4 0 0 0 0-8Z"
  })]
});
var _default = exports.default = NetworkSlash;
