// phosphor-react-native/lib/commonjs/thin/OfficeChair.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const OfficeChair = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "office-chair-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-office-chair-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M244 128a4 4 0 0 1-4 4h-20.18A44.06 44.06 0 0 1 176 172h-44v32h28a28 28 0 0 1 28 28 4 4 0 0 1-8 0 20 20 0 0 0-20-20h-28v20a4 4 0 0 1-8 0v-20H96a20 20 0 0 0-20 20 4 4 0 0 1-8 0 28 28 0 0 1 28-28h28v-32H80a44.06 44.06 0 0 1-43.82-40H16a4 4 0 0 1 0-8h24a4 4 0 0 1 4 4 36 36 0 0 0 36 36h96a36 36 0 0 0 36-36 4 4 0 0 1 4-4h24a4 4 0 0 1 4 4Zm-173.06 7.86a12 12 0 0 1-2.82-9.56l13.72-96A12.06 12.06 0 0 1 93.72 20h68.56a12.06 12.06 0 0 1 11.88 10.3l13.72 96A12 12 0 0 1 176 140H80a12 12 0 0 1-9.06-4.14Zm6-5.24A3.93 3.93 0 0 0 80 132h96a3.93 3.93 0 0 0 3-1.38 4 4 0 0 0 .94-3.19l-13.72-96a4 4 0 0 0-4-3.43h-68.5a4 4 0 0 0-4 3.43l-13.72 96a4 4 0 0 0 1 3.19Z"
  })]
});
var _default = exports.default = OfficeChair;
