// phosphor-react-native/lib/commonjs/thin/Onigiri.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Onigiri = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "onigiri-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-onigiri-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "m228.11 148.63-55.88-96-.06-.09a52 52 0 0 0-88.34 0l-.06.09-55.88 96A52 52 0 0 0 72.09 228h111.82a52 52 0 0 0 44.2-79.37ZM164 220H92v-52a4 4 0 0 1 4-4h64a4 4 0 0 1 4 4Zm58.36-22.61A43.43 43.43 0 0 1 183.91 220H172v-52a12 12 0 0 0-12-12H96a12 12 0 0 0-12 12v52H72.09a44 44 0 0 1-37.37-67.2.31.31 0 0 1 .05-.1l55.88-95.95a44 44 0 0 1 74.7 0l55.88 95.95a.31.31 0 0 1 .05.1 43.43 43.43 0 0 1 1.07 44.59Z"
  })]
});
var _default = exports.default = Onigiri;
