// phosphor-react-native/lib/commonjs/thin/Orange.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Orange = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "orange-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-orange-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M155.62 72.69A60.09 60.09 0 0 0 196 16a4 4 0 0 0-4-4h-8a60.09 60.09 0 0 0-56 38.5A60.09 60.09 0 0 0 72 12h-8a4 4 0 0 0 0 8h8a52.06 52.06 0 0 1 51.84 48.11 84 84 0 1 0 31.78 4.58ZM184 20h3.85A52.07 52.07 0 0 1 136 68h-3.85A52.07 52.07 0 0 1 184 20Zm-56 208a76 76 0 1 1 76-76 76.08 76.08 0 0 1-76 76Zm51.94-67.34A53.45 53.45 0 0 1 136.66 204a5 5 0 0 1-.66.05 4 4 0 0 1-.66-7.95 45.33 45.33 0 0 0 36.72-36.71 4 4 0 1 1 7.88 1.32Z"
  })]
});
var _default = exports.default = Orange;
