// phosphor-react-native/lib/commonjs/thin/PaintBucket.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PaintBucket = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "paint-bucket-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-paint-bucket-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M235.33 165.78a4 4 0 0 0-6.66 0C228 166.8 212 191 212 208a20 20 0 0 0 40 0c0-17-16-41.2-16.67-42.22ZM232 220a12 12 0 0 1-12-12c0-10.25 7.49-24.64 12-32.37 4.51 7.73 12 22.1 12 32.37a12 12 0 0 1-12 12Zm3.89-87.6a4 4 0 0 0-1.06-3.74L119.34 13.17a4 4 0 0 0-5.65 0L70.25 56.6 42.82 29.17a4 4 0 0 0-5.65 5.66L64.6 62.26 17.86 109a20 20 0 0 0 0 28.29l84.85 84.85a20 20 0 0 0 28.29 0L210.16 143l23.1-7.7a4 4 0 0 0 2.63-2.9Zm-29.15 3.29a4.06 4.06 0 0 0-1.57 1l-79.83 79.82a12 12 0 0 1-17 0l-84.83-84.88a12 12 0 0 1 0-17l46.74-46.71 34.2 34.2A24 24 0 0 0 141 133a24 24 0 0 0-30.86-36.51l-34.2-34.2 40.61-40.6 108.04 108.05Zm-94.05-31a16 16 0 1 1 0 22.64 16 16 0 0 1 0-22.64Z"
  })]
});
var _default = exports.default = PaintBucket;
