// phosphor-react-native/lib/commonjs/thin/PaintBrushHousehold.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PaintBrushHousehold = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "paint-brush-household-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-paint-brush-household-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M227.81 28.19a28 28 0 0 0-39.6 0l-.21.23-56.22 65.69-13.63-13.63a20 20 0 0 0-28.29 0l-76.69 76.69a4 4 0 0 0 0 5.65l80 80a4 4 0 0 0 5.65 0l76.69-76.69a20 20 0 0 0 0-28.29l-13.63-13.63L227.58 68l.23-.21a28 28 0 0 0 0-39.6ZM96 234.34 73.66 212l25.17-25.18a4 4 0 0 0-5.65-5.65L68 206.34 49.66 188l25.17-25.18a4 4 0 0 0-5.65-5.65L44 182.34 21.66 160 72 109.65 146.35 184ZM222.26 62l-68.85 59a4 4 0 0 0-.23 5.87l16.69 16.69a12 12 0 0 1 0 17L152 178.34 77.66 104l17.86-17.87a12 12 0 0 1 17 0l16.69 16.69a4 4 0 0 0 5.87-.23L194 33.74A20 20 0 0 1 222.26 62Z"
  })]
});
var _default = exports.default = PaintBrushHousehold;
