// phosphor-react-native/lib/commonjs/thin/PersonSimpleSki.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PersonSimpleSki = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "person-simple-ski-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-person-simple-ski-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M172 84a24 24 0 1 0-24-24 24 24 0 0 0 24 24Zm0-40a16 16 0 1 1-16 16 16 16 0 0 1 16-16ZM38.88 83.84l112 32.65 6.33 6.33A4 4 0 0 0 160 124h40a4 4 0 0 0 0-8h-38.34l-30.83-30.83a4 4 0 0 0-5.66 0l-12 12-72.05-21a4 4 0 0 0-2.24 7.68ZM128 93.65l11.07 11.08-17.14-5Zm106.22 107.27a65.8 65.8 0 0 1-55 8.43L22.88 163.83a4 4 0 1 1 2.24-7.68l78.27 22.79 33-33-49.49-14.1a4 4 0 1 1 2.2-7.69l56 16a4 4 0 0 1 1.73 6.67l-34.68 34.68 69.28 20.17a57.81 57.81 0 0 0 48.35-7.41 4 4 0 0 1 4.44 6.66Z"
  })]
});
var _default = exports.default = PersonSimpleSki;
