// phosphor-react-native/lib/commonjs/thin/Plug.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Plug = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "plug-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-plug-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M234.83 69.17a4 4 0 0 0-5.66 0L192 106.34 149.66 64l37.17-37.17a4 4 0 1 0-5.66-5.66L144 58.34l-29.17-29.17a4 4 0 0 0-5.66 5.66l9.17 9.17-55.8 55.8a36.05 36.05 0 0 0 0 50.91l18.55 18.54-51.92 51.92a4 4 0 0 0 5.66 5.66l51.92-51.92 18.54 18.55a36.06 36.06 0 0 0 50.91 0l55.8-55.8 9.17 9.17a4 4 0 0 0 5.66-5.66L197.66 112l37.17-37.17a4 4 0 0 0 0-5.66ZM150.54 187.8a28 28 0 0 1-39.59 0L68.2 145.05a28 28 0 0 1 0-39.59l55.8-55.8L206.34 132Z"
  })]
});
var _default = exports.default = Plug;
