// phosphor-react-native/lib/commonjs/thin/PushPin.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PushPin = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "push-pin-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-push-pin-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M232.49 84.2 171.8 23.51a12 12 0 0 0-17 0L99.45 79.07c-8.08-3-32.79-9.45-59 11.67a12 12 0 0 0-1 17.84l51.13 51.13-45.41 45.46a4 4 0 0 0 5.66 5.66l45.46-45.47 51.12 51.12a12 12 0 0 0 8.49 3.52h.85a12 12 0 0 0 8.73-4.74c19.83-26.36 16.51-47.18 11.71-58.57l55.3-55.49a12 12 0 0 0 0-17Zm-5.66 11.31L169.56 153a4 4 0 0 0-.75 4.61c10.43 20.85-1.62 42-9.73 52.83a4 4 0 0 1-6 .4L45.17 102.93a4 4 0 0 1 .33-6c12.94-10.4 25.36-13.19 34.94-13.19a43.9 43.9 0 0 1 18.16 3.64 4 4 0 0 0 4.62-.75l57.27-57.46a4 4 0 0 1 5.66 0l60.68 60.69a4 4 0 0 1 0 5.65Z"
  })]
});
var _default = exports.default = PushPin;
