// phosphor-react-native/lib/commonjs/thin/Ranking.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Ranking = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "ranking-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-ranking-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M116.21 101.26a4 4 0 0 1 2.53-5.05l12-4A4 4 0 0 1 136 96v40a4 4 0 0 1-8 0v-34.45l-6.74 2.24a4 4 0 0 1-5.05-2.53ZM244 208a4 4 0 0 1-4 4H16a4 4 0 0 1 0-8h12V104a12 12 0 0 1 12-12h44V56a12 12 0 0 1 12-12h64a12 12 0 0 1 12 12v76h44a12 12 0 0 1 12 12v60h12a4 4 0 0 1 4 4Zm-72-68v64h48v-60a4 4 0 0 0-4-4Zm-80 64h72V56a4 4 0 0 0-4-4H96a4 4 0 0 0-4 4Zm-56 0h48V100H40a4 4 0 0 0-4 4Z"
  })]
});
var _default = exports.default = Ranking;
