// phosphor-react-native/lib/commonjs/thin/ReplitLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const ReplitLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "replit-logo-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-replit-logo-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M216 92h-68V40a12 12 0 0 0-12-12H72a12 12 0 0 0-12 12v48a12 12 0 0 0 12 12h68v56H72a12 12 0 0 0-12 12v48a12 12 0 0 0 12 12h64a12 12 0 0 0 12-12v-52h68a12 12 0 0 0 12-12v-48a12 12 0 0 0-12-12Zm-76 124a4 4 0 0 1-4 4H72a4 4 0 0 1-4-4v-48a4 4 0 0 1 4-4h68Zm0-124H72a4 4 0 0 1-4-4V40a4 4 0 0 1 4-4h64a4 4 0 0 1 4 4Zm80 60a4 4 0 0 1-4 4h-68v-56h68a4 4 0 0 1 4 4Z"
  })]
});
var _default = exports.default = ReplitLogo;
