// phosphor-react-native/lib/commonjs/thin/Rocket.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Rocket = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "rocket-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-rocket-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M148 224a4 4 0 0 1-4 4h-32a4 4 0 0 1 0-8h32a4 4 0 0 1 4 4Zm-20-116a8 8 0 1 0-8-8 8 8 0 0 0 8 8Zm91.72 47-12.37 55.6a11.9 11.9 0 0 1-7.72 8.71 12.17 12.17 0 0 1-4 .69 11.94 11.94 0 0 1-7.43-2.6L159.85 196h-63.7l-28.34 21.4a11.94 11.94 0 0 1-7.43 2.6 12.17 12.17 0 0 1-4-.69 11.9 11.9 0 0 1-7.72-8.71L36.28 155a12 12 0 0 1 2.5-10.28l29.35-35.23c3.3-53.33 41.83-86.68 52.52-94.94a12 12 0 0 1 14.7 0c10.69 8.26 49.22 41.61 52.52 94.94l29.35 35.23a12 12 0 0 1 2.5 10.28ZM97.11 188h61.78C214.07 92.49 145 32.05 130.46 20.84a4 4 0 0 0-4.92 0C111 32.05 41.93 92.49 97.11 188Zm-7.52 2.93C75.12 165.56 68.93 142.52 68 122.06L44.92 149.8a4 4 0 0 0-.83 3.43l12.36 55.63a4 4 0 0 0 6.41 2.26l.09-.07Zm121.49-41.13L188 122.06c-.89 20.46-7.08 43.5-21.55 68.87l26.64 20.12.09.07a4 4 0 0 0 6.41-2.26l12.36-55.63a4 4 0 0 0-.87-3.43Z"
  })]
});
var _default = exports.default = Rocket;
