// phosphor-react-native/lib/commonjs/thin/Shower.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Shower = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "shower-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-shower-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M60 236a8 8 0 1 1-8-8 8 8 0 0 1 8 8Zm24-40a8 8 0 1 0 8 8 8 8 0 0 0-8-8Zm-64 0a8 8 0 1 0 8 8 8 8 0 0 0-8-8Zm32-32a8 8 0 1 0 8 8 8 8 0 0 0-8-8ZM252 40a4 4 0 0 1-4 4h-30.34l-29.93 29.93L165.86 202a12 12 0 0 1-8.17 9.44 12.09 12.09 0 0 1-3.69.56 12 12 0 0 1-8.46-3.52l-98-98A12 12 0 0 1 54 90.14l128-21.87 31.1-31.1A4 4 0 0 1 216 36h32a4 4 0 0 1 4 4Zm-72.89 36.89L55.37 98a4 4 0 0 0-2.19 6.78l98 98a4 4 0 0 0 6.78-2.17Z"
  })]
});
var _default = exports.default = Shower;
