// phosphor-react-native/lib/commonjs/thin/SmileyMelting.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SmileyMelting = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "smiley-melting-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-smiley-melting-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M172 140a8 8 0 1 1-8-8 8 8 0 0 1 8 8Zm-48-48a8 8 0 1 0-8 8 8 8 0 0 0 8-8Zm74.21-35.2A100 100 0 0 0 53.46 194.67a4 4 0 1 0 6-5.34 92 92 0 1 1 137.16 0 4 4 0 0 0 6 5.34 100 100 0 0 0-4.41-137.87ZM152 172h-16c-12 0-25.28-4.92-35.42-13.16C89.89 150.15 84 139.19 84 128a43.89 43.89 0 0 1 .61-7.34 4 4 0 0 0-7.89-1.32A52.17 52.17 0 0 0 76 128c0 29.29 32.25 52 60 52h16a12 12 0 0 1 0 24H96a20 20 0 0 0 0 40 4 4 0 0 0 0-8 12 12 0 0 1 0-24h56a20 20 0 0 0 0-40Z"
  })]
});
var _default = exports.default = SmileyMelting;
