// phosphor-react-native/lib/commonjs/thin/Spiral.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Spiral = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "spiral-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-spiral-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M244 144a4 4 0 0 1-8 0A100.11 100.11 0 0 0 136 44a92.1 92.1 0 0 0-92 92 84.09 84.09 0 0 0 84 84 76.08 76.08 0 0 0 76-76 68.07 68.07 0 0 0-68-68 60.07 60.07 0 0 0-60 60 52.06 52.06 0 0 0 52 52 44.05 44.05 0 0 0 44-44 36 36 0 0 0-36-36 28 28 0 0 0-28 28 20 20 0 0 0 20 20 12 12 0 0 0 12-12 4 4 0 0 0-4-4 4 4 0 0 1 0-8 12 12 0 0 1 12 12 20 20 0 0 1-20 20 28 28 0 0 1-28-28 36 36 0 0 1 36-36 44.05 44.05 0 0 1 44 44 52.06 52.06 0 0 1-52 52 60.07 60.07 0 0 1-60-60 68.07 68.07 0 0 1 68-68 76.08 76.08 0 0 1 76 76 84.09 84.09 0 0 1-84 84 92.1 92.1 0 0 1-92-92A100.11 100.11 0 0 1 136 36a108.12 108.12 0 0 1 108 108Z"
  })]
});
var _default = exports.default = Spiral;
