// phosphor-react-native/lib/commonjs/thin/SpotifyLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SpotifyLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "spotify-logo-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-spotify-logo-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 28a100 100 0 1 0 100 100A100.11 100.11 0 0 0 128 28Zm0 192a92 92 0 1 1 92-92 92.1 92.1 0 0 1-92 92Zm27.54-52.13a4 4 0 0 1-5.41 1.67 46.73 46.73 0 0 0-44.26 0 4.07 4.07 0 0 1-1.87.46 4 4 0 0 1-1.87-7.54 55.39 55.39 0 0 1 51.74 0 4 4 0 0 1 1.67 5.41Zm32-56a4 4 0 0 1-5.41 1.65 114.67 114.67 0 0 0-108.24 0 4 4 0 0 1-3.76-7.06 122.65 122.65 0 0 1 115.76 0 4 4 0 0 1 1.64 5.41Zm-16 28a4 4 0 0 1-5.41 1.65 81 81 0 0 0-76.24 0 4 4 0 0 1-3.76-7.06 89 89 0 0 1 83.76 0 4 4 0 0 1 1.64 5.42Z"
  })]
});
var _default = exports.default = SpotifyLogo;
