// phosphor-react-native/lib/commonjs/thin/SprayBottle.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SprayBottle = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "spray-bottle-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-spray-bottle-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M200 76a4 4 0 0 0 4-4 52.06 52.06 0 0 0-52-52H80a12 12 0 0 0-12 12v48a28 28 0 0 1-28 28 4 4 0 0 0 0 8 36 36 0 0 0 36-36v-4h40v28.62a27.85 27.85 0 0 1-10.51 21.86l-16 12.79A35.8 35.8 0 0 0 76 167.38V224a12 12 0 0 0 12 12h104a12 12 0 0 0 12-12v-12.53A266.86 266.86 0 0 0 167.09 76ZM76 32a4 4 0 0 1 4-4h72a44.06 44.06 0 0 1 43.82 40H76Zm120 179.47V224a4 4 0 0 1-4 4H88a4 4 0 0 1-4-4v-56.62a27.85 27.85 0 0 1 10.51-21.86l16-12.79A35.8 35.8 0 0 0 124 104.62V76h33.76A258.83 258.83 0 0 1 196 211.47Z"
  })]
});
var _default = exports.default = SprayBottle;
