// phosphor-react-native/lib/commonjs/thin/StarOfDavid.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const StarOfDavid = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "star-of-david-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-star-of-david-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "m196.61 128 30.86-54a4 4 0 0 0-3.47-6h-61.67l-30.86-54a4 4 0 0 0-6.94 0L93.66 68H32a4 4 0 0 0-3.47 6l30.85 54-30.85 54a4 4 0 0 0 3.47 6h61.66l30.87 54a4 4 0 0 0 6.94 0l30.86-54H224a4 4 0 0 0 3.47-6Zm20.5-52L192 119.94 166.9 76Zm-29.72 52-29.7 52H98.3l-29.71-52L98.3 76h59.39ZM128 24.06l25.12 44h-50.25ZM38.89 76h50.19L64 119.94Zm0 103.92L64 136.06 89.08 180Zm89.11 52-25.13-44h50.25ZM166.9 180l25.1-43.94L217.11 180Z"
  })]
});
var _default = exports.default = StarOfDavid;
