// phosphor-react-native/lib/commonjs/thin/Stethoscope.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Stethoscope = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "stethoscope-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-stethoscope-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M216 160a8 8 0 1 1-8-8 8 8 0 0 1 8 8Zm-4.17 35.79A44.06 44.06 0 0 1 168 236h-24a44.05 44.05 0 0 1-44-44v-44.15A60 60 0 0 1 44 88V40a4 4 0 0 1 4-4h24a4 4 0 0 1 0 8H52v44a52 52 0 0 0 52.69 52C133 139.63 156 115.93 156 87.17V44h-20a4 4 0 0 1 0-8h24a4 4 0 0 1 4 4v47.17c0 32-24.84 58.59-56 60.69V192a36 36 0 0 0 36 36h24a36.05 36.05 0 0 0 35.8-32.25 36 36 0 1 1 8 0ZM236 160a28 28 0 1 0-28 28 28 28 0 0 0 28-28Z"
  })]
});
var _default = exports.default = Stethoscope;
