// phosphor-react-native/lib/commonjs/thin/TennisBall.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const TennisBall = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "tennis-ball-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-tennis-ball-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M198.74 57.29A99.39 99.39 0 0 0 128 28h-.19a100 100 0 1 0 70.93 29.29ZM62.92 63a91.38 91.38 0 0 1 60.8-26.85 91.31 91.31 0 0 1-26.85 60.74 91.44 91.44 0 0 1-60.77 26.84A91.61 91.61 0 0 1 62.92 63Zm-26.83 68.74a99.36 99.36 0 0 0 66.44-29.2 99.25 99.25 0 0 0 29.21-66.45 91.9 91.9 0 0 1 88.17 88.17 100 100 0 0 0-95.65 95.65 91.9 91.9 0 0 1-88.17-88.17Zm157 61.31a91.38 91.38 0 0 1-60.8 26.85 92 92 0 0 1 87.62-87.63 91.61 91.61 0 0 1-26.83 60.78Z"
  })]
});
var _default = exports.default = TennisBall;
