// phosphor-react-native/lib/commonjs/thin/TextSubscript.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const TextSubscript = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "text-subscript-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-text-subscript-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M244 208a4 4 0 0 1-4 4h-48a4 4 0 0 1-3.2-6.4L232 148a20 20 0 0 0-4-28 20 20 0 0 0-28 4 20.23 20.23 0 0 0-2.89 5.37 4 4 0 0 1-7.55-2.66 28.34 28.34 0 0 1 4-7.52 28 28 0 0 1 44.72 33.7L200 204h40a4 4 0 0 1 4 4ZM146.62 53a4 4 0 0 0-5.64.4L92 109.89 43 53.38a4 4 0 1 0-6 5.24L86.71 116 37 173.38a4 4 0 1 0 6 5.24l49-56.51 49 56.51a4 4 0 0 0 6-5.24L97.29 116 147 58.62a4 4 0 0 0-.38-5.62Z"
  })]
});
var _default = exports.default = TextSubscript;
