// phosphor-react-native/lib/commonjs/thin/Ticket.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Ticket = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "ticket-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-ticket-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M232 100a4 4 0 0 0 4-4V64a12 12 0 0 0-12-12H32a12 12 0 0 0-12 12v32a4 4 0 0 0 4 4 28 28 0 0 1 0 56 4 4 0 0 0-4 4v32a12 12 0 0 0 12 12h192a12 12 0 0 0 12-12v-32a4 4 0 0 0-4-4 28 28 0 0 1 0-56ZM28 192v-28.22a36 36 0 0 0 0-71.56V64a4 4 0 0 1 4-4h60v136H32a4 4 0 0 1-4-4Zm168-64a36.06 36.06 0 0 0 32 35.78V192a4 4 0 0 1-4 4H100V60h124a4 4 0 0 1 4 4v28.22A36.06 36.06 0 0 0 196 128Z"
  })]
});
var _default = exports.default = Ticket;
