// phosphor-react-native/lib/commonjs/thin/TriangleDashed.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const TriangleDashed = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "triangle-dashed-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-triangle-dashed-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M156 216a4 4 0 0 1-4 4h-48a4 4 0 0 1 0-8h48a4 4 0 0 1 4 4Zm77.34-25.91-4.66-8.09a4 4 0 1 0-6.93 4l4.65 8.08a11.45 11.45 0 0 1 0 11.72 12.47 12.47 0 0 1-10.94 6.2H192a4 4 0 0 0 0 8h23.46a20.36 20.36 0 0 0 17.86-10.2 19.52 19.52 0 0 0 .02-19.71ZM64 212H40.54a12.47 12.47 0 0 1-10.94-6.2 11.45 11.45 0 0 1 0-11.72l4.65-8.08a4 4 0 1 0-6.93-4l-4.66 8.09a19.52 19.52 0 0 0 0 19.71A20.36 20.36 0 0 0 40.54 220H64a4 4 0 0 0 0-8Zm138.18-64a4 4 0 0 0 3.47-6l-23-40a4 4 0 1 0-6.93 4l23 40a4 4 0 0 0 3.46 2Zm-56.3-109.78a20.75 20.75 0 0 0-35.76 0L96.42 62a4 4 0 1 0 6.94 4l13.69-23.79a12.76 12.76 0 0 1 21.9 0L152.64 66a4 4 0 0 0 6.94-4Zm-67 62.31a4 4 0 0 0-5.49 1.47l-23 40a4 4 0 0 0 6.94 4l23-40a4 4 0 0 0-1.48-5.47Z"
  })]
});
var _default = exports.default = TriangleDashed;
