// phosphor-react-native/lib/commonjs/thin/Truck.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Truck = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "truck-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-truck-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "m251.71 118.52-14-35A12 12 0 0 0 226.58 76H188V64a4 4 0 0 0-4-4H32a12 12 0 0 0-12 12v112a12 12 0 0 0 12 12h20.29a28 28 0 0 0 55.42 0h56.58a28 28 0 0 0 55.42 0H240a12 12 0 0 0 12-12v-64a4 4 0 0 0-.29-1.48ZM188 84h38.58a4 4 0 0 1 3.72 2.51L242.09 116H188ZM28 72a4 4 0 0 1 4-4h148v72H28Zm52 140a20 20 0 1 1 20-20 20 20 0 0 1-20 20Zm84.29-24h-56.58a28 28 0 0 0-55.42 0H32a4 4 0 0 1-4-4v-36h152v18.71A28.05 28.05 0 0 0 164.29 188ZM192 212a20 20 0 1 1 20-20 20 20 0 0 1-20 20Zm52-28a4 4 0 0 1-4 4h-20.29A28 28 0 0 0 188 164.29V124h56Z"
  })]
});
var _default = exports.default = Truck;
