// phosphor-react-native/lib/commonjs/thin/UsersFour.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const UsersFour = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "users-four-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-users-four-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M29.6 123.2a4 4 0 0 0 5.6-.8 56 56 0 0 1 89.6 0 3.93 3.93 0 0 0 6.38 0 56 56 0 0 1 89.6 0 4 4 0 1 0 6.4-4.8 63.55 63.55 0 0 0-32.5-22.85 36 36 0 1 0-37.4 0 63.39 63.39 0 0 0-29.3 19 63.34 63.34 0 0 0-29.3-19 36 36 0 1 0-37.4 0A63.61 63.61 0 0 0 28.8 117.6a4 4 0 0 0 .8 5.6ZM148 64a28 28 0 1 1 28 28 28 28 0 0 1-28-28Zm-96 0a28 28 0 1 1 28 28 28 28 0 0 1-28-28Zm142.7 134.75a36 36 0 1 0-37.4 0A63.39 63.39 0 0 0 128 217.7a63.34 63.34 0 0 0-29.3-18.95 36 36 0 1 0-37.4 0 63.61 63.61 0 0 0-32.5 22.85 4 4 0 0 0 6.4 4.8 56 56 0 0 1 89.6 0 3.93 3.93 0 0 0 6.38 0 56 56 0 0 1 89.6 0 4 4 0 0 0 6.4-4.8 63.55 63.55 0 0 0-32.48-22.85ZM52 168a28 28 0 1 1 28 28 28 28 0 0 1-28-28Zm96 0a28 28 0 1 1 28 28 28 28 0 0 1-28-28Z"
  })]
});
var _default = exports.default = UsersFour;
