// phosphor-react-native/lib/commonjs/thin/Virus.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Virus = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "virus-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-virus-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M132 108a24 24 0 1 0-24 24 24 24 0 0 0 24-24Zm-24 16a16 16 0 1 1 16-16 16 16 0 0 1-16 16Zm80 4a12 12 0 1 1-12-12 12 12 0 0 1 12 12Zm-48 48a12 12 0 1 1-12-12 12 12 0 0 1 12 12Zm100-52h-20.09a91.64 91.64 0 0 0-24.12-58.13l15-15a4 4 0 1 0-5.66-5.66l-15 15A91.64 91.64 0 0 0 132 36.09V16a4 4 0 0 0-8 0v20.09a91.64 91.64 0 0 0-58.13 24.12l-15-15a4 4 0 0 0-5.66 5.66l15 15A91.64 91.64 0 0 0 36.09 124H16a4 4 0 0 0 0 8h20.09a91.64 91.64 0 0 0 24.12 58.13l-15 15a4 4 0 0 0 5.66 5.66l15-15A91.64 91.64 0 0 0 124 219.91V240a4 4 0 0 0 8 0v-20.09a91.64 91.64 0 0 0 58.13-24.12l15 15a4 4 0 0 0 5.66-5.66l-15-15A91.64 91.64 0 0 0 219.91 132H240a4 4 0 0 0 0-8Zm-112 88a84 84 0 1 1 84-84 84.09 84.09 0 0 1-84 84Z"
  })]
});
var _default = exports.default = Virus;
