// phosphor-react-native/lib/commonjs/thin/WebhooksLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const WebhooksLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "webhooks-logo-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-webhooks-logo-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M180.7 172h-72.89a44 44 0 1 1-79-30.41 4 4 0 0 1 6.4 4.81A35.67 35.67 0 0 0 28 168a36 36 0 0 0 72 0 4 4 0 0 1 4-4h76.7a12 12 0 1 1 0 8ZM64 180a12 12 0 0 0 9.33-19.54l37.11-60.3a4 4 0 0 0-1.31-5.51A36 36 0 1 1 161 49.58a4 4 0 1 0 7.33-3.19 44 44 0 1 0-66.71 52.83l-35.1 57.05A11.58 11.58 0 0 0 64 156a12 12 0 0 0 0 24Zm128-56a44 44 0 0 0-19.56 4.58l-35.11-57A12 12 0 1 0 128 76a12.24 12.24 0 0 0 2.52-.27L167.63 136a4 4 0 0 0 5.5 1.31A36 36 0 1 1 192 204a4 4 0 0 0 0 8 44 44 0 0 0 0-88Z"
  })]
});
var _default = exports.default = WebhooksLogo;
