// phosphor-react-native/lib/module/regular/ArticleNyTimes.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const ArticleNyTimes = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "article-ny-times__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-article-ny-times',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M128 96h104a8 8 0 0 1 0 16H128a8 8 0 0 1 0-16Zm104 32H128a8 8 0 0 0 0 16h104a8 8 0 0 0 0-16Zm0 32H80a8 8 0 0 0 0 16h152a8 8 0 0 0 0-16Zm0 32H80a8 8 0 0 0 0 16h152a8 8 0 0 0 0-16ZM96 144a8 8 0 0 0 0-16h-8V64h32v8a8 8 0 0 0 16 0V56a8 8 0 0 0-8-8H32a8 8 0 0 0-8 8v16a8 8 0 0 0 16 0v-8h32v64h-8a8 8 0 0 0 0 16Z"
  })]
});
export default ArticleNyTimes;
