// phosphor-react-native/lib/module/regular/BatteryWarningVertical.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const BatteryWarningVertical = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "battery-warning-vertical__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-battery-warning-vertical',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M120 136V96a8 8 0 0 1 16 0v40a8 8 0 0 1-16 0Zm8 24a12 12 0 1 0 12 12 12 12 0 0 0-12-12ZM96 16h64a8 8 0 0 0 0-16H96a8 8 0 0 0 0 16Zm104 40v168a24 24 0 0 1-24 24H80a24 24 0 0 1-24-24V56a24 24 0 0 1 24-24h96a24 24 0 0 1 24 24Zm-16 0a8 8 0 0 0-8-8H80a8 8 0 0 0-8 8v168a8 8 0 0 0 8 8h96a8 8 0 0 0 8-8Z"
  })]
});
export default BatteryWarningVertical;
