// phosphor-react-native/lib/module/regular/CellTower.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const CellTower = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "cell-tower__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-cell-tower',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M135.16 84.42a8 8 0 0 0-14.32 0l-72 144a8 8 0 0 0 14.31 7.16L77 208h102.1l13.79 27.58A8 8 0 0 0 200 240a8 8 0 0 0 7.15-11.58ZM128 105.89 155.06 160h-54.12ZM85 192l8-16h70.1l8 16Zm74.54-98.26a32 32 0 1 0-63 0 8 8 0 1 1-15.74 2.85 48 48 0 1 1 94.46 0 8 8 0 0 1-7.86 6.58 8.74 8.74 0 0 1-1.43-.13 8 8 0 0 1-6.48-9.3Zm-95.39 42.47a80 80 0 1 1 127.7 0 8 8 0 0 1-12.76-9.65 64 64 0 1 0-102.18 0 8 8 0 0 1-12.76 9.65Z"
  })]
});
export default CellTower;
