// phosphor-react-native/lib/module/regular/ChartScatter.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const ChartScatter = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "chart-scatter__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-chart-scatter',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M232 208a8 8 0 0 1-8 8H32a8 8 0 0 1-8-8V48a8 8 0 0 1 16 0v152h184a8 8 0 0 1 8 8Zm-100-48a12 12 0 1 0-12-12 12 12 0 0 0 12 12Zm-24-56a12 12 0 1 0-12-12 12 12 0 0 0 12 12Zm-32 72a12 12 0 1 0-12-12 12 12 0 0 0 12 12Zm96-48a12 12 0 1 0-12-12 12 12 0 0 0 12 12Zm24-40a12 12 0 1 0-12-12 12 12 0 0 0 12 12Zm-20 76a12 12 0 1 0 12-12 12 12 0 0 0-12 12Z"
  })]
});
export default ChartScatter;
