// phosphor-react-native/lib/module/regular/Cheese.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Cheese = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "cheese__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-cheese',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M184 32a7.81 7.81 0 0 0-2.3.34l-160 48A8 8 0 0 0 16 88v24a8 8 0 0 0 8 8h8a16.08 16.08 0 0 1 16 15.69A15.6 15.6 0 0 1 43.42 147a16.87 16.87 0 0 1-12 5.05H24a8 8 0 0 0-8 8v32a8 8 0 0 0 8 8h200a16 16 0 0 0 16-16V88a56.06 56.06 0 0 0-56-56Zm1.12 16a40.06 40.06 0 0 1 38.07 32H78.51Zm6.88 56a24 24 0 1 1-46.62-8h45.24a23.86 23.86 0 0 1 1.38 8ZM88 184a24 24 0 0 1 48 0Zm136 0h-72a40 40 0 0 0-80 0H32v-16a33 33 0 0 0 22.84-9.85A31.39 31.39 0 0 0 64 135.38 32.15 32.15 0 0 0 32 104v-8h96.81a40 40 0 1 0 78.38 0H224Z"
  })]
});
export default Cheese;
