// phosphor-react-native/lib/module/regular/DiceFour.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const DiceFour = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "dice-four__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-dice-four',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M192 32H64a32 32 0 0 0-32 32v128a32 32 0 0 0 32 32h128a32 32 0 0 0 32-32V64a32 32 0 0 0-32-32Zm16 160a16 16 0 0 1-16 16H64a16 16 0 0 1-16-16V64a16 16 0 0 1 16-16h128a16 16 0 0 1 16 16Zm-96-92a12 12 0 1 1-12-12 12 12 0 0 1 12 12Zm56 0a12 12 0 1 1-12-12 12 12 0 0 1 12 12Zm-56 56a12 12 0 1 1-12-12 12 12 0 0 1 12 12Zm56 0a12 12 0 1 1-12-12 12 12 0 0 1 12 12Z"
  })]
});
export default DiceFour;
