// phosphor-react-native/lib/module/regular/Equalizer.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Equalizer = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "equalizer__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-equalizer',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M80 96a8 8 0 0 1-8 8H24a8 8 0 0 1 0-16h48a8 8 0 0 1 8 8Zm-8 24H24a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm0 32H24a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm0 32H24a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm80-64h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm0 32h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm0 32h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm80-96h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm-48-16h48a8 8 0 0 0 0-16h-48a8 8 0 0 0 0 16Zm48 48h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm0 32h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm0 32h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Z"
  })]
});
export default Equalizer;
