// phosphor-react-native/lib/module/regular/Numpad.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Numpad = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "numpad__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-numpad',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M80 48a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm48-16a16 16 0 1 0 16 16 16 16 0 0 0-16-16Zm64 32a16 16 0 1 0-16-16 16 16 0 0 0 16 16ZM64 88a16 16 0 1 0 16 16 16 16 0 0 0-16-16Zm64 0a16 16 0 1 0 16 16 16 16 0 0 0-16-16Zm64 0a16 16 0 1 0 16 16 16 16 0 0 0-16-16ZM64 144a16 16 0 1 0 16 16 16 16 0 0 0-16-16Zm64 0a16 16 0 1 0 16 16 16 16 0 0 0-16-16Zm0 56a16 16 0 1 0 16 16 16 16 0 0 0-16-16Zm64-56a16 16 0 1 0 16 16 16 16 0 0 0-16-16Z"
  })]
});
export default Numpad;
