// phosphor-react-native/lib/module/regular/PresentationChart.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const PresentationChart = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "presentation-chart__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-presentation-chart',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M216 40h-80V24a8 8 0 0 0-16 0v16H40a16 16 0 0 0-16 16v120a16 16 0 0 0 16 16h39.36l-21.61 27a8 8 0 0 0 12.5 10l29.59-37h56.32l29.59 37a8 8 0 1 0 12.5-10l-21.61-27H216a16 16 0 0 0 16-16V56a16 16 0 0 0-16-16Zm0 136H40V56h176v120Zm-112-56v24a8 8 0 0 1-16 0v-24a8 8 0 0 1 16 0Zm32-16v40a8 8 0 0 1-16 0v-40a8 8 0 0 1 16 0Zm32-16v56a8 8 0 0 1-16 0V88a8 8 0 0 1 16 0Z"
  })]
});
export default PresentationChart;
