// phosphor-react-native/lib/module/regular/Tornado.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Tornado = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "tornado__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-tornado',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M232 40a8 8 0 0 1-8 8H56a8 8 0 0 1 0-16h168a8 8 0 0 1 8 8Zm-48 32a8 8 0 0 0-8-8H32a8 8 0 0 0 0 16h144a8 8 0 0 0 8-8Zm-16 32a8 8 0 0 0-8-8H56a8 8 0 0 0 0 16h104a8 8 0 0 0 8-8Zm16 32a8 8 0 0 0-8-8H88a8 8 0 0 0 0 16h88a8 8 0 0 0 8-8Zm0 24h-64a8 8 0 0 0 0 16h64a8 8 0 0 0 0-16Zm-24 32h-32a8 8 0 0 0 0 16h32a8 8 0 0 0 0-16Zm-32 32h-16a8 8 0 0 0 0 16h16a8 8 0 0 0 0-16Z"
  })]
});
export default Tornado;
