// phosphor-react-native/lib/commonjs/duotone/Asclepius.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Asclepius = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "asclepius-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-asclepius-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M88 48v8a32 32 0 0 1-32 32H32v-8a32 32 0 0 1 32-32Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M216 79v1a40 40 0 0 1-40 40h-40v80h8a16 16 0 0 0 10.67-27.93 8 8 0 0 1 10.66-11.92A32 32 0 0 1 144 216h-8v16a8 8 0 0 1-16 0v-16H96a8 8 0 0 1 0-16h24v-80H96a16 16 0 0 0 0 32 8 8 0 0 1 0 16 32 32 0 0 1 0-64h24V24a8 8 0 0 1 16 0v80h40a24 24 0 0 0 24-24v-1a23 23 0 0 0-23-23h-17a8 8 0 0 1 0-16h17a39 39 0 0 1 39 39ZM56 96H32a8 8 0 0 1-8-8v-8a40 40 0 0 1 40-40h32a8 8 0 0 1 0 16 40 40 0 0 1-40 40Zm24-40H64a24 24 0 0 0-24 24h16a24 24 0 0 0 24-24Z"
  })]
});
var _default = exports.default = Asclepius;
