// phosphor-react-native/lib/commonjs/duotone/CoffeeBean.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const CoffeeBean = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "coffee-bean-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-coffee-bean-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M183.79 183.79c-43.14 43.14-103.09 53.14-133.9 22.32s-20.82-90.76 22.32-133.9 103.09-53.14 133.9-22.32 20.82 90.76-22.32 133.9Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M211.75 44.25C195 27.47 170.37 20.79 142.46 25.44c-26.91 4.49-53.87 19.09-75.9 41.12s-36.63 49-41.12 75.9c-4.65 27.91 2 52.51 18.81 69.29C57.54 225.05 75.75 232 96.62 232a103.66 103.66 0 0 0 16.92-1.44c26.91-4.49 53.87-19.09 75.9-41.12s36.63-49 41.12-75.9c4.65-27.91-2.03-52.54-18.81-69.29ZM77.87 77.87c24.69-24.68 55.13-37.94 81.28-37.94a62.26 62.26 0 0 1 29 6.67 108.48 108.48 0 0 0-31.05 16.94c-19.2 15.16-31.63 36.32-36.94 62.89-9.72 48.58-44.7 65.18-67.48 70.84-23.97-28.51-13.28-80.92 25.19-119.4Zm100.26 100.26c-34.69 34.68-80.71 46.78-110.32 31.27a108.72 108.72 0 0 0 31.09-16.94c19.2-15.16 31.63-36.32 36.94-62.89 9.72-48.58 44.7-65.18 67.48-70.84 23.97 28.51 13.28 80.92-25.19 119.4Z"
  })]
});
var _default = exports.default = CoffeeBean;
