// phosphor-react-native/lib/commonjs/duotone/FilmReel.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const FilmReel = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "film-reel-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-film-reel-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 32a96 96 0 1 0 96 96 96 96 0 0 0-96-96ZM80 144a16 16 0 1 1 16-16 16 16 0 0 1-16 16Zm48 48a16 16 0 1 1 16-16 16 16 0 0 1-16 16Zm0-96a16 16 0 1 1 16-16 16 16 0 0 1-16 16Zm48 48a16 16 0 1 1 16-16 16 16 0 0 1-16 16Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M232 216h-48.64A103.95 103.95 0 1 0 128 232h104a8 8 0 0 0 0-16ZM40 128a88 88 0 1 1 88 88 88.1 88.1 0 0 1-88-88Zm88-24a24 24 0 1 0-24-24 24 24 0 0 0 24 24Zm0-32a8 8 0 1 1-8 8 8 8 0 0 1 8-8Zm24 104a24 24 0 1 0-24 24 24 24 0 0 0 24-24Zm-32 0a8 8 0 1 1 8 8 8 8 0 0 1-8-8Zm56-24a24 24 0 1 0-24-24 24 24 0 0 0 24 24Zm0-32a8 8 0 1 1-8 8 8 8 0 0 1 8-8Zm-96-16a24 24 0 1 0 24 24 24 24 0 0 0-24-24Zm0 32a8 8 0 1 1 8-8 8 8 0 0 1-8 8Z"
  })]
});
var _default = exports.default = FilmReel;
