// phosphor-react-native/lib/commonjs/duotone/Numpad.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Numpad = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "numpad-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-numpad-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M192 48v152a16 16 0 0 1-16 16H80a16 16 0 0 1-16-16V48Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M80 48a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm48-16a16 16 0 1 0 16 16 16 16 0 0 0-16-16Zm64 32a16 16 0 1 0-16-16 16 16 0 0 0 16 16ZM64 88a16 16 0 1 0 16 16 16 16 0 0 0-16-16Zm64 0a16 16 0 1 0 16 16 16 16 0 0 0-16-16Zm64 0a16 16 0 1 0 16 16 16 16 0 0 0-16-16ZM64 144a16 16 0 1 0 16 16 16 16 0 0 0-16-16Zm64 0a16 16 0 1 0 16 16 16 16 0 0 0-16-16Zm0 56a16 16 0 1 0 16 16 16 16 0 0 0-16-16Zm64-56a16 16 0 1 0 16 16 16 16 0 0 0-16-16Z"
  })]
});
var _default = exports.default = Numpad;
