// phosphor-react-native/lib/commonjs/duotone/Panorama.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Panorama = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "panorama-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-panorama-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M240 64v128a8 8 0 0 1-3.47 6.6l-68.25-68.25a8 8 0 0 0-11.31 0L131.31 156l-49.65-49.66a8 8 0 0 0-11.32 0L16 160.69V64a8 8 0 0 1 10.05-7.74C43.35 60.89 81.44 72 128 72s84.65-11.11 102-15.73A8 8 0 0 1 240 64Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M241.75 51.32a15.88 15.88 0 0 0-13.86-2.77l-3.48.94C205.61 54.56 170.61 64 128 64s-77.61-9.44-96.41-14.51l-3.48-.94A16 16 0 0 0 8 64v128a16 16 0 0 0 16 16 16.22 16.22 0 0 0 4.18-.55l3.18-.86C50.13 201.49 85.17 192 128 192s77.87 9.49 96.69 14.59l3.18.86A16 16 0 0 0 248 192V64a15.9 15.9 0 0 0-6.25-12.68ZM27.42 64.93C46.94 70.2 83.27 80 128 80s81.06-9.8 100.58-15.07L232 64v118.76l-58.07-58.07a16 16 0 0 0-22.63 0l-20 20-44-44a16 16 0 0 0-22.62 0L24 141.37V64Zm186.42 122.28a391.22 391.22 0 0 0-49-9L142.63 156l20-20Zm-186.71 3.93L24 192v-28l52-52 64.25 64.25q-6-.24-12.25-.25c-45 0-82.72 10.23-100.87 15.14ZM192 108a12 12 0 1 1 12 12 12 12 0 0 1-12-12Z"
  })]
});
var _default = exports.default = Panorama;
