// phosphor-react-native/lib/commonjs/duotone/SmileyMelting.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SmileyMelting = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "smiley-melting-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-smiley-melting-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M224 128a95.63 95.63 0 0 1-24.44 64H56.44A95.67 95.67 0 0 1 32 126.06C33 74.58 75.15 32.73 126.63 32A96 96 0 0 1 224 128Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M176 140a12 12 0 1 1-12-12 12 12 0 0 1 12 12Zm-48-48a12 12 0 1 0-12 12 12 12 0 0 0 12-12Zm73-38a103.24 103.24 0 0 0-73-30h-1.49a104 104 0 0 0-76 173.32 8 8 0 1 0 11.89-10.65 88 88 0 1 1 131.19 0 8 8 0 0 0 11.93 10.66A104 104 0 0 0 201 54Zm-49 114h-16c-21.74 0-48-17.84-48-40a41.33 41.33 0 0 1 .55-6.68 8 8 0 1 0-15.78-2.64A56.9 56.9 0 0 0 72 128c0 14.88 7.46 29.13 21 40.15 12.4 10.07 28.07 15.85 43 15.85h16a8 8 0 0 1 0 16H96a24 24 0 0 0 0 48 8 8 0 0 0 0-16 8 8 0 0 1 0-16h56a24 24 0 0 0 0-48Z"
  })]
});
var _default = exports.default = SmileyMelting;
