// phosphor-react-native/lib/module/fill/ChartScatter.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const ChartScatter = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "chart-scatter-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-chart-scatter-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M216 40H40a16 16 0 0 0-16 16v144a16 16 0 0 0 16 16h176a16 16 0 0 0 16-16V56a16 16 0 0 0-16-16Zm-28 32a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm0 56a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm-40-16a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm-24-40a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm-24 56a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm100 64H56a8 8 0 0 1-8-8V72a8 8 0 0 1 16 0v104h136a8 8 0 0 1 0 16Z"
  })]
});
export default ChartScatter;
