// phosphor-react-native/lib/module/fill/Crane.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Crane = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "crane-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-crane-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M228.12 17.14a8 8 0 0 0-7.88-.2L102 80H32a16 16 0 0 0-16 16v104a16 16 0 0 0 16 16h88a16 16 0 0 0 16-16v-32a7.31 7.31 0 0 0-.08-1.05v-.24a9.6 9.6 0 0 0-.22-1 .09.09 0 0 0 0-.05.64.64 0 0 1 0-.07L113.54 92 216 37.33V160h-16v-8a8 8 0 0 0-16 0v8a16 16 0 0 0 16 16h16a16 16 0 0 0 16-16V24a8 8 0 0 0-3.88-6.86ZM48 96v64H32V96ZM32 200v-24h88v24Z"
  })]
});
export default Crane;
