// phosphor-react-native/lib/module/fill/DotsSix.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const DotsSix = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "dots-six-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-dots-six-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M224 48H32a16 16 0 0 0-16 16v128a16 16 0 0 0 16 16h192a16 16 0 0 0 16-16V64a16 16 0 0 0-16-16ZM68 168a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-56a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm60 56a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-56a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm60 56a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-56a12 12 0 1 1 12-12 12 12 0 0 1-12 12Z"
  })]
});
export default DotsSix;
