// phosphor-react-native/lib/module/fill/DotsSixVertical.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const DotsSixVertical = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "dots-six-vertical-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-dots-six-vertical-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M192 16H64a16 16 0 0 0-16 16v192a16 16 0 0 0 16 16h128a16 16 0 0 0 16-16V32a16 16 0 0 0-16-16Zm-92 184a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-60a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-60a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm56 120a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-60a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-60a12 12 0 1 1 12-12 12 12 0 0 1-12 12Z"
  })]
});
export default DotsSixVertical;
