// phosphor-react-native/lib/module/fill/ListBullets.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const ListBullets = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "list-bullets-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-list-bullets-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M208 32H48a16 16 0 0 0-16 16v160a16 16 0 0 0 16 16h160a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16ZM68 188a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-48a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-48a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm124 92h-88a8 8 0 0 1 0-16h88a8 8 0 0 1 0 16Zm0-48h-88a8 8 0 0 1 0-16h88a8 8 0 0 1 0 16Zm0-48h-88a8 8 0 0 1 0-16h88a8 8 0 0 1 0 16Z"
  })]
});
export default ListBullets;
