// phosphor-react-native/lib/module/fill/ListPlus.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const ListPlus = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "list-plus-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-list-plus-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M208 32H48a16 16 0 0 0-16 16v160a16 16 0 0 0 16 16h160a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16ZM64 72h128a8 8 0 0 1 0 16H64a8 8 0 0 1 0-16Zm56 112H64a8 8 0 0 1 0-16h56a8 8 0 0 1 0 16Zm16-48H64a8 8 0 0 1 0-16h72a8 8 0 0 1 0 16Zm64 32h-16v16a8 8 0 0 1-16 0v-16h-16a8 8 0 0 1 0-16h16v-16a8 8 0 0 1 16 0v16h16a8 8 0 0 1 0 16Z"
  })]
});
export default ListPlus;
