// phosphor-react-native/lib/module/fill/SortDescending.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const SortDescending = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "sort-descending-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-sort-descending-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M208 32H48a16 16 0 0 0-16 16v160a16 16 0 0 0 16 16h160a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16ZM72 72h40a8 8 0 0 1 0 16H72a8 8 0 0 1 0-16Zm0 48h48a8 8 0 0 1 0 16H72a8 8 0 0 1 0-16Zm96 64H72a8 8 0 0 1 0-16h96a8 8 0 0 1 0 16Zm29.66-82.34a8 8 0 0 1-11.32 0L176 91.31V136a8 8 0 0 1-16 0V91.31l-10.34 10.35a8 8 0 0 1-11.32-11.32l24-24a8 8 0 0 1 11.32 0l24 24a8 8 0 0 1 0 11.32Z"
  })]
});
export default SortDescending;
