// phosphor-react-native/lib/commonjs/fill/AmazonLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const AmazonLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "amazon-logo-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-amazon-logo-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M152 136a24 24 0 1 1-24-24 24 24 0 0 1 24 24Zm80-8A104 104 0 1 1 128 24a104.11 104.11 0 0 1 104 104Zm-80-24a40 40 0 1 0 0 64 8 8 0 0 0 16 0v-64a40 40 0 0 0-73.87-21.29 8 8 0 0 0 13.54 8.52A24 24 0 0 1 152 104Zm44.81 65.61a8 8 0 0 0-11.2 1.58 72 72 0 0 1-115.22 0 8 8 0 1 0-12.78 9.62 88 88 0 0 0 140.78 0 8 8 0 0 0-1.58-11.2Z"
  })]
});
var _default = exports.default = AmazonLogo;
