// phosphor-react-native/lib/commonjs/fill/Asclepius.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Asclepius = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "asclepius-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-asclepius-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M136 128v72h11.62A12.25 12.25 0 0 0 160 188.53a12 12 0 0 0-5.93-10.87 4.08 4.08 0 0 1-2.06-3.49v-8.79a4 4 0 0 1 5.25-3.81A28.06 28.06 0 0 1 176 187.71c.16 15.56-13 28.29-28.59 28.29H136v16a8 8 0 0 1-16 0v-16H96a8 8 0 0 1-8-8.53 8.17 8.17 0 0 1 8.27-7.47H120v-72h-15.54c-8.6 0-16 6.6-16.44 15.19a16 16 0 0 0 12.87 16.51 3.94 3.94 0 0 1 3.11 3.89V172a4 4 0 0 1-4 4 36 36 0 0 1-36-36.87c.47-19.65 17-35.13 36.68-35.13H120V24a8 8 0 0 1 16 0v80h32a16 16 0 0 0 16-16.81C183.56 78.6 176.14 72 167.54 72H156a4 4 0 0 1-4-4V44a4 4 0 0 1 4-4h15.22c24.62 0 45.2 20.15 44.77 44.76A44 44 0 0 1 172 128ZM92.66 72H100a4 4 0 0 0 4-4V44a4 4 0 0 0-4-4H64a40 40 0 0 0-40 40v8a8 8 0 0 0 8 8h24a40 40 0 0 0 36.66-24Z"
  })]
});
var _default = exports.default = Asclepius;
