// phosphor-react-native/lib/commonjs/fill/Bandaids.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Bandaids = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "bandaids-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-bandaids-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 116a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm84.28 39.72a40 40 0 1 1-56.56 56.56L128 184.57l-27.72 27.71a40 40 0 1 1-56.56-56.56L71.43 128l-27.71-27.72a40 40 0 1 1 56.56-56.56L128 71.43l27.72-27.71a40 40 0 1 1 56.56 56.56L184.57 128Zm-95.59 17.53-33.94-33.94L55 167a24 24 0 1 0 34 34ZM161.94 128 128 94.06 94.06 128 128 161.94Zm39-39A24 24 0 1 0 167 55l-27.69 27.75 33.94 33.94Z"
  })]
});
var _default = exports.default = Bandaids;
