// phosphor-react-native/lib/commonjs/fill/BeerStein.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const BeerStein = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "beer-stein-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-beer-stein-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M216 88h-16V72a40 40 0 0 0-40-40h-11.18c-11.91-10.2-28-16-44.82-16-35.29 0-64 25.12-64 56v136a16 16 0 0 0 16 16h128a16 16 0 0 0 16-16v-8h16a24 24 0 0 0 24-24v-64a24 24 0 0 0-24-24Zm-112 96a8 8 0 0 1-16 0v-80a8 8 0 0 1 16 0Zm48 0a8 8 0 0 1-16 0v-80a8 8 0 0 1 16 0ZM57 64c4.46-18.24 23.85-32 47-32 13.87 0 27.06 5 36.21 13.78a8 8 0 0 0 5.53 2.22H160a24 24 0 0 1 22.62 16Zm167 112a8 8 0 0 1-8 8h-16v-80h16a8 8 0 0 1 8 8Z"
  })]
});
var _default = exports.default = BeerStein;
