// phosphor-react-native/lib/commonjs/fill/BluetoothX.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const BluetoothX = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "bluetooth-x-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-bluetooth-x-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M184 176a8 8 0 0 1-3.2 6.4l-64 48a8 8 0 0 1-4.8 1.6 7.9 7.9 0 0 1-4.11-1.14 8.3 8.3 0 0 1-3.9-7.18V144l-51.23 38.4a8 8 0 0 1-11.16-1.55 8.26 8.26 0 0 1 1.8-11.43L98.66 128 43.38 86.57a8.19 8.19 0 0 1-2.13-10.93 8 8 0 0 1 11.51-2L104 112V32.24a8.21 8.21 0 0 1 2.83-6.34 8 8 0 0 1 10-.3l33.62 25.2A4 4 0 0 1 152 54v52a4 4 0 0 1-1.6 3.2L125.34 128l55.5 41.6A8 8 0 0 1 184 176Zm53.47-77.87L219.37 80l18.11-18.11a8.21 8.21 0 0 0 .41-11.37 8 8 0 0 0-11.49-.18l-18.35 18.35-18.12-18.11a8.23 8.23 0 0 0-10.83-.88 8 8 0 0 0-.73 12L196.73 80l-18.15 18.13a8.2 8.2 0 0 0-.6 11.1 8 8 0 0 0 11.71.43l18.36-18.35 18.35 18.35a8 8 0 0 0 11.72-.43 8.21 8.21 0 0 0-.61-11.1Z"
  })]
});
var _default = exports.default = BluetoothX;
