// phosphor-react-native/lib/commonjs/fill/Boules.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Boules = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "boules-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-boules-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M232 128c0 56.63-47.38 104-104 104a103.67 103.67 0 0 1-31.52-4.89 4 4 0 0 1-1.62-6.65l125.6-125.61a4 4 0 0 1 6.65 1.62A103.69 103.69 0 0 1 232 128Zm-16.16-55.61a103.16 103.16 0 0 0-6.06-8.56 4 4 0 0 0-6-.33L63.5 203.82a4 4 0 0 0 .33 6 103.16 103.16 0 0 0 8.56 6.06 4 4 0 0 0 5-.54L215.3 77.39a4 4 0 0 0 .54-5Zm-23.67-26.17a103.16 103.16 0 0 0-8.56-6.06 4 4 0 0 0-5 .54L40.7 178.62a4 4 0 0 0-.54 5 103.16 103.16 0 0 0 6.06 8.56 4 4 0 0 0 6 .33L192.5 52.18a4 4 0 0 0-.33-5.96Zm-32.64-17.33A103.67 103.67 0 0 0 128 24C71.38 24 24 71.37 24 128a103.69 103.69 0 0 0 4.89 31.53 4 4 0 0 0 6.65 1.62L161.15 35.54a4 4 0 0 0-1.62-6.65Z"
  })]
});
var _default = exports.default = Boules;
