// phosphor-react-native/lib/commonjs/fill/Carrot.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Carrot = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "carrot-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-carrot-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M232 80h-32.56a64 64 0 0 1-10.19 77.26c-8.52 8.69-19.61 16.92-31.85 24.51a4 4 0 0 1-4.91-.59l-34.84-34.84a8 8 0 0 0-11.49.18 8.23 8.23 0 0 0 .41 11.38l29.88 29.88a4 4 0 0 1-1 6.39C95.74 214.79 53 228.54 46.78 230.48a16 16 0 0 1-21.26-21.26c2.73-8.71 29-90.27 64.86-133.35a4 4 0 0 1 5.9-.26l42.05 42.06a8 8 0 0 0 11.71-.43 8.19 8.19 0 0 0-.6-11.1l-41.36-41.36a4 4 0 0 1 .63-6.18 64 64 0 0 1 67.28-2V24a8 8 0 0 1 8.54-8 8.18 8.18 0 0 1 7.47 8.28v28.41l26.34-26.35a8 8 0 0 1 11.32 11.32L203.31 64h28.41a8.18 8.18 0 0 1 8.28 7.47 8 8 0 0 1-8 8.53Z"
  })]
});
var _default = exports.default = Carrot;
