// phosphor-react-native/lib/commonjs/fill/CurrencyNgn.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const CurrencyNgn = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "currency-ngn-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-currency-ngn-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M143.55 136H160v23ZM96 120h16.45L96 97Zm136 8A104 104 0 1 1 128 24a104.11 104.11 0 0 1 104 104Zm-32 0a8 8 0 0 0-8-8h-16V72a8 8 0 0 0-16 0v48h-27.88L94.51 67.35A8 8 0 0 0 80 72v48H64a8 8 0 0 0 0 16h16v48a8 8 0 0 0 16 0v-48h27.88l37.61 52.65A8 8 0 0 0 168 192a7.91 7.91 0 0 0 2.44-.38A8 8 0 0 0 176 184v-48h16a8 8 0 0 0 8-8Z"
  })]
});
var _default = exports.default = CurrencyNgn;
