// phosphor-react-native/lib/commonjs/fill/Drone.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Drone = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "drone-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-drone-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M189.66 77.66 160 107.31v41.38l29.66 29.65a8 8 0 0 1-11.32 11.32L148.69 160h-41.38l-29.65 29.66a8 8 0 0 1-11.32-11.32L96 148.69v-41.38L66.34 77.66a8 8 0 0 1 11.32-11.32L107.31 96h41.38l29.65-29.66a8 8 0 0 1 11.32 11.32Zm-46.28-6.12a8 8 0 0 0 10.21-4.87 28 28 0 1 1 35.74 35.74A8 8 0 0 0 192 118a7.86 7.86 0 0 0 2.67-.46 44 44 0 1 0-56.16-56.16 8 8 0 0 0 4.87 10.16Zm51.29 67a8 8 0 0 0-5.34 15.08 28 28 0 1 1-35.74 35.74 8 8 0 0 0-15.08 5.34 44 44 0 1 0 56.16-56.16Zm-82 46a8 8 0 0 0-10.21 4.87 28 28 0 1 1-35.74-35.74 8 8 0 0 0-5.34-15.08 44 44 0 1 0 56.16 56.16 8 8 0 0 0-4.92-10.29Zm-51.29-67A7.86 7.86 0 0 0 64 118a8 8 0 0 0 2.67-15.54 28 28 0 1 1 35.74-35.74 8 8 0 1 0 15.08-5.34 44 44 0 1 0-56.16 56.16Z"
  })]
});
var _default = exports.default = Drone;
