// phosphor-react-native/lib/commonjs/fill/EggCrack.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const EggCrack = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "egg-crack-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-egg-crack-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M216 152a88.11 88.11 0 0 1-87.8 88c-50.66.12-90.21-43-88.12-93.62 1.21-29.21 11.71-60.54 29.23-86.82C87.5 32.29 109.43 16 128 16c13.25 0 28.23 8.32 42.34 23a4 4 0 0 1 .09 5.44L122 98.67a8 8 0 0 0 4 13.09l24.61 6.15-6.51 32.52a8 8 0 0 0 6.28 9.41 7.7 7.7 0 0 0 1.62.16 8 8 0 0 0 7.83-6.43l8-40a8 8 0 0 0-5.9-9.33l-19.16-4.79 36.89-41.33a4 4 0 0 1 6.29.41c.24.34.47.68.7 1C205.3 87.54 216 121.23 216 152Z"
  })]
});
var _default = exports.default = EggCrack;
