// phosphor-react-native/lib/commonjs/fill/FileIni.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const FileIni = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "file-ini-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-file-ini-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M48 152v55.73a8.17 8.17 0 0 1-7.47 8.27 8 8 0 0 1-8.53-8v-55.73a8.17 8.17 0 0 1 7.47-8.27 8 8 0 0 1 8.53 8Zm71.47-8a8.17 8.17 0 0 0-7.47 8.25V183l-25.31-35.4a8.26 8.26 0 0 0-8-3.48A8 8 0 0 0 72 152v55.73a8.17 8.17 0 0 0 7.47 8.27 8 8 0 0 0 8.53-8v-31l25.49 35.69a8 8 0 0 0 10.38 2.31 8.23 8.23 0 0 0 4.13-7.25V152a8 8 0 0 0-8.53-8Zm40 0a8.17 8.17 0 0 0-7.47 8.25v55.46a8.17 8.17 0 0 0 7.47 8.25 8 8 0 0 0 8.53-8V152a8 8 0 0 0-8.53-8ZM216 88v135.75a8.15 8.15 0 0 1-6.81 8.16A8 8 0 0 1 200 224V124a4 4 0 0 0-4-4H44a4 4 0 0 1-4-4V40a16 16 0 0 1 16-16h96a8 8 0 0 1 5.66 2.34l56 56A8 8 0 0 1 216 88Zm-20 0-44-44v44Z"
  })]
});
var _default = exports.default = FileIni;
