// phosphor-react-native/lib/commonjs/fill/FileTxt.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const FileTxt = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "file-txt-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-file-txt-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M154.31 156.92 137.83 180l16.53 23.14a8.18 8.18 0 0 1-1.22 11 8 8 0 0 1-11.65-1.48L128 193.76l-13.49 18.89a8 8 0 0 1-11.64 1.49 8.17 8.17 0 0 1-1.23-11L118.17 180l-16.48-23.08a8.22 8.22 0 0 1 1.46-11.28 8 8 0 0 1 11.36 1.71L128 166.24l13.49-18.89a8 8 0 0 1 11.36-1.71 8.22 8.22 0 0 1 1.46 11.28ZM84 144H44.27a8.18 8.18 0 0 0-8.27 7.47 8 8 0 0 0 8 8.53h12v47.73a8.17 8.17 0 0 0 7.47 8.27 8 8 0 0 0 8.53-8v-48h11.73a8.18 8.18 0 0 0 8.27-7.47 8 8 0 0 0-8-8.53Zm128 0h-39.73a8.18 8.18 0 0 0-8.25 7.47 8 8 0 0 0 8 8.53h12v47.73a8.17 8.17 0 0 0 7.47 8.25 8 8 0 0 0 8.53-8V160h11.73a8.18 8.18 0 0 0 8.25-7.47 8 8 0 0 0-8-8.53ZM40 116V40a16 16 0 0 1 16-16h96a8 8 0 0 1 5.66 2.34l56 56A8 8 0 0 1 216 88v28a4 4 0 0 1-4 4H44a4 4 0 0 1-4-4Zm112-28h44l-44-44Z"
  })]
});
var _default = exports.default = FileTxt;
