// phosphor-react-native/lib/commonjs/fill/HeadCircuit.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const HeadCircuit = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "head-circuit-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-head-circuit-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M120 72a8 8 0 1 1 8 8 8 8 0 0 1-8-8Zm24 64a8 8 0 1 0 8-8 8 8 0 0 0-8 8Zm48.5 35.47a88.32 88.32 0 0 0 31.5-69.58q0-1.1-.09-2.19a4 4 0 0 0-4-3.75h-24.16l-23.13 27.75a24 24 0 1 1-12.28-10.25l25.51-30.62A8 8 0 0 1 192 80h23.14a4 4 0 0 0 3.77-5.35C207.27 42 176.86 18 140.74 16.08h-.59a4 4 0 0 0-4.15 4v29.25a24 24 0 1 1-16 0v-27a4 4 0 0 0-4.89-3.91A88.16 88.16 0 0 0 48 102l-22.45 43.14-.22.45a16 16 0 0 0 7.51 20.7l.25.12L56 176.9v31a16 16 0 0 0 16 16h40v8a8 8 0 0 0 8 8h71.77a8.42 8.42 0 0 0 4.06-1 8 8 0 0 0 4.11-8Z"
  })]
});
var _default = exports.default = HeadCircuit;
