// phosphor-react-native/lib/commonjs/fill/Jeep.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Jeep = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "jeep-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-jeep-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M248 103.47a8.17 8.17 0 0 0-8.27-7.47H232a8 8 0 0 0-.18-1.68l-10.64-49.67A16.08 16.08 0 0 0 205.53 32H50.47a16.08 16.08 0 0 0-15.65 12.65L24.18 94.32A8 8 0 0 0 24 96h-7.73A8.17 8.17 0 0 0 8 103.47a8 8 0 0 0 8 8.53h8v88a16 16 0 0 0 16 16h24a16 16 0 0 0 16-16v-16h20a4 4 0 0 0 4-4v-51.73a8.17 8.17 0 0 1 7.47-8.25 8 8 0 0 1 8.53 8v52a4 4 0 0 0 4 4h8a4 4 0 0 0 4-4v-51.75a8.17 8.17 0 0 1 7.47-8.25 8 8 0 0 1 8.53 8v52a4 4 0 0 0 4 4h20v16a16 16 0 0 0 16 16h24a16 16 0 0 0 16-16V112h8a8 8 0 0 0 8-8.53ZM68 144a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm120 0a12 12 0 1 1 12-12 12 12 0 0 1-12 12ZM40.18 96l10.29-48h155.06l10.29 48Z"
  })]
});
var _default = exports.default = Jeep;
