// phosphor-react-native/lib/commonjs/fill/LegoSmiley.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const LegoSmiley = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "lego-smiley-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-lego-smiley-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M184 48h-16V32a16 16 0 0 0-16-16h-48a16 16 0 0 0-16 16v16H72a32 32 0 0 0-32 32v96a32.06 32.06 0 0 0 24 31v17a16 16 0 0 0 16 16h96a16 16 0 0 0 16-16v-17a32.06 32.06 0 0 0 24-31V80a32 32 0 0 0-32-32Zm-28 52a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm4.27 58.77a61 61 0 0 1-64.54 0 8 8 0 0 1 8.54-13.54 45 45 0 0 0 47.46 0 8 8 0 0 1 8.54 13.54ZM104 32h48v16h-48Zm-4 68a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm76 124H80v-16h96Z"
  })]
});
var _default = exports.default = LegoSmiley;
