// phosphor-react-native/lib/commonjs/fill/NuclearPlant.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const NuclearPlant = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "nuclear-plant-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-nuclear-plant-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M152 32h24a8 8 0 0 0 8-8 8 8 0 0 1 16 0 24 24 0 0 1-24 24h-24a8 8 0 0 0-8 8 8 8 0 0 1-16 0 24 24 0 0 1 24-24Zm-48 32a8 8 0 0 0 8-8 40 40 0 0 1 40-40h8a8 8 0 0 0 0-16h-8a56.06 56.06 0 0 0-56 56 8 8 0 0 0 8 8Zm144 152a8 8 0 0 1-8 8H16a8 8 0 0 1 0-16h16.74c13.77-27.83 29.48-68.69 31.12-112.66A15.91 15.91 0 0 1 79.85 80h88.33a16 16 0 0 1 16 15.28c2.1 47.84 23.84 92.37 35.29 112.72H240a8 8 0 0 1 8 8ZM168.18 96h-16c1.77 43.72 17.39 84.32 31.09 112h18c-12.59-23.92-31.09-66.36-33.09-112Z"
  })]
});
var _default = exports.default = NuclearPlant;
