// phosphor-react-native/lib/commonjs/fill/PencilRuler.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PencilRuler = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "pencil-ruler-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-pencil-ruler-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M224 48v160a16 16 0 0 1-16 16h-48a16 16 0 0 1-16-16v-28a4 4 0 0 1 4-4h36a8 8 0 0 0 8-8.53 8.18 8.18 0 0 0-8.25-7.47H148a4 4 0 0 1-4-4v-16a4 4 0 0 1 4-4h36a8 8 0 0 0 8-8.53 8.18 8.18 0 0 0-8.25-7.47H148a4 4 0 0 1-4-4v-16a4 4 0 0 1 4-4h36a8 8 0 0 0 8-8.53 8.18 8.18 0 0 0-8.27-7.47H148a4 4 0 0 1-4-4V48a16 16 0 0 1 16-16h48a16 16 0 0 1 16 16ZM109.66 58.34A8 8 0 0 1 112 64v144a16 16 0 0 1-16 16H48a16 16 0 0 1-16-16V64a8 8 0 0 1 2.34-5.66l32-32a8 8 0 0 1 11.32 0ZM48 80v104h16V80Zm32 0v104h16V80ZM51.31 64h41.38L72 43.31Z"
  })]
});
var _default = exports.default = PencilRuler;
