// phosphor-react-native/lib/commonjs/fill/PersonSimpleThrow.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PersonSimpleThrow = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "person-simple-throw-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-person-simple-throw-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M96 56a32 32 0 1 1 32 32 32 32 0 0 1-32-32Zm125 43.36c-1.5-1.2-37.22-29-89.51 6.57-45.49 30.91-71.92 20.3-72.17 20.19a8 8 0 1 0-6.63 14.56c.61.28 7.49 3.27 19.67 3.27 14.21 0 35.64-4.11 62.77-21.29-2.28 29.41-12.73 83.47-73.43 101.68a8 8 0 1 0 4.6 15.32c34.83-10.45 59.45-32.34 73.2-65.08a141.86 141.86 0 0 0 5.1-14.33l22.08 18.4-14.27 42.82a8 8 0 0 0 15.18 5.06l16-48a8 8 0 0 0-2.47-8.68l-32.42-27a215.91 215.91 0 0 0 3-30.34c36.18-18.57 59-.85 59.28-.65a8 8 0 1 0 10-12.48ZM64 112a16 16 0 1 0-16-16 16 16 0 0 0 16 16Z"
  })]
});
var _default = exports.default = PersonSimpleThrow;
