// phosphor-react-native/lib/commonjs/fill/PinterestLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PinterestLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "pinterest-logo-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-pinterest-logo-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M240 128.7c-.38 56.49-46.46 102.73-102.94 103.29a104.16 104.16 0 0 1-25.94-3 4 4 0 0 1-2.91-4.86l8.64-34.55A60.57 60.57 0 0 0 144 196c37 0 66.7-33.45 63.81-73.36A72 72 0 1 0 69.24 155 8 8 0 0 0 80 159.29a8.19 8.19 0 0 0 4-10.49 56 56 0 1 1 107.86-24.93C194 154.4 171.73 180 144 180a44.87 44.87 0 0 1-23.14-6.44l14.9-59.62a8 8 0 0 0-15.52-3.88L93.38 217.51a4 4 0 0 1-5.71 2.59A104 104 0 0 1 32 126.88C32.6 70.52 78.67 24.52 135 24a104 104 0 0 1 105 104.7Z"
  })]
});
var _default = exports.default = PinterestLogo;
