// phosphor-react-native/lib/commonjs/fill/PixLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PixLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "pix-logo-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-pix-logo-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "m235.34 139.28-19.56 19.55A4 4 0 0 1 213 160h-41.68l-32-32 32-32H213a4 4 0 0 1 2.82 1.17l19.56 19.55a16 16 0 0 1-.04 22.56ZM67.05 80H88a8 8 0 0 1 5.65 2.34L128 116.68l34.35-34.34A8 8 0 0 1 168 80h21a4 4 0 0 0 2.83-6.83l-52.5-52.51a16 16 0 0 0-22.56 0L64.22 73.17A4 4 0 0 0 67.05 80ZM189 176h-21a8 8 0 0 1-5.65-2.34L128 139.31l-34.35 34.35A8 8 0 0 1 88 176H67a4 4 0 0 0-2.83 6.83l52.5 52.51a16 16 0 0 0 22.56 0l52.5-52.51A4 4 0 0 0 189 176Zm-72.26-48-32-32H43a4 4 0 0 0-2.82 1.17l-19.52 19.55a16 16 0 0 0 0 22.56l19.56 19.55A4 4 0 0 0 43 160h41.68Z"
  })]
});
var _default = exports.default = PixLogo;
