// phosphor-react-native/lib/commonjs/fill/Planet.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Planet = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "planet-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-planet-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M245.11 60.68c-7.65-13.19-27.85-16.16-58.5-8.66a96 96 0 0 0-153.8 88.28C5.09 169 5.49 186 10.9 195.32 16 204.16 26.64 208 40.64 208a124.11 124.11 0 0 0 28.79-4 96 96 0 0 0 153.78-88.25c12.51-13 20.83-25.35 23.66-35.92 1.96-7.32 1.37-13.76-1.76-19.15Zm-13.69 15c-6.11 22.78-48.65 57.31-87.52 79.64-67.81 39-113.62 41.52-119.16 32-1.46-2.51-.65-7.24 2.22-13a80.06 80.06 0 0 1 10.28-15.05 95.53 95.53 0 0 0 6.23 14.18 4 4 0 0 0 4 2.12 122.14 122.14 0 0 0 16.95-3.32c21.23-5.55 46.63-16.48 71.52-30.78s47-30.66 62.45-46.15a122.74 122.74 0 0 0 11.31-12.87 4 4 0 0 0 .17-4.52 96.26 96.26 0 0 0-9.1-12.46c14.21-2.35 27.37-2.17 30.5 3.24.92 1.57.97 3.92.15 6.98Z"
  })]
});
var _default = exports.default = Planet;
