// phosphor-react-native/lib/commonjs/fill/RainbowCloud.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const RainbowCloud = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "rainbow-cloud-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-rainbow-cloud-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M248 160a48.05 48.05 0 0 1-48 48h-48c-17.65 0-32-14.75-32-32.89s14.35-32.89 32-32.89a31 31 0 0 1 3.34.18A48 48 0 0 1 248 160ZM112 72a87.57 87.57 0 0 1 61.35 24.91 8 8 0 0 0 11.15-11.47A104 104 0 0 0 8 160v16a8 8 0 0 0 16 0v-16a88.1 88.1 0 0 1 88-88Zm0 32a55.58 55.58 0 0 1 33.13 10.84A8 8 0 1 0 154.6 102 72 72 0 0 0 40 160v16a8 8 0 0 0 16 0v-16a56.06 56.06 0 0 1 56-56Zm15.21 26.71a8 8 0 0 0-5.94-9.63A40 40 0 0 0 72 160v16a8 8 0 0 0 16 0v-16a24 24 0 0 1 29.57-23.35 8 8 0 0 0 9.64-5.94Z"
  })]
});
var _default = exports.default = RainbowCloud;
