// phosphor-react-native/lib/commonjs/fill/Screencast.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Screencast = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "screencast-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-screencast-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M56 208a8 8 0 0 1-8.24 8 8.28 8.28 0 0 1-7.76-8.24 8 8 0 0 0-7.76-7.76 8.28 8.28 0 0 1-8.24-7.76 8 8 0 0 1 8-8.24 24 24 0 0 1 24 24Zm-24-56a8 8 0 0 0-8 8.65 8.17 8.17 0 0 0 8.24 7.35A40 40 0 0 1 72 207.76a8.17 8.17 0 0 0 7.36 8.24 8 8 0 0 0 8.64-8 56.06 56.06 0 0 0-56-56Zm0-32a8 8 0 0 0-8 8.6 8.22 8.22 0 0 0 8.3 7.4 72.08 72.08 0 0 1 71.7 71.68 8.22 8.22 0 0 0 7.4 8.3 8 8 0 0 0 8.6-8A88.1 88.1 0 0 0 32 120Zm184-80H40a16 16 0 0 0-16 16v44.08a4 4 0 0 0 4.15 4 104.11 104.11 0 0 1 107.78 107.77 4 4 0 0 0 4 4.15H216a16 16 0 0 0 16-16V56a16 16 0 0 0-16-16Z"
  })]
});
var _default = exports.default = Screencast;
