// phosphor-react-native/lib/commonjs/fill/Screwdriver.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Screwdriver = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "screwdriver-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-screwdriver-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M233.23 22.75a50.46 50.46 0 0 0-71.31 0L108.68 76A15.92 15.92 0 0 0 104 87.3V104H87.17a16.14 16.14 0 0 0-9.66 3.24 8 8 0 0 0-.82.72l-8 8a16 16 0 0 0 0 22.63l18.7 18.71-77 77.05a8 8 0 0 0 11.32 11.32l77-77.06 18.71 18.71a16 16 0 0 0 22.62 0l8-8a8.08 8.08 0 0 0 .72-.83 16 16 0 0 0 3.25-9.66V152h16.69a15.86 15.86 0 0 0 11.3-4.7l53.23-53.23a50.43 50.43 0 0 0 0-71.32Zm-27.57 38.89-56 56a8 8 0 0 1-11.32-11.31l56-56a8 8 0 0 1 11.32 11.32Z"
  })]
});
var _default = exports.default = Screwdriver;
