// phosphor-react-native/lib/commonjs/fill/ScribbleLoop.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const ScribbleLoop = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "scribble-loop-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-scribble-loop-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 128a81.4 81.4 0 0 1 25.69 4.28C151.56 154.87 137.33 176 112 176c-15.8 0-24.06-10.85-24.06-21.58 0-6.59 3-12.75 8.56-17.35 7.12-5.93 18.02-9.07 31.5-9.07Zm96-80v160a16 16 0 0 1-16 16H48a16 16 0 0 1-16-16V48a16 16 0 0 1 16-16h160a16 16 0 0 1 16 16Zm-18.11 98.59a120.21 120.21 0 0 0-36.08-25.21c-.9-14.35-5.75-27.54-13.89-37.55C145.38 70.86 130.19 64 112 64c-35.56 0-61.32 33.76-62.4 35.2a8 8 0 0 0 12.79 9.62C62.61 108.53 84.51 80 112 80c13.4 0 24 4.68 31.5 13.92a47.54 47.54 0 0 1 9.48 21.4A96.75 96.75 0 0 0 128 112c-17.27 0-31.71 4.42-41.74 12.78C77 132.47 71.94 143 71.94 154.42 71.94 172.64 86 192 112 192a54 54 0 0 0 43.53-21.23A70 70 0 0 0 169 138.89a106.24 106.24 0 0 1 25.13 18.52 8 8 0 1 0 11.78-10.82Z"
  })]
});
var _default = exports.default = ScribbleLoop;
