// phosphor-react-native/lib/commonjs/fill/SpotifyLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SpotifyLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "spotify-logo-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-spotify-logo-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 24a104 104 0 1 0 104 104A104.11 104.11 0 0 0 128 24Zm31.07 145.74a8 8 0 0 1-10.81 3.33 42.79 42.79 0 0 0-40.52 0 8 8 0 0 1-7.48-14.14 59.33 59.33 0 0 1 55.48 0 8 8 0 0 1 3.33 10.81Zm16-28a8 8 0 0 1-10.82 3.3 77.07 77.07 0 0 0-72.48 0 8 8 0 0 1-7.52-14.12 93 93 0 0 1 87.52 0 8 8 0 0 1 3.29 10.84Zm16-28a8 8 0 0 1-10.83 3.29 110.62 110.62 0 0 0-104.46 0 8 8 0 0 1-7.54-14.12 126.67 126.67 0 0 1 119.54 0 8 8 0 0 1 3.28 10.85Z"
  })]
});
var _default = exports.default = SpotifyLogo;
