// phosphor-react-native/lib/commonjs/fill/Strategy.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Strategy = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "strategy-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-strategy-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M108 188a32 32 0 1 1-32-32 32 32 0 0 1 32 32Zm-54.34-70.34L68 103.31l14.34 14.35a8 8 0 0 0 11.32-11.32L79.31 92l14.35-14.34a8 8 0 0 0-11.32-11.32L68 80.69 53.66 66.34a8 8 0 0 0-11.32 11.32L56.69 92l-14.35 14.34a8 8 0 0 0 11.32 11.32ZM215.31 188l14.35-14.34a8 8 0 0 0-11.32-11.32L204 176.69l-14.34-14.35a8 8 0 0 0-11.32 11.32L192.69 188l-14.35 14.34a8 8 0 0 0 11.32 11.32L204 199.31l14.34 14.35a8 8 0 0 0 11.32-11.32ZM165.66 50.34 163.31 48H184a8 8 0 0 0 0-16h-40a8 8 0 0 0-8 8v40a8 8 0 0 0 16 0V59.31l2.34 2.35c17.93 17.93 17.9 35.4 14.71 46.9-4.64 16.77-19.36 31.77-35 35.68A8 8 0 0 0 136 160a8.13 8.13 0 0 0 1.95-.24c21.21-5.3 40.35-24.6 46.53-46.93 6.1-22.05-.58-44.24-18.82-62.49Z"
  })]
});
var _default = exports.default = Strategy;
