// phosphor-react-native/lib/commonjs/fill/SubtitlesSlash.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SubtitlesSlash = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "subtitles-slash-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-subtitles-slash-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M53.92 34.62a8 8 0 0 0-11.48-.37 8.23 8.23 0 0 0-.14 11.38L44.46 48H32a16 16 0 0 0-16 16v128a16 16 0 0 0 16 16h157.92l12.16 13.38a8 8 0 0 0 11.33.51 8.31 8.31 0 0 0 .3-11.51ZM104 128h13.19l14.54 16h-27.46a8.18 8.18 0 0 1-8.27-7.47 8 8 0 0 1 8-8.53Zm-48 0h16a8 8 0 0 1 8 8.53 8.18 8.18 0 0 1-8.27 7.47H56.27a8.18 8.18 0 0 1-8.27-7.47 8 8 0 0 1 8-8.53Zm96 48H56.27a8.18 8.18 0 0 1-8.27-7.47 8 8 0 0 1 8-8.53h90.28l11.9 13.09A8 8 0 0 1 152 176Zm88-112v128a16 16 0 0 1-5.19 11.78 4 4 0 0 1-5.7-.24L175 144h25a8 8 0 0 0 8-8.53 8.17 8.17 0 0 0-8.25-7.47h-39.3L93.79 54.69a4 4 0 0 1 3-6.69H224a16 16 0 0 1 16 16Z"
  })]
});
var _default = exports.default = SubtitlesSlash;
