// phosphor-react-native/lib/commonjs/fill/Subway.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Subway = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "subway-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-subway-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M156 176v-24h20v16a8 8 0 0 1-8 8Zm-16 0v-24h-24v24Zm36-88a8 8 0 0 0-8-8H88a8 8 0 0 0-8 8v48h96Zm-24-64h-48a72 72 0 0 0-72 72v112a8 8 0 0 0 8 8h36.58a4 4 0 0 0 3.58-2.21L91.06 192H88a24 24 0 0 1-24-24V88a24 24 0 0 1 24-24h80a24 24 0 0 1 24 24v80a24 24 0 0 1-24 24h-3.06l10.9 21.79a4 4 0 0 0 3.58 2.21H216a8 8 0 0 0 8-8V96a72 72 0 0 0-72-72Zm-4.94 168h-38.12l-9.1 18.21a4 4 0 0 0 3.58 5.79h49.16a4 4 0 0 0 3.58-5.79ZM80 168a8 8 0 0 0 8 8h12v-24H80Z"
  })]
});
var _default = exports.default = Subway;
