// phosphor-react-native/lib/commonjs/fill/SwimmingPool.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SwimmingPool = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "swimming-pool-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-swimming-pool-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M24 168a8 8 0 0 1 8-8c14.42 0 22.19 5.18 28.44 9.34C66 173.06 70.42 176 80 176s14-2.94 19.56-6.66c6.24-4.16 14-9.34 28.43-9.34s22.2 5.18 28.44 9.34c5.58 3.72 10 6.66 19.57 6.66s14-2.94 19.56-6.66c6.25-4.16 14-9.34 28.44-9.34a8 8 0 0 1 0 16c-9.58 0-14 2.94-19.56 6.66-6.25 4.16-14 9.34-28.44 9.34s-22.2-5.18-28.44-9.34C142 178.94 137.57 176 128 176s-14 2.94-19.56 6.66c-6.24 4.16-14 9.34-28.43 9.34s-22.19-5.18-28.44-9.34C46 178.94 41.58 176 32 176a8 8 0 0 1-8-8Zm200 32c-14.42 0-22.19 5.18-28.44 9.34C190 213.06 185.58 216 176 216s-14-2.94-19.57-6.66c-6.24-4.16-14-9.34-28.44-9.34s-22.19 5.18-28.43 9.34C94 213.06 89.57 216 80 216s-14-2.94-19.56-6.66C54.19 205.18 46.42 200 32 200a8 8 0 0 0 0 16c9.58 0 14 2.94 19.56 6.66 6.25 4.16 14 9.34 28.44 9.34s22.19-5.18 28.43-9.34c5.58-3.72 10-6.66 19.56-6.66s14 2.94 19.57 6.66c6.24 4.16 14 9.34 28.44 9.34s22.19-5.18 28.44-9.34c5.57-3.72 10-6.66 19.56-6.66a8 8 0 0 0 0-16ZM80 141.39V32a8 8 0 0 1 16 0v8h64v-8a8 8 0 0 1 16 0v111.29a8 8 0 0 1-16 0V136H96v5.39a8 8 0 0 1-16 0ZM96 72a8 8 0 0 0 8 8h48a8 8 0 0 0 0-16h-48a8 8 0 0 0-8 8Zm0 32a8 8 0 0 0 8 8h48a8 8 0 0 0 0-16h-48a8 8 0 0 0-8 8Z"
  })]
});
var _default = exports.default = SwimmingPool;
