// phosphor-react-native/lib/commonjs/fill/Tire.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Tire = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "tire-fill__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-tire-fill',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M184 128c0 22.09-7.16 40-16 40s-16-17.91-16-40 7.16-40 16-40 16 17.91 16 40Zm56 96a8 8 0 0 1-8 8H92c-33.64 0-60-45.68-60-104S58.36 24 92 24h72c33.64 0 60 45.68 60 104 0 37.47-10.88 69.73-27.59 88H232a8 8 0 0 1 8 8ZM57.87 111.81a7.93 7.93 0 0 0 4.64-1.49L80 97.83l13.28 9.49a8 8 0 0 0 9.3-13L84.65 81.49a8 8 0 0 0-9.3 0L53.21 97.3a8 8 0 0 0 4.66 14.51Zm46.67 47.89-19.89-14.21a8 8 0 0 0-9.3 0L56 159.29a8 8 0 1 0 9.3 13L80 161.83l15.24 10.88a8 8 0 1 0 9.3-13Zm89.2 32.37c9.19-17 14.26-39.74 14.26-64.07s-5.07-47.09-14.26-64.07C185.38 48.5 174.82 40 164 40s-21.38 8.5-29.74 23.93C125.07 80.91 120 103.67 120 128s5.07 47.09 14.26 64.07C142.62 207.5 153.18 216 164 216s21.38-8.5 29.74-23.93Z"
  })]
});
var _default = exports.default = Tire;
