// phosphor-react-native/lib/commonjs/regular/Gradient.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Gradient = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "gradient__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-gradient',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M24 104a8 8 0 0 1 8-8h80a8 8 0 0 1 0 16H32a8 8 0 0 1-8-8Zm200-8h-80a8 8 0 0 0 0 16h80a8 8 0 0 0 0-16ZM72 136H32a8 8 0 0 0 0 16h40a8 8 0 0 0 0-16Zm152 0h-40a8 8 0 0 0 0 16h40a8 8 0 0 0 0-16Zm-128 8a8 8 0 0 0 8 8h48a8 8 0 0 0 0-16h-48a8 8 0 0 0-8 8Zm-40 32H32a8 8 0 0 0 0 16h24a8 8 0 0 0 0-16Zm56 0H88a8 8 0 0 0 0 16h24a8 8 0 0 0 0-16Zm56 0h-24a8 8 0 0 0 0 16h24a8 8 0 0 0 0-16Zm56 0h-24a8 8 0 0 0 0 16h24a8 8 0 0 0 0-16ZM32 72h192a8 8 0 0 0 0-16H32a8 8 0 0 0 0 16Z"
  })]
});
var _default = exports.default = Gradient;
