// phosphor-react-native/lib/commonjs/regular/LegoSmiley.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const LegoSmiley = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "lego-smiley__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-lego-smiley',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M100 124a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm56-24a12 12 0 1 0 12 12 12 12 0 0 0-12-12Zm-4.27 45.23a45 45 0 0 1-47.46 0 8 8 0 0 0-8.54 13.54 61 61 0 0 0 64.54 0 8 8 0 0 0-8.54-13.54ZM216 80v96a32.06 32.06 0 0 1-24 31v17a16 16 0 0 1-16 16H80a16 16 0 0 1-16-16v-17a32.06 32.06 0 0 1-24-31V80a32 32 0 0 1 32-32h16V32a16 16 0 0 1 16-16h48a16 16 0 0 1 16 16v16h16a32 32 0 0 1 32 32ZM104 48h48V32h-48Zm72 176v-16H80v16Zm24-144a16 16 0 0 0-16-16H72a16 16 0 0 0-16 16v96a16 16 0 0 0 16 16h112a16 16 0 0 0 16-16Z"
  })]
});
var _default = exports.default = LegoSmiley;
