// phosphor-react-native/lib/commonjs/regular/PersonSimpleSki.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PersonSimpleSki = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "person-simple-ski__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-person-simple-ski',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M172 88a28 28 0 1 0-28-28 28 28 0 0 0 28 28Zm0-40a12 12 0 1 1-12 12 12 12 0 0 1 12-12ZM32.32 77.76a8 8 0 0 1 9.92-5.44L112 92.67l10.32-10.33a8 8 0 0 1 11.32 0L163.31 112H200a8 8 0 0 1 0 16h-40a8 8 0 0 1-5.66-2.34l-5.61-5.66-111-32.36a8 8 0 0 1-5.41-9.88Zm204.12 126.48a69.78 69.78 0 0 1-58.37 8.95l-156.3-45.52a8 8 0 1 1 4.47-15.36l76 22.13L128.73 148 85.8 135.69a8 8 0 0 1 4.4-15.39l56 16a8 8 0 0 1 3.46 13.35l-29.9 29.9 62.79 18.28a53.78 53.78 0 0 0 45-6.9 8 8 0 1 1 8.88 13.31Z"
  })]
});
var _default = exports.default = PersonSimpleSki;
