// phosphor-react-native/lib/commonjs/regular/Racquet.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Racquet = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "racquet__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-racquet',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M230 26.05C202-1.88 151.53 3.16 117.4 37.3c-31.79 31.79-38.33 77.77-16.51 106.49l-29.56 29.56-.68-.68a16 16 0 0 0-22.64 0L20.69 200a16 16 0 0 0 0 22.64l12.69 12.69a16 16 0 0 0 22.63 0L83.34 208a16 16 0 0 0 0-22.63l-.69-.69 29.56-29.56c11.29 8.58 25.24 12.79 40 12.79 22.72 0 47.25-10 66.54-29.3C252.83 104.47 257.88 54 230 26.05ZM44.7 224 32 211.31 59.33 184l6.33 6.34 6.34 6.33Zm179.53-120h-24.17V72h32a72.45 72.45 0 0 1-7.83 32ZM136 149.61A44.15 44.15 0 0 1 106.39 120H136ZM104 104a72.24 72.24 0 0 1 7.86-32H136v32Zm48-32h32v32h-32Zm77.67-16h-29.61V26.28a44.23 44.23 0 0 1 29.66 29.66Zm-45.82-32h.16v32h-32V31.76a72.47 72.47 0 0 1 31.89-7.86ZM136 42.06v13.88h-13.84a89.72 89.72 0 0 1 6.56-7.32 93.17 93.17 0 0 1 7.28-6.56Zm16 109.92V120h32v24.16a72.24 72.24 0 0 1-31.95 7.84Zm48-18.14V120h14a91.62 91.62 0 0 1-6.56 7.32 89.64 89.64 0 0 1-7.38 6.52Z"
  })]
});
var _default = exports.default = Racquet;
