// phosphor-react-native/lib/commonjs/regular/Rss.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Rss = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "rss__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-rss',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M106.91 149.09A71.53 71.53 0 0 1 128 200a8 8 0 0 1-16 0 56 56 0 0 0-56-56 8 8 0 0 1 0-16 71.53 71.53 0 0 1 50.91 21.09ZM56 80a8 8 0 0 0 0 16 104 104 0 0 1 104 104 8 8 0 0 0 16 0A120 120 0 0 0 56 80Zm118.79 1.21A166.9 166.9 0 0 0 56 32a8 8 0 0 0 0 16 151 151 0 0 1 107.48 44.52A151 151 0 0 1 208 200a8 8 0 0 0 16 0 166.9 166.9 0 0 0-49.21-118.79ZM60 184a12 12 0 1 0 12 12 12 12 0 0 0-12-12Z"
  })]
});
var _default = exports.default = Rss;
