// phosphor-react-native/lib/commonjs/regular/SkypeLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SkypeLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "skype-logo__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-skype-logo',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M168 152c0 17.64-17.94 32-40 32s-40-14.36-40-32a8 8 0 0 1 16 0c0 8.67 11 16 24 16s24-7.33 24-16c0-9.48-8.61-13-26.88-18.26-15.75-4.54-35.34-10.19-35.34-29.74 0-18.24 16.43-32 38.22-32 15.72 0 29.18 7.3 35.12 19a8 8 0 1 1-14.27 7.22C145.64 91.94 137.65 88 128 88c-12.67 0-22.22 6.88-22.22 16 0 7 9 10.1 23.77 14.36C145.78 123 168 129.45 168 152Zm64 24a56.06 56.06 0 0 1-56 56 55.38 55.38 0 0 1-31.17-9.47A96.09 96.09 0 0 1 33.47 111.17 55.38 55.38 0 0 1 24 80a56.06 56.06 0 0 1 56-56 55.38 55.38 0 0 1 31.17 9.47 96.09 96.09 0 0 1 111.36 111.36A55.38 55.38 0 0 1 232 176Zm-16 0a39.6 39.6 0 0 0-8.32-24.42 8 8 0 0 1-1.49-6.58A80.07 80.07 0 0 0 111 49.81a8 8 0 0 1-6.58-1.49A39.6 39.6 0 0 0 80 40a40 40 0 0 0-40 40 39.6 39.6 0 0 0 8.32 24.42 8 8 0 0 1 1.49 6.58A80.07 80.07 0 0 0 145 206.19a8 8 0 0 1 6.58 1.49A39.6 39.6 0 0 0 176 216a40 40 0 0 0 40-40Z"
  })]
});
var _default = exports.default = SkypeLogo;
