// phosphor-react-native/lib/commonjs/regular/Spiral.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Spiral = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "spiral__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-spiral',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M248 144a8 8 0 0 1-16 0 96.11 96.11 0 0 0-96-96 88.1 88.1 0 0 0-88 88 80.09 80.09 0 0 0 80 80 72.08 72.08 0 0 0 72-72 64.07 64.07 0 0 0-64-64 56.06 56.06 0 0 0-56 56 48.05 48.05 0 0 0 48 48 40 40 0 0 0 40-40 32 32 0 0 0-32-32 24 24 0 0 0-24 24 16 16 0 0 0 16 16 8 8 0 0 0 8-8 8 8 0 0 1 0-16 16 16 0 0 1 16 16 24 24 0 0 1-24 24 32 32 0 0 1-32-32 40 40 0 0 1 40-40 48.05 48.05 0 0 1 48 48 56.06 56.06 0 0 1-56 56 64.07 64.07 0 0 1-64-64 72.08 72.08 0 0 1 72-72 80.09 80.09 0 0 1 80 80 88.1 88.1 0 0 1-88 88 96.11 96.11 0 0 1-96-96A104.11 104.11 0 0 1 136 32a112.12 112.12 0 0 1 112 112Z"
  })]
});
var _default = exports.default = Spiral;
