// phosphor-react-native/lib/commonjs/regular/Strategy.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Strategy = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "strategy__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-strategy',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M76 152a36 36 0 1 0 36 36 36 36 0 0 0-36-36Zm0 56a20 20 0 1 1 20-20 20 20 0 0 1-20 20ZM42.34 106.34 56.69 92 42.34 77.66a8 8 0 0 1 11.32-11.32L68 80.69l14.34-14.35a8 8 0 0 1 11.32 11.32L79.31 92l14.35 14.34a8 8 0 0 1-11.32 11.32L68 103.31l-14.34 14.35a8 8 0 0 1-11.32-11.32Zm187.32 96a8 8 0 0 1-11.32 11.32L204 199.31l-14.34 14.35a8 8 0 0 1-11.32-11.32L192.69 188l-14.35-14.34a8 8 0 0 1 11.32-11.32L204 176.69l14.34-14.35a8 8 0 0 1 11.32 11.32L215.31 188Zm-45.19-89.51c-6.18 22.33-25.32 41.63-46.53 46.93a8.13 8.13 0 0 1-1.94.24 8 8 0 0 1-1.93-15.76c15.63-3.91 30.35-18.91 35-35.68 3.19-11.5 3.22-29-14.71-46.9L152 59.31V80a8 8 0 0 1-16 0V40a8 8 0 0 1 8-8h40a8 8 0 0 1 0 16h-20.69l2.35 2.34c18.24 18.25 24.92 40.44 18.81 62.49Z"
  })]
});
var _default = exports.default = Strategy;
