// phosphor-react-native/lib/module/light/ArticleNyTimes.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const ArticleNyTimes = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "article-ny-times-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-article-ny-times-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M128 98h104a6 6 0 0 1 0 12H128a6 6 0 0 1 0-12Zm104 32H128a6 6 0 0 0 0 12h104a6 6 0 0 0 0-12Zm0 32H80a6 6 0 0 0 0 12h152a6 6 0 0 0 0-12Zm0 32H80a6 6 0 0 0 0 12h152a6 6 0 0 0 0-12ZM96 142a6 6 0 0 0 0-12H86V62h36v10a6 6 0 0 0 12 0V56a6 6 0 0 0-6-6H32a6 6 0 0 0-6 6v16a6 6 0 0 0 12 0V62h36v68H64a6 6 0 0 0 0 12Z"
  })]
});
export default ArticleNyTimes;
