// phosphor-react-native/lib/module/light/BezierCurve.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const BezierCurve = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "bezier-curve-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-bezier-curve-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M219.44 146.2A94.66 94.66 0 0 0 173.92 86H240a6 6 0 0 0 0-12h-82.6a30 30 0 0 0-58.8 0H16a6 6 0 0 0 0 12h66.08a94.66 94.66 0 0 0-45.52 60.2 30 30 0 1 0 12.09 1.08 82.53 82.53 0 0 1 51.4-56.39 30 30 0 0 0 55.9 0 82.53 82.53 0 0 1 51.4 56.39 30 30 0 1 0 12.09-1.08ZM58 176a18 18 0 1 1-18-18 18 18 0 0 1 18 18Zm70-78a18 18 0 1 1 18-18 18 18 0 0 1-18 18Zm88 96a18 18 0 1 1 18-18 18 18 0 0 1-18 18Z"
  })]
});
export default BezierCurve;
