// phosphor-react-native/lib/module/light/Boules.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Boules = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "boules-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-boules-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M128 26a102 102 0 1 0 102 102A102.12 102.12 0 0 0 128 26Zm59.22 34.29L60.29 187.22a90.21 90.21 0 0 1-12.53-18.49l121-121a90.21 90.21 0 0 1 18.46 12.56Zm8.49 8.49a90.21 90.21 0 0 1 12.53 18.49l-121 121a90.21 90.21 0 0 1-18.49-12.53ZM128 38a89.67 89.67 0 0 1 28.79 4.72L42.72 156.79A90 90 0 0 1 128 38Zm0 180a89.67 89.67 0 0 1-28.79-4.72L213.28 99.21A90 90 0 0 1 128 218Z"
  })]
});
export default Boules;
