// phosphor-react-native/lib/module/light/CableCar.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const CableCar = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "cable-car-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-cable-car-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M245.91 31a6 6 0 0 0-7-4.85L15 66.1A6 6 0 0 0 16 78a6.53 6.53 0 0 0 1.07-.09L122 59.17V98H64a30 30 0 0 0-30 30v64a30 30 0 0 0 30 30h128a30 30 0 0 0 30-30v-64a30 30 0 0 0-30-30h-58V57l107.05-19.09a6 6 0 0 0 4.86-6.91ZM102 162v-52h52v52Zm-38-52h26v52H46v-34a18 18 0 0 1 18-18Zm128 100H64a18 18 0 0 1-18-18v-18h164v18a18 18 0 0 1-18 18Zm18-82v34h-44v-52h26a18 18 0 0 1 18 18Z"
  })]
});
export default CableCar;
