// phosphor-react-native/lib/module/light/DotsNine.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const DotsNine = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "dots-nine-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-dots-nine-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M70 60a10 10 0 1 1-10-10 10 10 0 0 1 10 10Zm58-10a10 10 0 1 0 10 10 10 10 0 0 0-10-10Zm68 20a10 10 0 1 0-10-10 10 10 0 0 0 10 10ZM60 118a10 10 0 1 0 10 10 10 10 0 0 0-10-10Zm68 0a10 10 0 1 0 10 10 10 10 0 0 0-10-10Zm68 0a10 10 0 1 0 10 10 10 10 0 0 0-10-10ZM60 186a10 10 0 1 0 10 10 10 10 0 0 0-10-10Zm68 0a10 10 0 1 0 10 10 10 10 0 0 0-10-10Zm68 0a10 10 0 1 0 10 10 10 10 0 0 0-10-10Z"
  })]
});
export default DotsNine;
