// phosphor-react-native/lib/module/light/GitFork.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const GitFork = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "git-fork-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-git-fork-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M222 64a30 30 0 1 0-36 29.4V112a10 10 0 0 1-10 10H80a10 10 0 0 1-10-10V93.4a30 30 0 1 0-12 0V112a22 22 0 0 0 22 22h42v28.6a30 30 0 1 0 12 0V134h42a22 22 0 0 0 22-22V93.4A30.05 30.05 0 0 0 222 64ZM46 64a18 18 0 1 1 18 18 18 18 0 0 1-18-18Zm100 128a18 18 0 1 1-18-18 18 18 0 0 1 18 18Zm46-110a18 18 0 1 1 18-18 18 18 0 0 1-18 18Z"
  })]
});
export default GitFork;
