// phosphor-react-native/lib/module/light/ShareNetwork.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const ShareNetwork = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "share-network-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-share-network-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M176 162a37.91 37.91 0 0 0-28.3 12.67l-48.9-31.43a37.89 37.89 0 0 0 0-30.48l48.9-31.43a38 38 0 1 0-6.5-10.09l-48.9 31.43a38 38 0 1 0 0 50.66l48.9 31.43A38 38 0 1 0 176 162Zm0-132a26 26 0 1 1-26 26 26 26 0 0 1 26-26ZM64 154a26 26 0 1 1 26-26 26 26 0 0 1-26 26Zm112 72a26 26 0 1 1 26-26 26 26 0 0 1-26 26Z"
  })]
});
export default ShareNetwork;
