// phosphor-react-native/lib/module/light/Television.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Television = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "television-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-television-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M216 66h-73.52l37.76-37.76a6 6 0 0 0-8.48-8.48L128 63.51 84.24 19.76a6 6 0 1 0-8.48 8.48L113.52 66H40a14 14 0 0 0-14 14v120a14 14 0 0 0 14 14h176a14 14 0 0 0 14-14V80a14 14 0 0 0-14-14ZM38 200V80a2 2 0 0 1 2-2h106v124H40a2 2 0 0 1-2-2Zm180 0a2 2 0 0 1-2 2h-58V78h58a2 2 0 0 1 2 2Zm-20-84a10 10 0 1 1-10-10 10 10 0 0 1 10 10Zm0 48a10 10 0 1 1-10-10 10 10 0 0 1 10 10Z"
  })]
});
export default Television;
