// phosphor-react-native/lib/module/light/TextColumns.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const TextColumns = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "text-columns-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-text-columns-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M118 64a6 6 0 0 1-6 6H40a6 6 0 0 1 0-12h72a6 6 0 0 1 6 6Zm-6 34H40a6 6 0 0 0 0 12h72a6 6 0 0 0 0-12Zm0 40H40a6 6 0 0 0 0 12h72a6 6 0 0 0 0-12Zm0 40H40a6 6 0 0 0 0 12h72a6 6 0 0 0 0-12Zm32-108h72a6 6 0 0 0 0-12h-72a6 6 0 0 0 0 12Zm72 28h-72a6 6 0 0 0 0 12h72a6 6 0 0 0 0-12Zm0 40h-72a6 6 0 0 0 0 12h72a6 6 0 0 0 0-12Zm0 40h-72a6 6 0 0 0 0 12h72a6 6 0 0 0 0-12Z"
  })]
});
export default TextColumns;
