// phosphor-react-native/lib/module/light/Tornado.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Tornado = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "tornado-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-tornado-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M230 40a6 6 0 0 1-6 6H56a6 6 0 0 1 0-12h168a6 6 0 0 1 6 6Zm-48 32a6 6 0 0 0-6-6H32a6 6 0 0 0 0 12h144a6 6 0 0 0 6-6Zm-16 32a6 6 0 0 0-6-6H56a6 6 0 0 0 0 12h104a6 6 0 0 0 6-6Zm16 32a6 6 0 0 0-6-6H88a6 6 0 0 0 0 12h88a6 6 0 0 0 6-6Zm2 26h-64a6 6 0 0 0 0 12h64a6 6 0 0 0 0-12Zm-24 32h-32a6 6 0 0 0 0 12h32a6 6 0 0 0 0-12Zm-32 32h-16a6 6 0 0 0 0 12h16a6 6 0 0 0 0-12Z"
  })]
});
export default Tornado;
