// phosphor-react-native/lib/commonjs/light/Ambulance.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Ambulance = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "ambulance-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-ambulance-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M82 120a6 6 0 0 1 6-6h18V96a6 6 0 0 1 12 0v18h18a6 6 0 0 1 0 12h-18v18a6 6 0 0 1-12 0v-18H88a6 6 0 0 1-6-6Zm172 0v64a14 14 0 0 1-14 14h-18.6a30 30 0 0 1-58.8 0h-53.2a30 30 0 0 1-58.8 0H32a14 14 0 0 1-14-14V72a14 14 0 0 1 14-14h152a6 6 0 0 1 6 6v10h36.58a13.93 13.93 0 0 1 13 8.8l14 35a6.1 6.1 0 0 1 .42 2.2Zm-64-34v28h49.14l-10.7-26.74a2 2 0 0 0-1.86-1.26ZM32 186h18.6a30 30 0 0 1 58.8 0h53.2a30.1 30.1 0 0 1 15.4-20.52V70H32a2 2 0 0 0-2 2v112a2 2 0 0 0 2 2Zm66 6a18 18 0 1 0-18 18 18 18 0 0 0 18-18Zm112 0a18 18 0 1 0-18 18 18 18 0 0 0 18-18Zm32-8v-58h-52v36.07c.66 0 1.33-.07 2-.07a30.05 30.05 0 0 1 29.4 24H240a2 2 0 0 0 2-2Z"
  })]
});
var _default = exports.default = Ambulance;
