// phosphor-react-native/lib/commonjs/light/Asclepius.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Asclepius = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "asclepius-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-asclepius-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M214 79v1a38 38 0 0 1-38 38h-42v84h10a18 18 0 0 0 12-31.42 6 6 0 1 1 8-8.94A30 30 0 0 1 144 214h-10v18a6 6 0 0 1-12 0v-18H96a6 6 0 0 1 0-12h26v-84H96a18 18 0 0 0 0 36 6 6 0 0 1 0 12 30 30 0 0 1 0-60h26V24a6 6 0 0 1 12 0v82h42a26 26 0 0 0 26-26v-1a25 25 0 0 0-25-25h-17a6 6 0 0 1 0-12h17a37 37 0 0 1 37 37ZM56 94H32a6 6 0 0 1-6-6v-8a38 38 0 0 1 38-38h32a6 6 0 0 1 0 12h-2v2a38 38 0 0 1-38 38Zm26-40H64a26 26 0 0 0-26 26v2h18a26 26 0 0 0 26-26Z"
  })]
});
var _default = exports.default = Asclepius;
