// phosphor-react-native/lib/commonjs/light/Backpack.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Backpack = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "backpack-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-backpack-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M166 42.34V32a22 22 0 0 0-22-22h-32a22 22 0 0 0-22 22v10.34A54.07 54.07 0 0 0 42 96v120a14 14 0 0 0 14 14h144a14 14 0 0 0 14-14V96a54.07 54.07 0 0 0-48-53.66ZM112 22h32a10 10 0 0 1 10 10v10h-52V32a10 10 0 0 1 10-10Zm58 140H86v-10a10 10 0 0 1 10-10h64a10 10 0 0 1 10 10Zm-84 12h52v10a6 6 0 0 0 12 0v-10h20v44H86Zm116 42a2 2 0 0 1-2 2h-18v-66a22 22 0 0 0-22-22H96a22 22 0 0 0-22 22v66H56a2 2 0 0 1-2-2V96a42 42 0 0 1 42-42h64a42 42 0 0 1 42 42ZM150 88a6 6 0 0 1-6 6h-32a6 6 0 0 1 0-12h32a6 6 0 0 1 6 6Z"
  })]
});
var _default = exports.default = Backpack;
