// phosphor-react-native/lib/commonjs/light/Basketball.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Basketball = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "basketball-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-basketball-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 26a102 102 0 1 0 102 102A102.12 102.12 0 0 0 128 26ZM60 69.09A89.23 89.23 0 0 1 81.78 122H38.2A89.65 89.65 0 0 1 60 69.09ZM38.2 134h43.58A89.23 89.23 0 0 1 60 186.91 89.65 89.65 0 0 1 38.2 134Zm83.8 83.8a89.66 89.66 0 0 1-53.5-22.34A101.18 101.18 0 0 0 93.82 134H122Zm0-95.8H93.82A101.18 101.18 0 0 0 68.5 60.54 89.66 89.66 0 0 1 122 38.2Zm95.8 0h-43.58A89.23 89.23 0 0 1 196 69.09 89.65 89.65 0 0 1 217.8 122ZM134 38.2a89.66 89.66 0 0 1 53.5 22.34A101.18 101.18 0 0 0 162.18 122H134Zm0 179.6V134h28.18a101.18 101.18 0 0 0 25.32 61.46A89.66 89.66 0 0 1 134 217.8Zm62-30.89A89.23 89.23 0 0 1 174.22 134h43.58a89.65 89.65 0 0 1-21.8 52.91Z"
  })]
});
var _default = exports.default = Basketball;
