// phosphor-react-native/lib/commonjs/light/BeerBottle.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const BeerBottle = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "beer-bottle-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-beer-bottle-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "m244.24 43.76-32-32a6 6 0 0 0-8.48 8.48l3.11 3.11-57.38 43-38.67 7.74a5.92 5.92 0 0 0-3.06 1.64l-83.32 83.34a22 22 0 0 0 0 31.11l41.38 41.38a22 22 0 0 0 31.11 0l83.31-83.32a5.92 5.92 0 0 0 1.64-3.06l7.74-38.67 43-57.38 3.11 3.11a6 6 0 0 0 8.48-8.48ZM88.44 223.07a10 10 0 0 1-14.14 0L32.93 181.7a10 10 0 0 1 0-14.14l7.07-7.07L95.51 216ZM104 207.51 48.49 152 96 104.49 151.51 160Zm75.2-107.11a5.88 5.88 0 0 0-1.08 2.42L170.47 141 160 151.51 104.49 96 115 85.53l38.22-7.65a5.88 5.88 0 0 0 2.42-1.08l59.84-44.88 8.64 8.64Z"
  })]
});
var _default = exports.default = BeerBottle;
