// phosphor-react-native/lib/commonjs/light/Bomb.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Bomb = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "bomb-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-bomb-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M248 34a6 6 0 0 0-6 6 54.1 54.1 0 0 1-3.57 17.77C234.06 68.69 226.72 74 216 74c-12.91 0-21.76-12.38-31.12-25.49C174.67 34.21 163.09 18 144 18c-15.74 0-27.35 8.22-33.57 23.77a65.36 65.36 0 0 0-4 16.23H88a14 14 0 0 0-14 14v10.83A86 86 0 0 0 112 246h1.55A86 86 0 0 0 150 82.83V72a14 14 0 0 0-14-14h-17.47a51.39 51.39 0 0 1 3.15-12c4.38-10.74 11.69-16 22.32-16 12.91 0 21.75 12.38 31.12 25.49C185.33 69.79 196.91 86 216 86c15.74 0 27.35-8.22 33.57-23.77A66.46 66.46 0 0 0 254 40a6 6 0 0 0-6-6ZM138 72v14.66a6 6 0 0 0 3.6 5.49 74 74 0 1 1-59.2 0 6 6 0 0 0 3.6-5.49V72a2 2 0 0 1 2-2h48a2 2 0 0 1 2 2Zm-33 130.08a6 6 0 0 1-1 11.92 6.74 6.74 0 0 1-1-.08A55.48 55.48 0 0 1 58.08 169a6 6 0 1 1 11.84-2A43.29 43.29 0 0 0 105 202.08Z"
  })]
});
var _default = exports.default = Bomb;
