// phosphor-react-native/lib/commonjs/light/BowlSteam.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const BowlSteam = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "bowl-steam-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-bowl-steam-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M224 114H32a6 6 0 0 0-6 6 102.35 102.35 0 0 0 56 91.06V216a14 14 0 0 0 14 14h64a14 14 0 0 0 14-14v-4.94A102.35 102.35 0 0 0 230 120a6 6 0 0 0-6-6Zm-58.5 87.84a6 6 0 0 0-3.5 5.45V216a2 2 0 0 1-2 2H96a2 2 0 0 1-2-2v-8.71a6 6 0 0 0-3.5-5.45A90.35 90.35 0 0 1 38.2 126h179.6a90.35 90.35 0 0 1-52.3 75.84ZM83.32 56.24c5.05-6.28 7.41-11.73 6.48-15-.72-2.53-3.64-3.57-3.67-3.58A6 6 0 0 1 88 26a5.81 5.81 0 0 1 1.78.27c.91.28 8.9 3 11.48 11.44 2.26 7.39-.63 16.15-8.58 26.05-5.05 6.28-7.41 11.73-6.48 15 .72 2.53 3.64 3.57 3.67 3.58A6 6 0 0 1 88 94a5.81 5.81 0 0 1-1.78-.27c-.91-.28-8.9-3-11.48-11.44-2.26-7.39.63-16.15 8.58-26.05Zm40 0c5.05-6.28 7.41-11.73 6.48-15-.72-2.53-3.64-3.57-3.67-3.58A6 6 0 0 1 128 26a5.81 5.81 0 0 1 1.78.27c.91.28 8.9 3 11.48 11.44 2.26 7.39-.63 16.15-8.58 26.05-5.05 6.28-7.41 11.73-6.48 15 .72 2.53 3.64 3.57 3.67 3.58A6 6 0 0 1 128 94a5.81 5.81 0 0 1-1.78-.27c-.91-.28-8.9-3-11.48-11.44-2.26-7.39.63-16.15 8.58-26.05Zm40 0c5.05-6.28 7.41-11.73 6.48-15-.72-2.53-3.64-3.57-3.67-3.58A6 6 0 0 1 168 26a5.81 5.81 0 0 1 1.78.27c.91.28 8.9 3 11.48 11.44 2.26 7.39-.63 16.15-8.58 26.05-5.05 6.28-7.41 11.73-6.48 15 .72 2.53 3.64 3.57 3.67 3.58A6 6 0 0 1 168 94a5.81 5.81 0 0 1-1.78-.27c-.91-.28-8.9-3-11.48-11.44-2.26-7.39.63-16.15 8.58-26.05Z"
  })]
});
var _default = exports.default = BowlSteam;
