// phosphor-react-native/lib/commonjs/light/CameraSlash.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const CameraSlash = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "camera-slash-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-camera-slash-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M52.44 36a6 6 0 0 0-8.88 8l12.69 14H48a22 22 0 0 0-22 22v112a22 22 0 0 0 22 22h150.07l5.49 6a6 6 0 0 0 8.88-8.08Zm53.06 76.2 40.11 44.13A29.67 29.67 0 0 1 128 162a30 30 0 0 1-22.5-49.84ZM48 202a10 10 0 0 1-10-10V80a10 10 0 0 1 10-10h19.16l30.23 33.25a42 42 0 0 0 56.33 62L187.16 202ZM230 80v106a6 6 0 0 1-12 0V80a10 10 0 0 0-10-10h-32a6 6 0 0 1-5-2.67L156.78 46H99.21l-.23.34a6 6 0 0 1-10-6.65l2-3A6 6 0 0 1 96 34h64a6 6 0 0 1 5 2.67L179.21 58H208a22 22 0 0 1 22 22Z"
  })]
});
var _default = exports.default = CameraSlash;
