// phosphor-react-native/lib/commonjs/light/CashRegister.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const CashRegister = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "cash-register-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-cash-register-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "m237.82 158.54-22.48-89.93A14 14 0 0 0 201.75 58H134V40a14 14 0 0 0-14-14H80a14 14 0 0 0-14 14v18H54.25a14 14 0 0 0-13.59 10.6l-22.48 89.94A6 6 0 0 0 18 160v32a14 14 0 0 0 14 14h192a14 14 0 0 0 14-14v-32a6 6 0 0 0-.18-1.46ZM78 40a2 2 0 0 1 2-2h40a2 2 0 0 1 2 2v18H78ZM52.31 71.51A2 2 0 0 1 54.25 70h147.5a2 2 0 0 1 1.94 1.51L224.32 154H31.68ZM224 194H32a2 2 0 0 1-2-2v-26h196v26a2 2 0 0 1-2 2ZM66 96a6 6 0 0 1 6-6h16a6 6 0 0 1 0 12H72a6 6 0 0 1-6-6Zm48 0a6 6 0 0 1 6-6h16a6 6 0 0 1 0 12h-16a6 6 0 0 1-6-6Zm48 0a6 6 0 0 1 6-6h16a6 6 0 0 1 0 12h-16a6 6 0 0 1-6-6Zm-96 32a6 6 0 0 1 6-6h16a6 6 0 0 1 0 12H72a6 6 0 0 1-6-6Zm48 0a6 6 0 0 1 6-6h16a6 6 0 0 1 0 12h-16a6 6 0 0 1-6-6Zm48 0a6 6 0 0 1 6-6h16a6 6 0 0 1 0 12h-16a6 6 0 0 1-6-6Z"
  })]
});
var _default = exports.default = CashRegister;
