// phosphor-react-native/lib/commonjs/light/CellTower.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const CellTower = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "cell-tower-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-cell-tower-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M133.37 85.32a6 6 0 0 0-10.74 0l-72 144a6 6 0 0 0 10.74 5.37L75.71 206h104.58l14.34 28.68A6 6 0 0 0 200 238a5.87 5.87 0 0 0 2.68-.64 6 6 0 0 0 2.69-8.05Zm-5.37 16.1L158.29 162H97.71ZM81.71 194l10-20h72.58l10 20Zm79.74-99.9A33.59 33.59 0 0 0 162 88a34 34 0 0 0-68 0 33.59 33.59 0 0 0 .55 6.1 6 6 0 1 1-11.81 2.13 46 46 0 1 1 90.52 0 6 6 0 0 1-5.89 4.94 5.64 5.64 0 0 1-1.08-.1 6 6 0 0 1-4.84-6.97ZM65.75 135A78 78 0 1 1 206 88a77.33 77.33 0 0 1-15.75 47 6 6 0 1 1-9.57-7.24A65.42 65.42 0 0 0 194 88a66 66 0 0 0-132 0 65.42 65.42 0 0 0 13.32 39.76 6 6 0 1 1-9.57 7.24Z"
  })]
});
var _default = exports.default = CellTower;
