// phosphor-react-native/lib/commonjs/light/ContactlessPayment.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const ContactlessPayment = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "contactless-payment-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-contactless-payment-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M95.31 101.2a57.37 57.37 0 0 1 0 53.6 6 6 0 0 1-10.62-5.6 44.75 44.75 0 0 0 0-42.4 6 6 0 1 1 10.62-5.6Zm47.86-34.49a6 6 0 0 0-2.46 8.12 112.67 112.67 0 0 1 0 106.34 6 6 0 1 0 10.58 5.66 124.65 124.65 0 0 0 0-117.66 6 6 0 0 0-8.12-2.46Zm-28 16a6 6 0 0 0-2.48 8.12 79 79 0 0 1 0 74.36 6 6 0 0 0 10.6 5.64 91 91 0 0 0 0-85.64 6 6 0 0 0-8.11-2.49ZM230 128A102 102 0 1 1 128 26a102.12 102.12 0 0 1 102 102Zm-12 0a90 90 0 1 0-90 90 90.1 90.1 0 0 0 90-90Z"
  })]
});
var _default = exports.default = ContactlessPayment;
