// phosphor-react-native/lib/commonjs/light/DropSlash.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const DropSlash = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "drop-slash-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-drop-slash-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M52.44 36a6 6 0 0 0-8.88 8l23.7 26.07C50.51 94.93 42 119.77 42 144a86 86 0 0 0 147.28 60.33L203.56 220a6 6 0 0 0 8.88-8.08ZM128 218a74.09 74.09 0 0 1-74-74c0-21 7.27-42.76 21.59-64.73l105.59 116.15A73.81 73.81 0 0 1 128 218ZM91.4 49.07a6 6 0 0 1-.2-8.49 245.6 245.6 0 0 1 33.36-29.49 6 6 0 0 1 6.88 0 251.42 251.42 0 0 1 41.09 38C199.66 80.26 214 113.09 214 144a86.2 86.2 0 0 1-3.08 22.87 6 6 0 0 1-5.78 4.41 6.2 6.2 0 0 1-1.59-.21 6 6 0 0 1-4.2-7.38A74 74 0 0 0 202 144c0-59.63-59-108.94-74-120.52a253.83 253.83 0 0 0-28.12 25.38 6 6 0 0 1-8.48.21Z"
  })]
});
var _default = exports.default = DropSlash;
