// phosphor-react-native/lib/commonjs/light/EscalatorUp.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const EscalatorUp = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "escalator-up-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-escalator-up-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M224 42h-56a6 6 0 0 0-4.41 1.93L69.37 146H32a14 14 0 0 0-14 14v40a14 14 0 0 0 14 14h56a6 6 0 0 0 4.41-1.93L186.63 110H224a14 14 0 0 0 14-14V56a14 14 0 0 0-14-14Zm2 54a2 2 0 0 1-2 2h-40a6 6 0 0 0-4.41 1.93L85.37 202H32a2 2 0 0 1-2-2v-40a2 2 0 0 1 2-2h40a6 6 0 0 0 4.41-1.93L170.63 54H224a2 2 0 0 1 2 2Zm2.24 75.76a6 6 0 1 1-8.48 8.48L206 166.49V208a6 6 0 0 1-12 0v-41.51l-13.76 13.75a6 6 0 0 1-8.48-8.48l24-24a6 6 0 0 1 8.48 0Z"
  })]
});
var _default = exports.default = EscalatorUp;
