// phosphor-react-native/lib/commonjs/light/GlobeSimple.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const GlobeSimple = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "globe-simple-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-globe-simple-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 26a102 102 0 1 0 102 102A102.12 102.12 0 0 0 128 26Zm89.8 96h-43.91c-1.54-40.77-18.48-68.23-30.43-82.67A90.19 90.19 0 0 1 217.8 122ZM128 215.83a110 110 0 0 1-15.19-19.45A128.37 128.37 0 0 1 94.13 134h67.74a128.37 128.37 0 0 1-18.68 62.38A110 110 0 0 1 128 215.83ZM94.13 122a128.37 128.37 0 0 1 18.68-62.38A110 110 0 0 1 128 40.17a110 110 0 0 1 15.19 19.45A128.37 128.37 0 0 1 161.87 122Zm18.41-82.67c-12 14.44-28.89 41.9-30.43 82.67H38.2a90.19 90.19 0 0 1 74.34-82.67ZM38.2 134h43.91c1.54 40.77 18.48 68.23 30.43 82.67A90.19 90.19 0 0 1 38.2 134Zm105.26 82.67c11.95-14.44 28.89-41.9 30.43-82.67h43.91a90.19 90.19 0 0 1-74.34 82.67Z"
  })]
});
var _default = exports.default = GlobeSimple;
