// phosphor-react-native/lib/commonjs/light/Gradient.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Gradient = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "gradient-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-gradient-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M26 104a6 6 0 0 1 6-6h80a6 6 0 0 1 0 12H32a6 6 0 0 1-6-6Zm198-6h-80a6 6 0 0 0 0 12h80a6 6 0 0 0 0-12ZM72 138H32a6 6 0 0 0 0 12h40a6 6 0 0 0 0-12Zm152 0h-40a6 6 0 0 0 0 12h40a6 6 0 0 0 0-12Zm-126 6a6 6 0 0 0 6 6h48a6 6 0 0 0 0-12h-48a6 6 0 0 0-6 6Zm-42 34H32a6 6 0 0 0 0 12h24a6 6 0 0 0 0-12Zm56 0H88a6 6 0 0 0 0 12h24a6 6 0 0 0 0-12Zm56 0h-24a6 6 0 0 0 0 12h24a6 6 0 0 0 0-12Zm56 0h-24a6 6 0 0 0 0 12h24a6 6 0 0 0 0-12ZM32 70h192a6 6 0 0 0 0-12H32a6 6 0 0 0 0 12Z"
  })]
});
var _default = exports.default = Gradient;
