// phosphor-react-native/lib/commonjs/light/GraphicsCard.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const GraphicsCard = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "graphics-card-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-graphics-card-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M232 50H16a6 6 0 0 0-6 6v152a6 6 0 0 0 12 0v-18h20v18a6 6 0 0 0 12 0v-18h20v18a6 6 0 0 0 12 0v-18h20v18a6 6 0 0 0 12 0v-18h114a14 14 0 0 0 14-14V64a14 14 0 0 0-14-14Zm2 126a2 2 0 0 1-2 2H22V62h210a2 2 0 0 1 2 2Zm-58-94a38 38 0 1 0 38 38 38 38 0 0 0-38-38Zm26 38a25.81 25.81 0 0 1-3.88 13.64l-35.76-35.76A26 26 0 0 1 202 120Zm-52 0a25.81 25.81 0 0 1 3.88-13.64l35.76 35.76A26 26 0 0 1 150 120ZM80 82a38 38 0 1 0 38 38 38 38 0 0 0-38-38Zm26 38a25.81 25.81 0 0 1-3.88 13.64L66.36 97.88A26 26 0 0 1 106 120Zm-52 0a25.81 25.81 0 0 1 3.88-13.64l35.76 35.76A26 26 0 0 1 54 120Z"
  })]
});
var _default = exports.default = GraphicsCard;
