// phosphor-react-native/lib/commonjs/light/HandPalm.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const HandPalm = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "hand-palm-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-hand-palm-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M188 90a25.8 25.8 0 0 0-14 4.11V60a26 26 0 0 0-40.59-21.51A26 26 0 0 0 82 44v10.11A26 26 0 0 0 42 76v76a86 86 0 0 0 172 0v-36a26 26 0 0 0-26-26Zm14 62a74 74 0 0 1-148 0V76a14 14 0 0 1 28 0v44a6 6 0 0 0 12 0V44a14 14 0 0 1 28 0v68a6 6 0 0 0 12 0V60a14 14 0 0 1 28 0v70.39A46.07 46.07 0 0 0 122 176a6 6 0 0 0 12 0 34 34 0 0 1 34-34 6 6 0 0 0 6-6v-20a14 14 0 0 1 28 0Z"
  })]
});
var _default = exports.default = HandPalm;
