// phosphor-react-native/lib/commonjs/light/Hospital.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Hospital = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "hospital-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-hospital-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M248 210h-10v-82a14 14 0 0 0-14-14h-58V48a14 14 0 0 0-14-14H56a14 14 0 0 0-14 14v162H32a6 6 0 0 0 0 12h216a6 6 0 0 0 0-12Zm-24-84a2 2 0 0 1 2 2v82h-60v-84ZM54 48a2 2 0 0 1 2-2h96a2 2 0 0 1 2 2v162h-20v-50a6 6 0 0 0-6-6H80a6 6 0 0 0-6 6v50H54Zm68 162H86v-44h36ZM74 96a6 6 0 0 1 6-6h18V72a6 6 0 0 1 12 0v18h18a6 6 0 0 1 0 12h-18v18a6 6 0 0 1-12 0v-18H80a6 6 0 0 1-6-6Z"
  })]
});
var _default = exports.default = Hospital;
