// phosphor-react-native/lib/commonjs/light/Hurricane.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Hurricane = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "hurricane-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-hurricane-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 98a30 30 0 1 0 30 30 30 30 0 0 0-30-30Zm0 48a18 18 0 1 1 18-18 18 18 0 0 1-18 18Zm27.91-99.37 9.89-37.09a6 6 0 0 0-7.06-7.41 193.94 193.94 0 0 0-58 24.82C62.32 51.66 42 86.6 42 128a86.1 86.1 0 0 0 58.09 81.37l-9.89 37.08a6 6 0 0 0 7.06 7.42 193.94 193.94 0 0 0 58-24.82C193.68 204.34 214 169.4 214 128a86.09 86.09 0 0 0-58.09-81.37Zm-6.86 172.13a187.63 187.63 0 0 1-44.55 20.67l8.68-32.57a6 6 0 0 0-4.26-7.34A74.06 74.06 0 0 1 54 128c0-37.53 17.82-68.07 53-90.77a187.93 187.93 0 0 1 44.5-20.66l-8.68 32.57a6 6 0 0 0 4.26 7.34A74.06 74.06 0 0 1 202 128c0 37.53-17.82 68.07-52.95 90.76Z"
  })]
});
var _default = exports.default = Hurricane;
