// phosphor-react-native/lib/commonjs/light/LinuxLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const LinuxLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "linux-logo-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-linux-logo-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M227.74 212.69a6 6 0 0 1-8.42-.94C218 210.05 186 169.17 186 88a58 58 0 0 0-116 0c0 81.17-31.95 122.05-33.31 123.75a6 6 0 0 1-9.38-7.49C27.68 203.79 58 164.56 58 88a70 70 0 0 1 140 0c0 76.63 30.38 115.87 30.69 116.26a6 6 0 0 1-.95 8.43ZM100 90a10 10 0 1 0 10 10 10 10 0 0 0-10-10Zm66 10a10 10 0 1 0-10 10 10 10 0 0 0 10-10Zm-67.31 30.63a6 6 0 0 0-5.37 10.74l32 16a6 6 0 0 0 2.68.63 6.07 6.07 0 0 0 2.69-.63l32-16a6 6 0 0 0-5.37-10.74L128 145.29ZM128 178a52.07 52.07 0 0 0-45.24 27.08 6 6 0 0 0 10.49 5.84 39.33 39.33 0 0 1 69.51 0A6 6 0 0 0 168 214a5.89 5.89 0 0 0 2.91-.76 6 6 0 0 0 2.33-8.16A52.09 52.09 0 0 0 128 178Z"
  })]
});
var _default = exports.default = LinuxLogo;
