// phosphor-react-native/lib/commonjs/light/MicrophoneStage.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const MicrophoneStage = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "microphone-stage-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-microphone-stage-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M168 18a69.94 69.94 0 0 0-69.26 80l-70 95.46a13.92 13.92 0 0 0 1.39 18.17l14.3 14.3a13.93 13.93 0 0 0 18.17 1.39l95.46-70A70 70 0 1 0 168 18Zm58 70a57.65 57.65 0 0 1-13 36.52L131.49 43A57.95 57.95 0 0 1 226 88ZM55.5 217.59a2 2 0 0 1-2.6-.2L38.61 203.1a2 2 0 0 1-.2-2.6l64.22-87.56a70.32 70.32 0 0 0 40.44 40.43ZM110 88a57.73 57.73 0 0 1 13-36.52L204.53 133A58 58 0 0 1 110 88Zm-1.75 59.75a6 6 0 0 1 0 8.49l-8 8a6 6 0 1 1-8.49-8.49l8-8a6 6 0 0 1 8.5-.01Z"
  })]
});
var _default = exports.default = MicrophoneStage;
