// phosphor-react-native/lib/commonjs/light/Onigiri.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Onigiri = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "onigiri-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-onigiri-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M229.82 147.6 174 51.67l-.09-.15a54 54 0 0 0-91.74 0l-.09.15-55.9 95.93A54 54 0 0 0 72.09 230h111.82a54 54 0 0 0 45.91-82.4ZM162 218H94v-50a2 2 0 0 1 2-2h64a2 2 0 0 1 2 2Zm58.61-21.58a41.47 41.47 0 0 1-36.7 21.58H174v-50a14 14 0 0 0-14-14H96a14 14 0 0 0-14 14v50h-9.91a42 42 0 0 1-35.67-64.15l.08-.14 55.87-95.93a42 42 0 0 1 71.26 0l55.87 95.93.08.14a41.48 41.48 0 0 1 1.02 42.57Z"
  })]
});
var _default = exports.default = Onigiri;
