// phosphor-react-native/lib/commonjs/light/PaintBrush.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PaintBrush = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "paint-brush-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-paint-brush-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M224 26c-20.8 0-44.11 11.41-69.3 33.9-18.08 16.16-33.7 35-44.4 49.1A58 58 0 0 0 34 164c0 32.07-20.43 46.39-21.35 47A6 6 0 0 0 16 222h76a58 58 0 0 0 55-76.3c14.08-10.67 32.92-26.32 49.08-44.4C218.59 76.11 230 52.8 230 32a6 6 0 0 0-6-6ZM92 210H30.65C37.92 200.85 46 185.78 46 164a46 46 0 1 1 46 46Zm29.49-95.91c3.6-4.67 7.88-10 12.71-15.69a78.17 78.17 0 0 1 23.4 23.4c-5.67 4.83-11 9.11-15.69 12.71a58.38 58.38 0 0 0-20.42-20.42Zm45.2-.3a90.24 90.24 0 0 0-24.48-24.48C163.05 66.46 191 42 217.56 38.44 214 65 189.54 93 166.69 113.79Z"
  })]
});
var _default = exports.default = PaintBrush;
