// phosphor-react-native/lib/commonjs/light/Panorama.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Panorama = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "panorama-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-panorama-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M240.53 52.9a13.88 13.88 0 0 0-12.12-2.42l-3.48.94C206 56.51 170.87 66 128 66s-78-9.49-96.93-14.58l-3.48-.94A14 14 0 0 0 10 64v128a13.95 13.95 0 0 0 17.61 13.52l3.18-.85C49.7 199.53 84.91 190 128 190s78.3 9.53 97.21 14.66l3.18.85A14 14 0 0 0 246 192V64a13.93 13.93 0 0 0-5.47-11.1ZM22.78 62.42A2 2 0 0 1 24 62a2.06 2.06 0 0 1 .52.08l3.44.92C47.37 68.24 83.54 78 128 78s80.63-9.76 100.06-15l3.44-.92A2 2 0 0 1 234 64v123.57l-61.47-61.47a14 14 0 0 0-19.8 0l-21.42 21.41L85.9 102.1a14 14 0 0 0-19.8 0L22 146.2V64a2 2 0 0 1 .78-1.58ZM220.41 191a400.39 400.39 0 0 0-56.52-10.86L139.8 156l21.41-21.41a2 2 0 0 1 2.83 0Zm-192.76 2.07-3.15.86A2 2 0 0 1 22 192v-28.83l52.58-52.58a2 2 0 0 1 2.84 0l67.91 67.91c-5.63-.32-11.42-.5-17.33-.5-44.69 0-82.28 10.18-100.35 15.07ZM194 108a10 10 0 1 1 10 10 10 10 0 0 1-10-10Z"
  })]
});
var _default = exports.default = Panorama;
