// phosphor-react-native/lib/commonjs/light/PencilRuler.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PencilRuler = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "pencil-ruler-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-pencil-ruler-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M208 34h-48a14 14 0 0 0-14 14v160a14 14 0 0 0 14 14h48a14 14 0 0 0 14-14V48a14 14 0 0 0-14-14Zm2 174a2 2 0 0 1-2 2h-48a2 2 0 0 1-2-2v-34h26a6 6 0 0 0 0-12h-26v-28h26a6 6 0 0 0 0-12h-26V94h26a6 6 0 0 0 0-12h-26V48a2 2 0 0 1 2-2h48a2 2 0 0 1 2 2ZM76.24 27.76a6 6 0 0 0-8.48 0l-32 32A6 6 0 0 0 34 64v144a14 14 0 0 0 14 14h48a14 14 0 0 0 14-14V64a6 6 0 0 0-1.76-4.24ZM46 178V78h20v100ZM78 78h20v100H78Zm-6-37.51L97.51 66h-51ZM96 210H48a2 2 0 0 1-2-2v-18h52v18a2 2 0 0 1-2 2Z"
  })]
});
var _default = exports.default = PencilRuler;
