// phosphor-react-native/lib/commonjs/light/Pentagon.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Pentagon = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "pentagon-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-pentagon-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "m224.35 89.15-87.94-68.33-.13-.1a14 14 0 0 0-16.56 0l-.13.1-87.94 68.33a14 14 0 0 0-5 15.45l32 107.56v.13A14 14 0 0 0 72 222h112a14 14 0 0 0 13.33-9.71s0-.08 0-.13l32-107.56a14 14 0 0 0-4.98-15.45ZM217.9 101v.12l-32 107.54A2 2 0 0 1 184 210H72a2 2 0 0 1-1.89-1.34l-32-107.54V101a2 2 0 0 1 .72-2.23l.13-.1 87.91-68.3a2 2 0 0 1 2.28 0l87.91 68.3.13.1a2 2 0 0 1 .71 2.23Z"
  })]
});
var _default = exports.default = Pentagon;
