// phosphor-react-native/lib/commonjs/light/PinterestLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PinterestLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "pinterest-logo-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-pinterest-logo-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M222 112c0 22.05-7.7 42.19-21.68 56.73-13 13.52-30.61 21.27-48.32 21.27-19.42 0-31.55-7.07-38.63-13.64l-11.53 49A6 6 0 0 1 96 230a5.89 5.89 0 0 1-1.37-.16 6 6 0 0 1-4.47-7.21l32-136a6 6 0 1 1 11.68 2.74L116.75 162c2.8 4.33 12.46 16 35.25 16 28.51 0 58-24.69 58-66a74 74 0 1 0-138.1 37 6 6 0 1 1-10.39 6A86 86 0 1 1 222 112Z"
  })]
});
var _default = exports.default = PinterestLogo;
