// phosphor-react-native/lib/commonjs/light/PlugsConnected.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PlugsConnected = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "plugs-connected-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-plugs-connected-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M236.24 19.76a6 6 0 0 0-8.48 0l-53.82 53.81-6.79-6.78a30 30 0 0 0-42.42 0L100 91.51l-7.76-7.75a6 6 0 0 0-8.48 8.48l7.75 7.76-24.72 24.73a30 30 0 0 0 0 42.42l6.78 6.79-53.81 53.82a6 6 0 1 0 8.48 8.48l53.82-53.81 6.79 6.78a30 30 0 0 0 42.42 0L156 164.49l7.76 7.75a6 6 0 0 0 8.48-8.48l-7.75-7.76 24.72-24.73a30 30 0 0 0 0-42.42l-6.78-6.79 53.81-53.82a6 6 0 0 0 0-8.48Zm-113.45 161a18 18 0 0 1-25.46 0l-22.06-22.09a18 18 0 0 1 0-25.46L100 108.49 147.51 156Zm57.94-57.94L156 147.51 108.49 100l24.72-24.73a18 18 0 0 1 25.46 0l22.06 22.06a18 18 0 0 1 0 25.46Zm-90.3-88.59a6 6 0 0 1 11.14-4.46l8 20a6 6 0 1 1-11.14 4.46Zm-64 59.54a6 6 0 0 1 7.8-3.34l20 8a6 6 0 1 1-4.46 11.14l-20-8a6 6 0 0 1-3.34-7.8Zm203.14 68.46a6 6 0 0 1-7.8 3.34l-20-8a6 6 0 0 1 4.46-11.14l20 8a6 6 0 0 1 3.34 7.8Zm-64 59.54a6 6 0 1 1-11.14 4.46l-8-20a6 6 0 0 1 11.14-4.46Z"
  })]
});
var _default = exports.default = PlugsConnected;
