// phosphor-react-native/lib/commonjs/light/PushPin.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PushPin = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "push-pin-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-push-pin-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M233.91 82.79 173.22 22.1a14 14 0 0 0-19.81 0L98.93 76.77c-9.52-3.25-34-8.34-59.71 12.41A14 14 0 0 0 38.1 110l49.71 49.71-44.05 44a6 6 0 1 0 8.48 8.48l44.05-44.05L146 217.89a14 14 0 0 0 9.9 4.11h1a14 14 0 0 0 10.19-5.54c19.72-26.21 17.15-47.23 12.46-59.3l54.37-54.55a14 14 0 0 0-.01-19.82Zm-8.49 11.31-57.27 57.46a6 6 0 0 0-1.11 6.92c9.94 19.88-1.71 40.32-9.54 50.72a2 2 0 0 1-3 .2L46.58 101.51a2 2 0 0 1 .18-3c12.5-10.09 24.5-12.76 33.7-12.76a42.13 42.13 0 0 1 17.25 3.41 6 6 0 0 0 6.93-1.16l57.26-57.41a2 2 0 0 1 2.83 0l60.69 60.68a2 2 0 0 1 0 2.83Z"
  })]
});
var _default = exports.default = PushPin;
