// phosphor-react-native/lib/commonjs/light/RocketLaunch.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const RocketLaunch = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "rocket-launch-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-rocket-launch-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M221.86 47.24a14 14 0 0 0-13.11-13.1c-12.31-.73-43.77.39-69.88 26.5L133.52 66H74.35a13.9 13.9 0 0 0-9.89 4.1l-34.35 34.34a14 14 0 0 0 7.94 23.76l39.13 5.46 45.16 45.16L127.8 218a14 14 0 0 0 23.76 7.92l34.35-34.35a13.91 13.91 0 0 0 4.1-9.89v-59.2l5.35-5.35c26.1-26.13 27.23-57.57 26.5-69.89ZM38.11 115a2 2 0 0 1 .49-2l34.34-34.42a2 2 0 0 1 1.41-.58h47.17l-43.65 43.64-38.14-5.32a1.93 1.93 0 0 1-1.62-1.32ZM178 181.65a2 2 0 0 1-.59 1.41l-34.33 34.34a2 2 0 0 1-3.4-1.11l-5.32-38.16L178 134.48Zm8.87-73L128 167.51 88.49 128l58.87-58.88a78.47 78.47 0 0 1 60.69-23 2 2 0 0 1 1.83 1.88 78.47 78.47 0 0 1-23 60.64ZM100 190.31C95.68 199.84 81.13 222 40 222a6 6 0 0 1-6-6c0-41.13 22.16-55.68 31.69-60a6 6 0 1 1 5 10.92c-7 3.17-22.53 13.52-24.47 42.91 29.39-1.94 39.74-17.52 42.91-24.47a6 6 0 1 1 10.92 5Z"
  })]
});
var _default = exports.default = RocketLaunch;
