// phosphor-react-native/lib/commonjs/light/Rss.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Rss = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "rss-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-rss-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M105.5 150.5A69.54 69.54 0 0 1 126 200a6 6 0 0 1-12 0 58 58 0 0 0-58-58 6 6 0 0 1 0-12 69.54 69.54 0 0 1 49.5 20.5ZM56 82a6 6 0 0 0 0 12 106 106 0 0 1 106 106 6 6 0 0 0 12 0A118 118 0 0 0 56 82Zm117.38.62A164.92 164.92 0 0 0 56 34a6 6 0 0 0 0 12 153 153 0 0 1 108.89 45.11A153 153 0 0 1 210 200a6 6 0 0 0 12 0 164.92 164.92 0 0 0-48.62-117.38ZM60 186a10 10 0 1 0 10 10 10 10 0 0 0-10-10Z"
  })]
});
var _default = exports.default = Rss;
