// phosphor-react-native/lib/commonjs/light/Scales.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Scales = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "scales-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-scales-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "m237.57 133.77-32-80a6 6 0 0 0-6.86-3.63L134 64.52V40a6 6 0 0 0-12 0v27.19l-67.3 15a6 6 0 0 0-4.27 3.63l-32 80A6.1 6.1 0 0 0 18 168c0 21.86 23.31 30 38 30s38-8.14 38-30a6.1 6.1 0 0 0-.43-2.23L64.19 92.33 122 79.48V210h-18a6 6 0 0 0 0 12h48a6 6 0 0 0 0-12h-18V76.81l56.21-12.49-27.78 69.45A6.1 6.1 0 0 0 162 136c0 21.86 23.31 30 38 30s38-8.14 38-30a6.1 6.1 0 0 0-.43-2.23ZM56 186a36.89 36.89 0 0 1-17.48-4.56c-5.37-3.13-8.15-7.18-8.49-12.37l26-64.91 26 64.91C81.06 182.85 62.58 186 56 186Zm144-32a36.89 36.89 0 0 1-17.48-4.56c-5.37-3.13-8.15-7.18-8.49-12.37l26-64.91 26 64.91C225.06 150.85 206.58 154 200 154Z"
  })]
});
var _default = exports.default = Scales;
