// phosphor-react-native/lib/commonjs/light/Seat.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Seat = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "seat-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-seat-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M222 232a6 6 0 0 1-6 6H112a6 6 0 1 1 0-12h104a6 6 0 0 1 6 6Zm0-72v32a14 14 0 0 1-14 14h-93.89a13.94 13.94 0 0 1-12.53-7.74l-58.11-116a14.06 14.06 0 0 1 0-12.52l22.12-44a14 14 0 0 1 18.58-6.35l33.74 14.24.34.17a14 14 0 0 1 6.27 18.78l-.09.16-14.25 26.42a2 2 0 0 0 0 1.73l31.78 64a2 2 0 0 0 1.78 1.09H208A14 14 0 0 1 222 160Zm-12 0a2 2 0 0 0-2-2h-64.23a13.94 13.94 0 0 1-12.52-7.74l-31.78-64a14 14 0 0 1 0-12.5l.09-.17 14.25-26.42a2 2 0 0 0 .08-1.47 2 2 0 0 0-.89-1.1L79.35 30.37l-.35-.16a2 2 0 0 0-.89-.21 2 2 0 0 0-1.79 1.1l-22.12 44a2 2 0 0 0 0 1.78l58.12 116a2 2 0 0 0 1.79 1.11H208a2 2 0 0 0 2-2Z"
  })]
});
var _default = exports.default = Seat;
