// phosphor-react-native/lib/commonjs/light/SlackLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SlackLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "slack-logo-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-slack-logo-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M218 128a30 30 0 1 0-36-48V56a30 30 0 0 0-54-18 30 30 0 1 0-48 36H56a30 30 0 0 0-18 54 30 30 0 1 0 36 48v24a30 30 0 0 0 54 18 30 30 0 1 0 48-36h24a30 30 0 0 0 18-54Zm-18-42a18 18 0 0 1 0 36h-18v-18a18 18 0 0 1 18-18Zm-48-48a18 18 0 0 1 18 18v48a18 18 0 0 1-18 18h-18V56a18 18 0 0 1 18-18ZM86 56a18 18 0 0 1 36 0v18h-18a18 18 0 0 1-18-18Zm-48 48a18 18 0 0 1 18-18h48a18 18 0 0 1 18 18v18H56a18 18 0 0 1-18-18Zm18 66a18 18 0 0 1 0-36h18v18a18 18 0 0 1-18 18Zm48 48a18 18 0 0 1-18-18v-48a18 18 0 0 1 18-18h18v66a18 18 0 0 1-18 18Zm66-18a18 18 0 0 1-36 0v-18h18a18 18 0 0 1 18 18Zm30-30h-48a18 18 0 0 1-18-18v-18h66a18 18 0 0 1 0 36Z"
  })]
});
var _default = exports.default = SlackLogo;
