// phosphor-react-native/lib/commonjs/light/SmileyMelting.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SmileyMelting = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "smiley-melting-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-smiley-melting-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M174 140a10 10 0 1 1-10-10 10 10 0 0 1 10 10Zm-48-48a10 10 0 1 0-10 10 10 10 0 0 0 10-10Zm73.62-36.63A102 102 0 0 0 52 196a6 6 0 1 0 8.94-8 90.09 90.09 0 0 1 65.78-150H128a90 90 0 0 1 67.07 150 6 6 0 0 0 8.95 8 102 102 0 0 0-4.41-140.63ZM152 170h-16c-22.65 0-50-18.73-50-42a43.15 43.15 0 0 1 .58-7 6 6 0 1 0-11.83-2 54 54 0 0 0-.75 9c0 14.26 7.2 28 20.27 38.6 12 9.79 27.26 15.4 41.73 15.4h16a10 10 0 0 1 0 20H96a22 22 0 0 0 0 44 6 6 0 0 0 0-12 10 10 0 0 1 0-20h56a22 22 0 0 0 0-44Z"
  })]
});
var _default = exports.default = SmileyMelting;
