// phosphor-react-native/lib/commonjs/light/Speedometer.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Speedometer = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "speedometer-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-speedometer-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "m115.76 155.76 96-96a6 6 0 0 1 8.48 8.48l-96 96a6 6 0 0 1-8.48-8.48ZM128 86a65.9 65.9 0 0 1 21.08 3.44 6 6 0 0 0 3.83-11.38 78 78 0 0 0-102.43 82.6 6 6 0 0 0 6 5.34 5.12 5.12 0 0 0 .67 0 6 6 0 0 0 5.3-6.62A69 69 0 0 1 62 152a66.08 66.08 0 0 1 66-66Zm98 15.9a6 6 0 1 0-10.68 5.48 98.35 98.35 0 0 1 5.16 77.25 2 2 0 0 1-1.91 1.37H37.46a2.07 2.07 0 0 1-1.91-1.41A98.23 98.23 0 0 1 128 54h.9a97 97 0 0 1 43.71 10.72A6 6 0 1 0 178.1 54 108.92 108.92 0 0 0 129 42h-1A110.06 110.06 0 0 0 24.23 188.58 14.08 14.08 0 0 0 37.46 198h181.07a14.06 14.06 0 0 0 13.22-9.37A110.34 110.34 0 0 0 226 101.9Z"
  })]
});
var _default = exports.default = Speedometer;
