// phosphor-react-native/lib/commonjs/light/Spiral.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Spiral = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "spiral-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-spiral-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M246 144a6 6 0 0 1-12 0 98.11 98.11 0 0 0-98-98 90.1 90.1 0 0 0-90 90 82.1 82.1 0 0 0 82 82 74.09 74.09 0 0 0 74-74 66.08 66.08 0 0 0-66-66 58.07 58.07 0 0 0-58 58 50.06 50.06 0 0 0 50 50 42 42 0 0 0 42-42 34 34 0 0 0-34-34 26 26 0 0 0-26 26 18 18 0 0 0 18 18 10 10 0 0 0 10-10 2 2 0 0 0-2-2 6 6 0 0 1 0-12 14 14 0 0 1 14 14 22 22 0 0 1-22 22 30 30 0 0 1-30-30 38 38 0 0 1 38-38 46.06 46.06 0 0 1 46 46 54.06 54.06 0 0 1-54 54 62.07 62.07 0 0 1-62-62 70.08 70.08 0 0 1 70-70 78.09 78.09 0 0 1 78 78 86.1 86.1 0 0 1-86 86 94.11 94.11 0 0 1-94-94A102.12 102.12 0 0 1 136 34a110.13 110.13 0 0 1 110 110Z"
  })]
});
var _default = exports.default = Spiral;
