// phosphor-react-native/lib/commonjs/light/Toilet.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Toilet = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "toilet-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-toilet-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M118 64a6 6 0 0 1-6 6H96a6 6 0 0 1 0-12h16a6 6 0 0 1 6 6Zm52.14 132 3.72 26A14 14 0 0 1 160 238H96a14 14 0 0 1-13.86-16l3.72-26A94.1 94.1 0 0 1 34 112a6 6 0 0 1 6-6h18V40a14 14 0 0 1 14-14h112a14 14 0 0 1 14 14v66h18a6 6 0 0 1 6 6 94.1 94.1 0 0 1-51.86 84ZM70 106h116V40a2 2 0 0 0-2-2H72a2 2 0 0 0-2 2Zm88.71 94.84a94 94 0 0 1-61.42 0L94 223.72a2 2 0 0 0 .47 1.59A2 2 0 0 0 96 226h64a2 2 0 0 0 1.51-.69 2 2 0 0 0 .47-1.59ZM209.78 118H46.22a82 82 0 0 0 163.56 0Z"
  })]
});
var _default = exports.default = Toilet;
