// phosphor-react-native/lib/commonjs/light/TrayArrowUp.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const TrayArrowUp = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "tray-arrow-up-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-tray-arrow-up-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M208 34H48a14 14 0 0 0-14 14v160a14 14 0 0 0 14 14h160a14 14 0 0 0 14-14V48a14 14 0 0 0-14-14ZM48 46h160a2 2 0 0 1 2 2v106h-30.69a13.9 13.9 0 0 0-9.89 4.1l-19.32 19.31a2 2 0 0 1-1.41.59h-41.38a2 2 0 0 1-1.41-.59L86.59 158.1a13.94 13.94 0 0 0-9.9-4.1H46V48a2 2 0 0 1 2-2Zm160 164H48a2 2 0 0 1-2-2v-42h30.69a2 2 0 0 1 1.41.59l19.31 19.31a13.94 13.94 0 0 0 9.9 4.1h41.38a13.9 13.9 0 0 0 9.89-4.1l19.32-19.31a2 2 0 0 1 1.41-.59H210v42a2 2 0 0 1-2 2ZM91.76 108.24a6 6 0 0 1 0-8.48l32-32a6 6 0 0 1 8.48 0l32 32a6 6 0 1 1-8.48 8.48L134 86.49V152a6 6 0 0 1-12 0V86.49l-21.76 21.75a6 6 0 0 1-8.48 0Z"
  })]
});
var _default = exports.default = TrayArrowUp;
