// phosphor-react-native/lib/commonjs/light/TrolleySuitcase.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const TrolleySuitcase = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "trolley-suitcase-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-trolley-suitcase-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M88 158h128a14 14 0 0 0 14-14V72a14 14 0 0 0-14-14h-34V40a14 14 0 0 0-14-14h-32a14 14 0 0 0-14 14v18H88a14 14 0 0 0-14 14v72a14 14 0 0 0 14 14Zm46-118a2 2 0 0 1 2-2h32a2 2 0 0 1 2 2v18h-36ZM86 72a2 2 0 0 1 2-2h128a2 2 0 0 1 2 2v72a2 2 0 0 1-2 2H88a2 2 0 0 1-2-2Zm0 152a14 14 0 1 1-14-14 14 14 0 0 1 14 14Zm144 0a14 14 0 1 1-14-14 14 14 0 0 1 14 14Zm16-40a6 6 0 0 1-6 6H32a6 6 0 0 1 0-12h10V75.31a2 2 0 0 0-.59-1.41L19.76 52.24a6 6 0 1 1 8.48-8.48L49.9 65.41a13.94 13.94 0 0 1 4.1 9.9V178h186a6 6 0 0 1 6 6Z"
  })]
});
var _default = exports.default = TrolleySuitcase;
