// phosphor-react-native/lib/commonjs/light/UserSound.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const UserSound = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "user-sound-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-user-sound-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M139 166.26a66 66 0 1 0-62 0c-22 6.22-41.88 19.15-57.61 37.88a6 6 0 0 0 9.18 7.72C49.11 187.45 77.31 174 108 174s58.9 13.45 79.41 37.86a6 6 0 1 0 9.18-7.72C180.86 185.41 161 172.48 139 166.26ZM54 108a54 54 0 1 1 54 54 54.06 54.06 0 0 1-54-54Zm151.52-41.61a106.33 106.33 0 0 1 0 83.22 6 6 0 0 1-11-4.71 94.29 94.29 0 0 0 0-73.8 6 6 0 0 1 11-4.71ZM246 108a137.16 137.16 0 0 1-11.12 54.37 6 6 0 0 1-11-4.74 126.41 126.41 0 0 0 0-99.26 6 6 0 0 1 11-4.74A137.16 137.16 0 0 1 246 108Z"
  })]
});
var _default = exports.default = UserSound;
