// phosphor-react-native/lib/commonjs/light/UsersFour.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const UsersFour = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "users-four-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-users-four-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M28.4 124.8a6 6 0 0 0 8.4-1.2 54 54 0 0 1 86.4 0 6 6 0 0 0 8.4 1.19 5.59 5.59 0 0 0 1.19-1.19 54 54 0 0 1 86.4 0 6 6 0 0 0 9.6-7.21 65.74 65.74 0 0 0-29.69-22.26 38 38 0 1 0-46.22 0A65.32 65.32 0 0 0 128 110.7a65.32 65.32 0 0 0-24.89-16.57 38 38 0 1 0-46.22 0A65.69 65.69 0 0 0 27.2 116.4a6 6 0 0 0 1.2 8.4ZM176 38a26 26 0 1 1-26 26 26 26 0 0 1 26-26Zm-96 0a26 26 0 1 1-26 26 26 26 0 0 1 26-26Zm119.11 160.13a38 38 0 1 0-46.22 0A65.32 65.32 0 0 0 128 214.7a65.32 65.32 0 0 0-24.89-16.57 38 38 0 1 0-46.22 0A65.69 65.69 0 0 0 27.2 220.4a6 6 0 1 0 9.6 7.2 54 54 0 0 1 86.4 0 6 6 0 0 0 8.4 1.19 5.59 5.59 0 0 0 1.19-1.19 54 54 0 0 1 86.4 0 6 6 0 0 0 9.6-7.21 65.74 65.74 0 0 0-29.68-22.26ZM80 142a26 26 0 1 1-26 26 26 26 0 0 1 26-26Zm96 0a26 26 0 1 1-26 26 26 26 0 0 1 26-26Z"
  })]
});
var _default = exports.default = UsersFour;
