// phosphor-react-native/lib/commonjs/light/UserCircleCheck.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const UserCircleCheck = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "user-circle-check-light__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-user-circle-check-light',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M221.68 106.08a6 6 0 0 0-4.92 6.91A91.66 91.66 0 0 1 218 128a89.65 89.65 0 0 1-24.49 61.64 77.53 77.53 0 0 0-40-31.38 46 46 0 1 0-51 0 77.53 77.53 0 0 0-40 31.38A89.95 89.95 0 0 1 128 38a91.57 91.57 0 0 1 15 1.24 6 6 0 1 0 2-11.83 101.9 101.9 0 1 0 83.6 83.6 6 6 0 0 0-6.92-4.93ZM94 120a34 34 0 1 1 34 34 34 34 0 0 1-34-34Zm-22.56 78a66 66 0 0 1 113.12 0 89.8 89.8 0 0 1-113.12 0Zm164.8-153.76-32 32a6 6 0 0 1-8.48 0l-16-16a6 6 0 0 1 8.48-8.48L200 63.51l27.76-27.75a6 6 0 0 1 8.48 8.48Z"
  })]
});
var _default = exports.default = UserCircleCheck;
