// phosphor-react-native/lib/module/thin/ArticleNyTimes.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const ArticleNyTimes = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "article-ny-times-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-article-ny-times-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M128 100h104a4 4 0 0 1 0 8H128a4 4 0 0 1 0-8Zm104 32H128a4 4 0 0 0 0 8h104a4 4 0 0 0 0-8Zm0 32H80a4 4 0 0 0 0 8h152a4 4 0 0 0 0-8Zm0 32H80a4 4 0 0 0 0 8h152a4 4 0 0 0 0-8ZM96 140a4 4 0 0 0 0-8H84V60h40v12a4 4 0 0 0 8 0V56a4 4 0 0 0-4-4H32a4 4 0 0 0-4 4v16a4 4 0 0 0 8 0V60h40v72H64a4 4 0 0 0 0 8Z"
  })]
});
export default ArticleNyTimes;
