// phosphor-react-native/lib/module/thin/BatteryMedium.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const BatteryMedium = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "battery-medium-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-battery-medium-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M200 60H32a20 20 0 0 0-20 20v96a20 20 0 0 0 20 20h168a20 20 0 0 0 20-20V80a20 20 0 0 0-20-20Zm12 116a12 12 0 0 1-12 12H32a12 12 0 0 1-12-12V80a12 12 0 0 1 12-12h168a12 12 0 0 1 12 12ZM100 96v64a4 4 0 0 1-8 0V96a4 4 0 0 1 8 0Zm-40 0v64a4 4 0 0 1-8 0V96a4 4 0 0 1 8 0Zm192 0v64a4 4 0 0 1-8 0V96a4 4 0 0 1 8 0Z"
  })]
});
export default BatteryMedium;
