// phosphor-react-native/lib/module/thin/DiceFour.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const DiceFour = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "dice-four-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-dice-four-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M192 36H64a28 28 0 0 0-28 28v128a28 28 0 0 0 28 28h128a28 28 0 0 0 28-28V64a28 28 0 0 0-28-28Zm20 156a20 20 0 0 1-20 20H64a20 20 0 0 1-20-20V64a20 20 0 0 1 20-20h128a20 20 0 0 1 20 20Zm-104-92a8 8 0 1 1-8-8 8 8 0 0 1 8 8Zm56 0a8 8 0 1 1-8-8 8 8 0 0 1 8 8Zm-56 56a8 8 0 1 1-8-8 8 8 0 0 1 8 8Zm56 0a8 8 0 1 1-8-8 8 8 0 0 1 8 8Z"
  })]
});
export default DiceFour;
