// phosphor-react-native/lib/module/thin/Dresser.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Dresser = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "dresser-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-dresser-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M140 192a4 4 0 0 1-4 4h-16a4 4 0 0 1 0-8h16a4 4 0 0 1 4 4ZM120 68h16a4 4 0 0 0 0-8h-16a4 4 0 0 0 0 8Zm16 56h-16a4 4 0 0 0 0 8h16a4 4 0 0 0 0-8Zm76-84v176a12 12 0 0 1-12 12H56a12 12 0 0 1-12-12V40a12 12 0 0 1 12-12h144a12 12 0 0 1 12 12ZM52 156h152v-56H52Zm0-116v52h152V40a4 4 0 0 0-4-4H56a4 4 0 0 0-4 4Zm152 176v-52H52v52a4 4 0 0 0 4 4h144a4 4 0 0 0 4-4Z"
  })]
});
export default Dresser;
