// phosphor-react-native/lib/module/thin/FunnelSimpleX.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const FunnelSimpleX = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "funnel-simple-x-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-funnel-simple-x-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M192 140H64a4 4 0 0 1 0-8h128a4 4 0 0 1 0 8Zm40-56H24a4 4 0 0 0 0 8h208a4 4 0 0 0 0-8Zm-104 96h-24a4 4 0 0 0 0 8h24a4 4 0 0 0 0-8Zm90.83-6.83a4 4 0 0 0-5.66 0L192 194.34l-21.17-21.17a4 4 0 0 0-5.66 5.66L186.34 200l-21.17 21.17a4 4 0 0 0 5.66 5.66L192 205.66l21.17 21.17a4 4 0 0 0 5.66-5.66L197.66 200l21.17-21.17a4 4 0 0 0 0-5.66Z"
  })]
});
export default FunnelSimpleX;
