// phosphor-react-native/lib/module/thin/MusicNotesPlus.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const MusicNotesPlus = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "music-notes-plus-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-music-notes-plus-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M228 48a4 4 0 0 1-4 4h-20v20a4 4 0 0 1-8 0V52h-20a4 4 0 0 1 0-8h20V24a4 4 0 0 1 8 0v20h20a4 4 0 0 1 4 4Zm-16 64v52a32.06 32.06 0 1 1-8-21.13V112a4 4 0 0 1 8 0Zm-8 52a24 24 0 1 0-24 24 24 24 0 0 0 24-24ZM84 107.12V196a32.06 32.06 0 1 1-8-21.13V56a4 4 0 0 1 3-3.88l56-14a4 4 0 0 1 2 7.76L84 59.12v39.76l75-18.76a4 4 0 0 1 2 7.76ZM76 196a24 24 0 1 0-24 24 24 24 0 0 0 24-24Z"
  })]
});
export default MusicNotesPlus;
