// phosphor-react-native/lib/module/thin/TextColumns.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const TextColumns = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "text-columns-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-text-columns-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M116 64a4 4 0 0 1-4 4H40a4 4 0 0 1 0-8h72a4 4 0 0 1 4 4Zm-4 36H40a4 4 0 0 0 0 8h72a4 4 0 0 0 0-8Zm0 40H40a4 4 0 0 0 0 8h72a4 4 0 0 0 0-8Zm0 40H40a4 4 0 0 0 0 8h72a4 4 0 0 0 0-8Zm32-112h72a4 4 0 0 0 0-8h-72a4 4 0 0 0 0 8Zm72 32h-72a4 4 0 0 0 0 8h72a4 4 0 0 0 0-8Zm0 40h-72a4 4 0 0 0 0 8h72a4 4 0 0 0 0-8Zm0 40h-72a4 4 0 0 0 0 8h72a4 4 0 0 0 0-8Z"
  })]
});
export default TextColumns;
