// phosphor-react-native/lib/module/thin/Tornado.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Tornado = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "tornado-thin__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-tornado-thin',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M228 40a4 4 0 0 1-4 4H56a4 4 0 0 1 0-8h168a4 4 0 0 1 4 4Zm-48 32a4 4 0 0 0-4-4H32a4 4 0 0 0 0 8h144a4 4 0 0 0 4-4Zm-16 32a4 4 0 0 0-4-4H56a4 4 0 0 0 0 8h104a4 4 0 0 0 4-4Zm16 32a4 4 0 0 0-4-4H88a4 4 0 0 0 0 8h88a4 4 0 0 0 4-4Zm4 28h-64a4 4 0 0 0 0 8h64a4 4 0 0 0 0-8Zm-24 32h-32a4 4 0 0 0 0 8h32a4 4 0 0 0 0-8Zm-32 32h-16a4 4 0 0 0 0 8h16a4 4 0 0 0 0-8Z"
  })]
});
export default Tornado;
