// phosphor-react-native/lib/commonjs/bold/Aperture.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Aperture = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "aperture-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-aperture-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M204.41 51.63a108 108 0 1 0 0 152.74 107.38 107.38 0 0 0 0-152.74Zm-17 17A83.85 83.85 0 0 1 196.26 79L169 111.09l-23.3-65.21a83.52 83.52 0 0 1 41.73 22.72Zm-118.85 0a83.44 83.44 0 0 1 51.11-24.2l14.16 39.65-68.12-12.47c.93-1.02 1.88-2.02 2.87-3.01ZM48 153.7a84.48 84.48 0 0 1 3.4-60.3l41.44 7.6Zm20.55 33.7a83.94 83.94 0 0 1-8.81-10.4L87 144.91l23.3 65.21a83.53 83.53 0 0 1-41.72-22.72Zm36.36-63.61 15.18-17.85 23.06 4.21 7.88 22.06-15.17 17.85-23.06-4.21Zm82.49 63.61a83.49 83.49 0 0 1-51.11 24.2L122.15 172l68.14 12.44c-.93.97-1.88 1.97-2.86 2.96ZM163.16 155 208 102.3a84.43 84.43 0 0 1-3.41 60.3Z"
  })]
});
var _default = exports.default = Aperture;
