// phosphor-react-native/lib/commonjs/bold/Axe.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Axe = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "axe-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-axe-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M254.93 96.43A20 20 0 0 0 238.56 83a122 122 0 0 1-39.86-12.46l3.92-3.92a32 32 0 0 0-45.26-45.26l-6.69 6.69a20 20 0 0 0-30.15-2.21L81.84 64.52a20 20 0 0 0 2.21 30.16l-74.69 74.7a32 32 0 0 0 45.25 45.26l71.92-71.93A122 122 0 0 1 139 182.57a20 20 0 0 0 34 11.58L250.13 117a19.86 19.86 0 0 0 4.8-20.57Zm-80.6-58.1a8 8 0 1 1 11.32 11.32l-7.58 7.57a135.65 135.65 0 0 1-11.94-10.68ZM37.64 197.67a8 8 0 0 1-11.31-11.32l76.21-76.21a135.65 135.65 0 0 1 10.68 11.94Zm123.88-26c-9-48.5-39.27-78.44-59.38-93.47l32-32c15 20.11 45 50.34 93.45 59.38Z"
  })]
});
var _default = exports.default = Axe;
