// phosphor-react-native/lib/commonjs/bold/BabyCarriage.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const BabyCarriage = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "baby-carriage-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-baby-carriage-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M160 28h-8a20 20 0 0 0-20 20v52H58.16A40.07 40.07 0 0 0 20 72a12 12 0 0 0 0 24 16 16 0 0 1 16 16 84.09 84.09 0 0 0 84 84h40a84 84 0 0 0 0-168Zm48.06 48.12A59.58 59.58 0 0 1 218.79 100h-40.58ZM160 52a59.66 59.66 0 0 1 29.83 8L156 87V52Zm0 120h-40a60.1 60.1 0 0 1-58.79-48h157.58A60.1 60.1 0 0 1 160 172Zm-52 52a20 20 0 1 1-20-20 20 20 0 0 1 20 20Zm104 0a20 20 0 1 1-20-20 20 20 0 0 1 20 20Z"
  })]
});
var _default = exports.default = BabyCarriage;
