// phosphor-react-native/lib/commonjs/bold/Balloon.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Balloon = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "balloon-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-balloon-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 12a92.1 92.1 0 0 0-92 92c0 24.53 9.55 50.13 26.19 70.22 10 12 21.56 21.07 34.05 26.76L85 227.27A12 12 0 0 0 96 244h64a12 12 0 0 0 11-16.73L159.76 201c12.49-5.69 24.08-14.73 34.05-26.76C210.45 154.13 220 128.53 220 104a92.1 92.1 0 0 0-92-92Zm13.8 208h-27.6l5.35-12.49a73.1 73.1 0 0 0 16.9 0Zm33.53-61.09C161.93 175.09 145.12 184 128 184s-33.93-8.91-47.33-25.09C67.73 143.29 60 122.76 60 104a68 68 0 0 1 136 0c0 18.76-7.73 39.29-20.67 54.91Zm-6.34-47q-.6.06-1.2.06a12 12 0 0 1-11.93-10.81 28 28 0 0 0-19.47-23.91 12 12 0 1 1 7.22-22.89 51.94 51.94 0 0 1 36.13 44.42A12 12 0 0 1 169 111.94Z"
  })]
});
var _default = exports.default = Balloon;
