// phosphor-react-native/lib/commonjs/bold/Barn.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Barn = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "barn-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-barn-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M100 84a12 12 0 0 1 12-12h32a12 12 0 0 1 0 24h-32a12 12 0 0 1-12-12Zm152 116a12 12 0 0 1-12 12H16a12 12 0 0 1 0-24h4v-48.69A12 12 0 0 1 6.24 121l40-56a11.91 11.91 0 0 1 2.94-2.9l67.56-46.65.17-.11a19.94 19.94 0 0 1 22.18 0l.17.11 67.56 46.65a11.91 11.91 0 0 1 2.94 2.9l40 56A12 12 0 0 1 236 139.31V188h4a12 12 0 0 1 12 12ZM44 188h20v-64a12 12 0 0 1 12-12h104a12 12 0 0 1 12 12v64h20v-78.55l-20.55-28.76L128 36.87 64.55 80.69 44 109.45Zm124-52h-54.1l54.1 38.07Zm-80 52h58.1L88 147.12Z"
  })]
});
var _default = exports.default = Barn;
