// phosphor-react-native/lib/commonjs/bold/BaseballCap.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const BaseballCap = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "baseball-cap-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-baseball-cap-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 20A108.12 108.12 0 0 0 20 128v56a27.86 27.86 0 0 0 15.26 24.93 28 28 0 0 0 29.28-2.34C76.2 198.11 96.68 188 128 188s51.8 10.1 63.46 18.58A28 28 0 0 0 236 184v-56A108.12 108.12 0 0 0 128 20Zm84 108v1.87a170 170 0 0 0-33.29-14.3 170.81 170.81 0 0 0-23.45-67A84.14 84.14 0 0 1 212 128Zm-58.46-18.12a174.42 174.42 0 0 0-51.08 0A150 150 0 0 1 128 50.94a150.07 150.07 0 0 1 25.54 58.94Zm-52.8-61.31a170.81 170.81 0 0 0-23.45 67A170 170 0 0 0 44 129.87V128a84.14 84.14 0 0 1 56.74-79.43Zm109.11 139a4 4 0 0 1-4.28-.36C191 176.61 165.77 164 128 164s-63 12.61-77.57 23.18a4 4 0 0 1-4.28.36A3.76 3.76 0 0 1 44 184v-25.86a148 148 0 0 1 168 0V184a3.76 3.76 0 0 1-2.15 3.54Z"
  })]
});
var _default = exports.default = BaseballCap;
