// phosphor-react-native/lib/commonjs/bold/Baseball.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Baseball = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "baseball-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-baseball-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 20a108 108 0 1 0 108 108A108.12 108.12 0 0 0 128 20ZM77 194.66A108.9 108.9 0 0 0 85.54 182a12 12 0 1 0-20.77-12 83.28 83.28 0 0 1-4.77 7.27 83.73 83.73 0 0 1 0-98.54A83.28 83.28 0 0 1 64.77 86a12 12 0 1 0 20.77-12A108.9 108.9 0 0 0 77 61.34a83.76 83.76 0 0 1 102.06 0 108.9 108.9 0 0 0-8.6 12.66 12 12 0 1 0 20.77 12 83.28 83.28 0 0 1 4.77-7.27 83.73 83.73 0 0 1 0 98.54 83.28 83.28 0 0 1-4.74-7.28 12 12 0 1 0-20.77 12 108.9 108.9 0 0 0 8.51 12.67 83.76 83.76 0 0 1-102.06 0Zm22.2-80.14A110 110 0 0 1 100 128a110 110 0 0 1-.83 13.48A12 12 0 0 1 87.27 152a12.83 12.83 0 0 1-1.5-.09 12 12 0 0 1-10.42-13.39A85.75 85.75 0 0 0 76 128a85.75 85.75 0 0 0-.65-10.52 12 12 0 1 1 23.82-3Zm71.06 37.39a12.83 12.83 0 0 1-1.5.09 12 12 0 0 1-11.9-10.52 109.88 109.88 0 0 1 0-27 12 12 0 1 1 23.82 3 85.46 85.46 0 0 0 0 21 12 12 0 0 1-10.45 13.43Z"
  })]
});
var _default = exports.default = Baseball;
