// phosphor-react-native/lib/commonjs/bold/BeachBall.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const BeachBall = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "beach-ball-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-beach-ball-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 20a108 108 0 1 0 108 108A108.12 108.12 0 0 0 128 20Zm84 108a84.26 84.26 0 0 1-1.32 14.74 204.79 204.79 0 0 0-18.52-31.49 176.78 176.78 0 0 1 19.08 5.62A85 85 0 0 1 212 128Zm-9.61-38.94a204.25 204.25 0 0 0-30-5.47 204.25 204.25 0 0 0-5.47-30 84.53 84.53 0 0 1 35.47 35.47Zm-63.26-44.3a176.78 176.78 0 0 1 5.62 19.08 204.79 204.79 0 0 0-31.49-18.52 82.49 82.49 0 0 1 25.87-.56Zm-58.06 13.6a179.84 179.84 0 0 1 49.8 25.06 204.19 204.19 0 0 0-85.23 28.05 84.23 84.23 0 0 1 35.43-53.11ZM45 140.65a180.19 180.19 0 0 1 104.62-34.22 180.19 180.19 0 0 1-34.22 104.62 84.22 84.22 0 0 1-70.4-70.4Zm99.58 69.71a204.19 204.19 0 0 0 28.05-85.23 179.84 179.84 0 0 1 25.06 49.8 84.23 84.23 0 0 1-53.16 35.43Z"
  })]
});
var _default = exports.default = BeachBall;
