// phosphor-react-native/lib/commonjs/bold/Bone.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Bone = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "bone-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-bone-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M234.74 58.31a39.89 39.89 0 0 0-23.35-13.7 40 40 0 1 0-74.86 25.87l-66 66.05a40 40 0 1 0-25.87 74.86 40 40 0 1 0 74.86-25.86l66.05-66.06a40 40 0 0 0 49.22-61.16Zm-18.64 36.2a16 16 0 0 1-20.54 3 20 20 0 0 0-24.84 2.76l-70.4 70.4a20 20 0 0 0-2.75 24.84 16 16 0 1 1-29.41 6A12 12 0 0 0 56.3 187.7a11.61 11.61 0 0 0-1.85.14 16 16 0 1 1 6-29.4 20 20 0 0 0 24.83-2.76l70.4-70.4a20 20 0 0 0 2.76-24.84 16 16 0 1 1 29.4-6 12 12 0 0 0 13.71 13.71 16 16 0 0 1 14.55 26.36Z"
  })]
});
var _default = exports.default = Bone;
