// phosphor-react-native/lib/commonjs/bold/Brain.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Brain = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "brain-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-brain-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M252 124a60.14 60.14 0 0 0-32-53.08 52 52 0 0 0-92-32.11 52 52 0 0 0-92 32.11 60 60 0 0 0 0 106.14 52 52 0 0 0 92 32.13 52 52 0 0 0 92-32.13A60.05 60.05 0 0 0 252 124ZM88 204a28 28 0 0 1-26.85-20.07c1 0 1.89.07 2.85.07h8a12 12 0 0 0 0-24h-8a36 36 0 0 1-12-69.95 12 12 0 0 0 8-11.32V72a28 28 0 0 1 56 0v60.18a51.61 51.61 0 0 0-7.2-3.85 12 12 0 1 0-9.6 22A28 28 0 0 1 88 204Zm104-44h-8a12 12 0 0 0 0 24h8c1 0 1.9 0 2.85-.07a28 28 0 1 1-38-33.61 12 12 0 1 0-9.6-22 51.61 51.61 0 0 0-7.2 3.85V72a28 28 0 0 1 56 0v6.73a12 12 0 0 0 8 11.32 36 36 0 0 1-12 70Zm16-44a12 12 0 0 1-12 12 40 40 0 0 1-40-40v-4a12 12 0 0 1 24 0v4a16 16 0 0 0 16 16 12 12 0 0 1 12 12ZM100 88a40 40 0 0 1-40 40 12 12 0 0 1 0-24 16 16 0 0 0 16-16v-4a12 12 0 0 1 24 0Z"
  })]
});
var _default = exports.default = Brain;
