// phosphor-react-native/lib/commonjs/bold/CastleTurret.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const CastleTurret = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "castle-turret-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-castle-turret-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M216 212h-12v-95l10.14-10.14A19.86 19.86 0 0 0 220 92.69V48a20 20 0 0 0-20-20H56a20 20 0 0 0-20 20v44.69a19.86 19.86 0 0 0 5.86 14.14L52 117v95H40a12 12 0 0 0 0 24h176a12 12 0 0 0 0-24ZM72.49 103.51 60 91V52h28v24a12 12 0 0 0 24 0V52h32v24a12 12 0 0 0 24 0V52h28v39l-12.49 12.48A12 12 0 0 0 180 112v100h-16v-44a36 36 0 0 0-72 0v44H76V112a12 12 0 0 0-3.51-8.49ZM140 212h-24v-44a12 12 0 0 1 24 0Z"
  })]
});
var _default = exports.default = CastleTurret;
