// phosphor-react-native/lib/commonjs/bold/CloudMoon.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const CloudMoon = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "cloud-moon-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-cloud-moon-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M172 68c-1.66 0-3.31.06-4.95.16a75.93 75.93 0 0 0-58-62.23 12 12 0 0 0-14.37 14.38A52.05 52.05 0 0 1 32.3 82.68a12 12 0 0 0-14.37 14.39 76.61 76.61 0 0 0 27.91 43.27A56 56 0 0 0 92 228h80a80 80 0 0 0 0-160Zm-52.11-32a51.64 51.64 0 0 1 23.68 37.17 80.39 80.39 0 0 0-45.18 43.15A56.5 56.5 0 0 0 92 116a55.69 55.69 0 0 0-28.23 7.66 52.69 52.69 0 0 1-15.63-15.77A76.11 76.11 0 0 0 119.89 36ZM172 204H92a32 32 0 0 1 0-64h.28c-.11 1.1-.2 2.2-.26 3.3a12 12 0 0 0 24 1.4 55.78 55.78 0 0 1 1.74-11l.15-.55A56.06 56.06 0 1 1 172 204Z"
  })]
});
var _default = exports.default = CloudMoon;
