// phosphor-react-native/lib/commonjs/bold/CompassTool.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const CompassTool = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "compass-tool-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-compass-tool-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M218.68 125.46a12 12 0 1 0-21.37-10.92 75.15 75.15 0 0 1-27.66 29.64l-13.5-30.39A44 44 0 0 0 140 37.68V24a12 12 0 0 0-24 0v13.68a44 44 0 0 0-16.15 76.11L53 219.12A12 12 0 0 0 59.13 235a11.86 11.86 0 0 0 4.87 1 12 12 0 0 0 11-7.13l23.67-53.26A99.52 99.52 0 0 0 128 180a102.81 102.81 0 0 0 29.39-4.32L181 228.87a12 12 0 0 0 11 7.13 11.85 11.85 0 0 0 4.86-1 12 12 0 0 0 6.14-15.88l-23.51-52.9a99.39 99.39 0 0 0 39.19-40.76ZM128 60a20 20 0 1 1-20 20 20 20 0 0 1 20-20Zm0 96a75.8 75.8 0 0 1-19.52-2.53l13.3-29.92a43.21 43.21 0 0 0 12.44 0l13.33 30A79.11 79.11 0 0 1 128 156Z"
  })]
});
var _default = exports.default = CompassTool;
