// phosphor-react-native/lib/commonjs/bold/DropboxLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const DropboxLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "dropbox-logo-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-dropbox-logo-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M238.83 146.13 201.08 120l37.75-26.13a12 12 0 0 0 0-19.74l-52-36a12 12 0 0 0-13.66 0L128 69.41 82.83 38.13a12 12 0 0 0-13.66 0l-52 36a12 12 0 0 0 0 19.74L54.92 120l-37.75 26.13a12 12 0 0 0 0 19.74l52 36a12 12 0 0 0 13.66 0L128 170.59l45.17 31.28a12 12 0 0 0 13.66 0l52-36a12 12 0 0 0 0-19.74ZM128 141.41 97.08 120 128 98.59 158.92 120Zm52-78.81L210.92 84 180 105.4 149.08 84Zm-104 0L106.92 84 76 105.4 45.08 84Zm0 114.8L45.08 156 76 134.6l30.92 21.4Zm104 0L149.08 156 180 134.6l30.92 21.4Zm-25.27 32.09a12 12 0 0 1-3 16.7l-16.86 11.68a12 12 0 0 1-13.66 0l-16.86-11.68A12 12 0 1 1 118 206.46l10 6.94 10-6.94a12 12 0 0 1 16.73 3.03Z"
  })]
});
var _default = exports.default = DropboxLogo;
