// phosphor-react-native/lib/commonjs/bold/FinnTheHuman.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const FinnTheHuman = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "finn-the-human-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-finn-the-human-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M160 96H96a52 52 0 0 0 0 104h64a52 52 0 0 0 0-104Zm0 80H96a28 28 0 0 1 0-56h64a28 28 0 0 1 0 56Zm-48-28a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm64 0a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm36-112a36 36 0 0 0-33.94 24H77.94A36 36 0 0 0 8 72v76a88.1 88.1 0 0 0 88 88h64a88.1 88.1 0 0 0 88-88V72a36 36 0 0 0-36-36Zm12 112a64.07 64.07 0 0 1-64 64H96a64.07 64.07 0 0 1-64-64V72a12 12 0 0 1 24 0 12 12 0 0 0 12 12h120a12 12 0 0 0 12-12 12 12 0 0 1 24 0Z"
  })]
});
var _default = exports.default = FinnTheHuman;
