// phosphor-react-native/lib/commonjs/bold/FireTruck.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const FireTruck = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "fire-truck-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-fire-truck-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "m255.14 115.54-14-35A19.9 19.9 0 0 0 222.58 68H204v-4a12 12 0 0 0-24 0v94.06A36.23 36.23 0 0 0 158.06 180h-44.12a36 36 0 0 0-67.88 0H36v-36a12 12 0 0 0-24 0v40a20 20 0 0 0 20 20h14.06a36 36 0 0 0 67.88 0h44.12a36 36 0 0 0 67.88 0H236a20 20 0 0 0 20-20v-64a12 12 0 0 0-.86-4.46ZM219.88 92l6.4 16H204V92ZM80 204a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm112 0a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm33.94-24A36.23 36.23 0 0 0 204 158.06V132h28v48ZM24 92a12 12 0 0 0 0 24h128a12 12 0 0 0 0-24h-12V76h12a12 12 0 0 0 0-24H24a12 12 0 0 0 0 24h12v16Zm92 0h-16V76h16ZM60 76h16v16H60Z"
  })]
});
var _default = exports.default = FireTruck;
