// phosphor-react-native/lib/commonjs/bold/FourK.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const FourK = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "four-k-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-four-k-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M20 44a12 12 0 0 1 12-12h192a12 12 0 0 1 0 24H32a12 12 0 0 1-12-12Zm204 156H32a12 12 0 0 0 0 24h192a12 12 0 0 0 0-24ZM148 68a12 12 0 0 0-12 12v96a12 12 0 0 0 24 0v-27.49l8.44-9.65 25.19 43.19A12 12 0 0 0 214.37 170l-29.26-50.14L213 87.9a12 12 0 1 0-18-15.8l-35 40V80a12 12 0 0 0-12-12ZM80 176v-12H36a12 12 0 0 1-9.47-19.37l56-72A12 12 0 0 1 104 80v60h4a12 12 0 0 1 0 24h-4v12a12 12 0 0 1-24 0Zm0-36v-25l-19.46 25Z"
  })]
});
var _default = exports.default = FourK;
