// phosphor-react-native/lib/commonjs/bold/GasCan.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const GasCan = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "gas-can-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-gas-can-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M200 20h-68.69a19.86 19.86 0 0 0-14.14 5.86l-11.51 11.51-15.52-15.51a20 20 0 0 0-28.28 0l-24 24a20 20 0 0 0 0 28.28l15.51 15.52-11.51 11.51A19.86 19.86 0 0 0 36 115.31V216a20 20 0 0 0 20 20h144a20 20 0 0 0 20-20V40a20 20 0 0 0-20-20ZM57.66 60 76 41.66l12.69 12.68-18.35 18.35ZM196 212H60v-95l18.83-18.86L133 44h63ZM136 68a12 12 0 0 1 12-12h20a12 12 0 0 1 0 24h-20a12 12 0 0 1-12-12Zm39.5 65.37L147.21 156l28.29 22.63a12 12 0 0 1-15 18.74l-32.5-26-32.5 26a12 12 0 0 1-15-18.74L108.79 156 80.5 133.37a12 12 0 0 1 15-18.74l32.5 26 32.5-26a12 12 0 0 1 15 18.74Z"
  })]
});
var _default = exports.default = GasCan;
