// phosphor-react-native/lib/commonjs/bold/GlobeSimpleX.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const GlobeSimpleX = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "globe-simple-x-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-globe-simple-x-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M224.49 176.49 209 192l15.52 15.51a12 12 0 0 1-17 17L192 209l-15.51 15.52a12 12 0 0 1-17-17L175 192l-15.52-15.51a12 12 0 0 1 17-17L192 175l15.51-15.52a12 12 0 0 1 17 17ZM236 128a12 12 0 0 1-12 12H100.54c4.37 49.56 34.63 74.37 35 74.63A12 12 0 0 1 128 236a108 108 0 1 1 108-108Zm-77.79-78.36A144.3 144.3 0 0 1 179.56 116h31.57a84.22 84.22 0 0 0-52.92-66.36Zm-40.57 13.59A119.74 119.74 0 0 0 100.53 116h54.94c-3-33.9-18.09-56.22-27.45-67a107.75 107.75 0 0 0-10.38 14.23ZM44.87 116h31.57a144.3 144.3 0 0 1 21.35-66.36A84.22 84.22 0 0 0 44.87 116Zm31.57 24H44.87a84.22 84.22 0 0 0 52.92 66.36A144.3 144.3 0 0 1 76.44 140Z"
  })]
});
var _default = exports.default = GlobeSimpleX;
