// phosphor-react-native/lib/commonjs/bold/Goggles.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Goggles = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "goggles-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-goggles-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M180 60H76a76.08 76.08 0 0 0-76 76v4a24 24 0 0 0 13.74 21.69c3.15 8.71 10.51 16.75 21.52 23.27 11.52 6.81 25.6 11 36.74 11a44.06 44.06 0 0 0 42.32-32h27.36A44.06 44.06 0 0 0 184 196c18.53 0 50.62-12.81 58.31-34.33A24 24 0 0 0 256 140v-4a76.08 76.08 0 0 0-76-76ZM76 84h104a52.07 52.07 0 0 1 51.13 42.6 65 65 0 0 0-10.39-7.6c-11.52-6.81-25.6-11-36.74-11a44.06 44.06 0 0 0-42.32 32h-27.36A44.06 44.06 0 0 0 72 108c-13.4 0-33.9 6.71-47.13 18.56A52.08 52.08 0 0 1 76 84Zm-4 88c-14.13 0-36-12.15-36-20 0-2.74 3.55-7.61 11.48-12.3 9-5.32 18.8-7.7 24.52-7.7a20 20 0 0 1 0 40Zm136.52-7.7c-9 5.32-18.8 7.7-24.52 7.7a20 20 0 0 1 0-40c14.13 0 36 12.15 36 20 0 2.74-3.55 7.61-11.48 12.3Z"
  })]
});
var _default = exports.default = Goggles;
