// phosphor-react-native/lib/commonjs/bold/Golf.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Golf = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "golf-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-golf-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M180 100a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm-48 16a16 16 0 1 0 16 16 16 16 0 0 0-16-16Zm88-20a92 92 0 1 1-92-92 92.1 92.1 0 0 1 92 92Zm-24 0a68 68 0 1 0-68 68 68.07 68.07 0 0 0 68-68Zm-32 101.86a108.61 108.61 0 0 1-72 0 12 12 0 0 0-8 22.63 131.83 131.83 0 0 0 32 7V244a12 12 0 0 0 24 0v-16.54a131.83 131.83 0 0 0 32-7 12 12 0 0 0-8-22.63Z"
  })]
});
var _default = exports.default = Golf;
