// phosphor-react-native/lib/commonjs/bold/GoogleCardboardLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const GoogleCardboardLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "google-cardboard-logo-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-google-cardboard-logo-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M224 44H32a20 20 0 0 0-20 20v128a20 20 0 0 0 20 20h64a12 12 0 0 0 8.49-3.51L128 185l23.51 23.52A12 12 0 0 0 160 212h64a20 20 0 0 0 20-20V64a20 20 0 0 0-20-20Zm-4 144h-55l-22.83-22.83a20 20 0 0 0-28.28 0L91 188H36V68h184ZM82 160a34 34 0 1 0-34-34 34 34 0 0 0 34 34Zm0-44a10 10 0 1 1-10 10 10 10 0 0 1 10-10Zm92 44a34 34 0 1 0-34-34 34 34 0 0 0 34 34Zm0-44a10 10 0 1 1-10 10 10 10 0 0 1 10-10Z"
  })]
});
var _default = exports.default = GoogleCardboardLogo;
