// phosphor-react-native/lib/commonjs/bold/GoogleDriveLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const GoogleDriveLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "google-drive-logo-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-google-drive-logo-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M241 141.65 172.59 29.51l-.07-.11a19.85 19.85 0 0 0-17-9.4h-55.08a19.85 19.85 0 0 0-17 9.4l-.07.11L15 141.65a20 20 0 0 0-.36 20.46L42 209.92A20 20 0 0 0 59.33 220h137.34A20 20 0 0 0 214 209.92l27.32-47.81a20 20 0 0 0-.32-20.46ZM211.88 140h-37.09L142 85.33l18.12-30.19ZM54.8 184l-11.44-20h23.45Zm40-20h66.4l19.2 32H75.6Zm14.4-24 18.8-31.34L146.8 140Zm80 24h23.45l-11.45 20ZM128 62l-10.81-18h21.62Zm-32.11-6.86L114 85.33 81.21 140H44.12Z"
  })]
});
var _default = exports.default = GoogleDriveLogo;
