// phosphor-react-native/lib/commonjs/bold/Hockey.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Hockey = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "hockey-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-hockey-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M224 148h-91.25L41.14 40.23a12 12 0 0 0-18.28 15.54l95.16 112 .06.07L156.46 213a20 20 0 0 0 15.24 7H224a20 20 0 0 0 20-20v-32a20 20 0 0 0-20-20Zm-70.85 24H180v24h-6.45ZM220 196h-16v-24h16Zm-66.34-83.77 61.2-72a12 12 0 0 1 18.28 15.54l-61.2 72a12 12 0 1 1-18.28-15.54Zm-44.14 70.45a12 12 0 0 0-16.91 1.37L82.45 196H76v-24.07A12 12 0 0 0 74.7 148H32a20 20 0 0 0-20 20v32a20 20 0 0 0 20 20h52.3a20 20 0 0 0 15.24-7l11.35-13.36a12 12 0 0 0-1.37-16.96ZM36 172h16v24H36Z"
  })]
});
var _default = exports.default = Hockey;
