// phosphor-react-native/lib/commonjs/bold/Hoodie.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Hoodie = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "hoodie-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-hoodie-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "m244.64 118.31-54.27-81.4A20 20 0 0 0 173.73 28H82.27a20 20 0 0 0-16.64 8.91l-54.27 81.4a20 20 0 0 0-2.65 16.36l21.43 78.59A20 20 0 0 0 49.43 228H76a20 20 0 0 0 20-20v-12h64v12a20 20 0 0 0 20 20h26.57a20 20 0 0 0 19.29-14.74l21.43-78.59a20 20 0 0 0-2.65-16.36ZM84 172V70.64l12 7.5V136a12 12 0 0 0 24 0V93.15l1.64 1a12 12 0 0 0 12.72 0l1.64-1V128a12 12 0 0 0 24 0V78.14l12-7.5V172Zm44-102.15L99.46 52h57.08ZM52.49 204l-20.15-73.89L60 88.62V176a20 20 0 0 0 12 18.32V204Zm151 0H184v-9.68A20 20 0 0 0 196 176V88.62l27.66 41.49Z"
  })]
});
var _default = exports.default = Hoodie;
