// phosphor-react-native/lib/commonjs/bold/Horse.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Horse = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "horse-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-horse-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M144 104a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm92 25.53A108.3 108.3 0 0 1 130.18 236h-2.25A107.31 107.31 0 0 1 56 208.94a12 12 0 1 1 16-17.88 81.42 81.42 0 0 0 16.2 11.26l21.54-29.62c-20.4-7-41.91-3.33-54.43-1.21a35.79 35.79 0 0 1-35.25-14.43c-.16-.21-.31-.43-.45-.65l-13.78-22A12 12 0 0 1 9.43 118L108 53.51V32a12 12 0 0 1 12-12h8a108 108 0 0 1 108 109.53Zm-24-.33a84 84 0 0 0-80-85.11V60a12 12 0 0 1-5.43 10l-94.16 61.61 7.31 11.68a12 12 0 0 0 11.58 4.54c16.23-2.75 49.4-8.36 79.64 8A36 36 0 0 0 164 120a12 12 0 0 1 24 0 60.09 60.09 0 0 1-53.64 59.66l-22.45 30.87a87.57 87.57 0 0 0 17.8 1.45A84.29 84.29 0 0 0 212 129.2Z"
  })]
});
var _default = exports.default = Horse;
