// phosphor-react-native/lib/commonjs/bold/IceCream.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const IceCream = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "ice-cream-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-ice-cream-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M228 116a32.14 32.14 0 0 0-16.37-27.92 84 84 0 0 0-167.26 0 32 32 0 0 0 12.51 59.77l53.75 94.07a20 20 0 0 0 34.74 0l53.75-94.07A32 32 0 0 0 228 116Zm-169.61-7.84A12 12 0 0 0 68 96.4V96a60 60 0 0 1 120 0v.4a12 12 0 0 0 9.61 11.76A8 8 0 0 1 196 124H60a8 8 0 0 1-1.61-15.84Zm113 39.84-16.5 28.88-16.5-28.88Zm-86.78 0h26.14l30.32 53.06L128 223.94Z"
  })]
});
var _default = exports.default = IceCream;
