// phosphor-react-native/lib/commonjs/bold/Invoice.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Invoice = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "invoice-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-invoice-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M28 132a12 12 0 0 1 0-24h28a4 4 0 0 0 0-8H40a28 28 0 0 1-3.38-55.79A12 12 0 0 1 59.3 44H64a12 12 0 0 1 0 24H40a4 4 0 0 0 0 8h16a28 28 0 0 1 3.38 55.79 12 12 0 0 1-22.68.21Zm208-76v136a20 20 0 0 1-20 20H40a20 20 0 0 1-20-20v-28a12 12 0 0 1 24 0v24h112v-24H88a12 12 0 0 1 0-24h68v-24h-44a12 12 0 0 1 0-24h100V68H104a12 12 0 0 1 0-24h120a12 12 0 0 1 12 12Zm-56 84h32v-24h-32Zm32 48v-24h-32v24Z"
  })]
});
var _default = exports.default = Invoice;
