// phosphor-react-native/lib/commonjs/bold/LightningA.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const LightningA = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "lightning-a-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-lightning-a-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M179.76 110.75a12 12 0 0 0-6.85-8.56L126 81.12l12.9-54.35a12 12 0 0 0-20.48-10.92L23.19 118.7a12 12 0 0 0 3.89 19.1l47 21.08-12.96 54.34a12 12 0 0 0 20.49 10.93l95.19-102.86a12 12 0 0 0 2.96-10.54ZM95 174.31l4.64-19.54a12 12 0 0 0-6.76-13.72l-40.76-18.3L105 65.69l-4.64 19.54A12 12 0 0 0 107.08 99l40.77 18.3Zm147.7 36.32-36-72a12 12 0 0 0-21.47 0l-36 72a12 12 0 1 0 21.46 10.73l4.68-9.36h41.17l4.68 9.36a12 12 0 1 0 21.47-10.73ZM187.41 188l8.59-17.17 8.58 17.17Z"
  })]
});
var _default = exports.default = LightningA;
