// phosphor-react-native/lib/commonjs/bold/LinuxLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const LinuxLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "linux-logo-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-linux-logo-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M231.49 217.38a12 12 0 0 1-16.89-1.9C213.19 213.72 180 171.24 180 88a52 52 0 1 0-104 0c0 83.24-33.21 125.72-34.62 127.48a12 12 0 0 1-18.78-14.95C23 200 52 162.06 52 88a76 76 0 1 1 152 0c0 74.19 29.1 112.16 29.4 112.54a12 12 0 0 1-1.91 16.84ZM104 120a16 16 0 1 0-16-16 16 16 0 0 0 16 16Zm64-16a16 16 0 1 0-16 16 16 16 0 0 0 16-16Zm-72.74 51 28 12a12 12 0 0 0 9.45 0l28-12a12 12 0 0 0-9.45-22l-23.27 10-23.27-10a12 12 0 0 0-9.46 22ZM128 184a57.12 57.12 0 0 0-38.66 15.15 12 12 0 0 0 16.23 17.69 32.86 32.86 0 0 1 44.85 0 12 12 0 1 0 16.23-17.69A57.1 57.1 0 0 0 128 184Z"
  })]
});
var _default = exports.default = LinuxLogo;
