// phosphor-react-native/lib/commonjs/bold/MicrosoftTeamsLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const MicrosoftTeamsLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "microsoft-teams-logo-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-microsoft-teams-logo-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M116 104a12 12 0 0 1-12 12h-4v36a12 12 0 0 1-24 0v-36h-4a12 12 0 0 1 0-24h32a12 12 0 0 1 12 12Zm124-11.26V152a44 44 0 0 1-42 44 68 68 0 0 1-122 4H36a20 20 0 0 1-20-20V76a20 20 0 0 1 20-20h56a44 44 0 0 1 80.51-24.53A36 36 0 0 1 221.94 76h1.32A16.76 16.76 0 0 1 240 92.74Zm-60-37.65V56a44.14 44.14 0 0 1-4.77 20H188a12 12 0 1 0-8-20.91ZM116 56h24a19.93 19.93 0 0 1 14.86 6.66A20.25 20.25 0 0 0 156 56a20 20 0 0 0-40 0ZM40 176h96V80H40Zm140-8v-68h-20v80a20 20 0 0 1-20 20h-34.14A44 44 0 0 0 180 168Zm36-68h-12v68c0 .79 0 1.57-.06 2.35A20 20 0 0 0 216 152Z"
  })]
});
var _default = exports.default = MicrosoftTeamsLogo;
