// phosphor-react-native/lib/commonjs/bold/Mosque.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Mosque = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "mosque-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-mosque-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M220 124a32 32 0 0 0-8.06 1c-1.29-41.54-29.35-62.32-50-77.58C150.18 38.73 140 31.21 140 24a12 12 0 0 0-24 0c0 7.21-10.18 14.73-22 23.45-20.63 15.26-48.69 36-50 77.58A32 32 0 0 0 4 156v52a12 12 0 0 0 12 12h224a12 12 0 0 0 12-12v-52a32 32 0 0 0-32-32ZM44 196H28v-40a8 8 0 0 1 16 0Zm64.31-129.26c7-5.14 14-10.36 19.69-16.27 5.68 5.91 12.74 11.13 19.69 16.27C164.8 79.4 182.4 92.44 186.9 116H69.1c4.5-23.56 22.1-36.6 39.21-49.26ZM188 196h-24v-28a12 12 0 0 0-24 0v28h-24v-28a12 12 0 0 0-24 0v28H68v-56h120Zm40 0h-16v-40a8 8 0 0 1 16 0Z"
  })]
});
var _default = exports.default = Mosque;
