// phosphor-react-native/lib/commonjs/bold/NyTimesLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const NyTimesLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "ny-times-logo-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-ny-times-logo-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M192 148a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm-4-40a11.84 11.84 0 0 1-5.07-1.13l-64.68-30.18a68 68 0 0 0-57.83 59.89l65.93-35.17A12 12 0 0 1 144 112v98.11a67.71 67.71 0 0 0 44.75-35.52 12 12 0 1 1 21.42 10.82 91.54 91.54 0 0 1-76 50.38 12.13 12.13 0 0 1-2.17.21c-.35 0-.68 0-1-.05-1 0-2 .05-3 .05a92 92 0 0 1-81.64-134.39A40 40 0 0 1 68 28a11.84 11.84 0 0 1 5.07 1.13l117.24 54.7A16 16 0 0 0 188 52a12 12 0 0 1 0-24 40 40 0 0 1 0 80ZM80 192.12v-38.79l-17.39 9.28A68.13 68.13 0 0 0 80 192.12ZM120 132l-16 8.53v67.08a67.1 67.1 0 0 0 16 3.9ZM86.5 61.88l-20.81-9.71a16 16 0 0 0-5.58 29.74A92 92 0 0 1 86.5 61.88Z"
  })]
});
var _default = exports.default = NyTimesLogo;
