// phosphor-react-native/lib/commonjs/bold/Onigiri.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Onigiri = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "onigiri-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-onigiri-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "m235 144.51-55.87-95.86a2.75 2.75 0 0 0-.17-.29 60 60 0 0 0-101.96 0 2.75 2.75 0 0 0-.17.29L21 144.51A60 60 0 0 0 72.1 236h111.8a60 60 0 0 0 51.1-91.49ZM156 212h-56v-40h56Zm59.36-18.5A35.55 35.55 0 0 1 183.9 212H180v-44a20 20 0 0 0-20-20H96a20 20 0 0 0-20 20v44h-3.9a36 36 0 0 1-30.58-55l.17-.28 55.82-95.84a36 36 0 0 1 61 0l55.82 95.85.17.28a35.58 35.58 0 0 1 .85 36.49Z"
  })]
});
var _default = exports.default = Onigiri;
