// phosphor-react-native/lib/commonjs/bold/Parachute.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Parachute = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "parachute-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-parachute-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M236 120A108.12 108.12 0 0 0 128 12 108.12 108.12 0 0 0 20 120a12 12 0 0 0 3.85 8.79l.22.2c.17.15.35.31.54.45l.19.16L116 198v14h-4a12 12 0 0 0 0 24h32a12 12 0 0 0 0-24h-4v-14l91.2-68.4a12 12 0 0 0 4.8-9.6Zm-24.85-12h-31.52c-1.79-29.27-10-50.53-18.93-65.36A84.27 84.27 0 0 1 211.15 108Zm-110.73 0C103.15 68.76 119 48.11 128 39.38c9 8.73 24.85 29.38 27.58 68.62ZM95.3 42.64c-9 14.83-17.14 36.09-18.93 65.36H44.85A84.27 84.27 0 0 1 95.3 42.64ZM68 132h48v36Zm72 0h48l-48 36Z"
  })]
});
var _default = exports.default = Parachute;
