// phosphor-react-native/lib/commonjs/bold/PencilCircle.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PencilCircle = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "pencil-circle-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-pencil-circle-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M204.37 51.6A108.08 108.08 0 1 0 236 128a108.09 108.09 0 0 0-31.63-76.4ZM92 200a12 12 0 1 1 24 0v11.11a83.78 83.78 0 0 1-24-7.22Zm48 0a12 12 0 1 1 24 0v3.89a83.78 83.78 0 0 1-24 7.22Zm-33.86-52h43.72l7.57 16.42a35.95 35.95 0 0 0-29.43 8.8 35.95 35.95 0 0 0-29.43-8.79Zm11.08-24L128 100.62 138.78 124ZM188 186.79V176a12.15 12.15 0 0 0-1.1-5l-48-104a12 12 0 0 0-21.8 0l-48 104a12.15 12.15 0 0 0-1.1 5v10.77a84 84 0 1 1 120 0Z"
  })]
});
var _default = exports.default = PencilCircle;
