// phosphor-react-native/lib/commonjs/bold/PenNib.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PenNib = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "pen-nib-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-pen-nib-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M252 92.68a19.86 19.86 0 0 0-5.86-14.14L177.46 9.85a20 20 0 0 0-28.29 0l-27.82 27.82-57.24 21.47A20 20 0 0 0 51.4 74.58L28.16 214A12 12 0 0 0 40 228a11.9 11.9 0 0 0 2-.16l139.45-23.25a20.07 20.07 0 0 0 15.44-12.7l21.46-57.25 27.82-27.82A19.85 19.85 0 0 0 252 92.68Zm-76.8 88.62L75 198l33-33a34 34 0 1 0-17-17l-33 33L74.7 80.8 125 61.94 194.05 131ZM112 134a10 10 0 1 1 10 10 10 10 0 0 1-10-10Zm96-23-63-63 18.34-18.34 63 63Z"
  })]
});
var _default = exports.default = PenNib;
