// phosphor-react-native/lib/commonjs/bold/Pentagram.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Pentagram = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "pentagram-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-pentagram-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M243 89.83A19.94 19.94 0 0 0 224 76h-58.15l-18.8-57.95A19.93 19.93 0 0 0 109 18L90.15 76H32a20 20 0 0 0-11.7 36.22l47.1 33.89-18.07 55.7a20 20 0 0 0 30.74 22.39L128 189.71l47.89 34.45a20 20 0 0 0 30.78-22.35l-18.07-55.7 47.15-33.92A20 20 0 0 0 243 89.83ZM128 37.11 140.62 76h-25.24ZM44.42 100h37.94l-7.18 22.13Zm31.51 97.6 11.93-36.78 19.59 14.1Zm19.71-60.75 12-36.85h40.82l12 36.85L128 160.14Zm52.91 38.07 19.59-14.1 11.93 36.78Zm32.27-52.79L173.64 100h37.94Z"
  })]
});
var _default = exports.default = Pentagram;
