// phosphor-react-native/lib/commonjs/bold/PersonSimpleThrow.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PersonSimpleThrow = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "person-simple-throw-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-person-simple-throw-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 92a36 36 0 1 0-36-36 36 36 0 0 0 36 36Zm0-48a12 12 0 1 1-12 12 12 12 0 0 1 12-12ZM44 96a20 20 0 1 1 20 20 20 20 0 0 1-20-20Zm184 17.6a12 12 0 0 1-19.51 9.4s-21-15.48-53.34.28c-.34 4.75-1 10.94-2.13 18l30.66 25.55a12 12 0 0 1 3.7 13l-16 48a12 12 0 1 1-22.76-7.59L162 180l-15.44-12.86a136.39 136.39 0 0 1-5.72 14.23c-14.64 31-39.34 51.93-73.42 62.15A11.82 11.82 0 0 1 64 244a12 12 0 0 1-3.45-23.5c37.84-11.35 60.77-38.71 68.41-81.44-24 13.54-43.11 17-56.46 17a65.87 65.87 0 0 1-21.5-3.75 12 12 0 0 1 9.72-21.93c.43.17 25.42 9.53 68.5-19.76 54.71-37.18 94.26-6.38 94.26-6.38a12 12 0 0 1 4.52 9.36Z"
  })]
});
var _default = exports.default = PersonSimpleThrow;
