// phosphor-react-native/lib/commonjs/bold/Pipe.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Pipe = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "pipe-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-pipe-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M232 100h-16V60h16a12 12 0 0 0 0-24h-20a20 20 0 0 0-16-8h-24a20 20 0 0 0-16 8h-12A108.12 108.12 0 0 0 36 144v12a20 20 0 0 0-8 16v24a20 20 0 0 0 8 16v20a12 12 0 0 0 24 0v-16h40v16a12 12 0 0 0 24 0v-20a20 20 0 0 0 8-16v-24a20 20 0 0 0-8-16v-12a20 20 0 0 1 20-20h12a20 20 0 0 0 16 8h24a20 20 0 0 0 16-8h20a12 12 0 0 0 0-24Zm-124 76v16H52v-16Zm-8-32v8H60v-8a84.09 84.09 0 0 1 84-84h8v40h-8a44.05 44.05 0 0 0-44 44Zm76-92h16v56h-16Z"
  })]
});
var _default = exports.default = Pipe;
