// phosphor-react-native/lib/commonjs/bold/Pizza.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Pizza = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "pizza-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-pizza-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M243.43 62.05a19.93 19.93 0 0 0-9.06-12.38 205.51 205.51 0 0 0-212.74 0 20 20 0 0 0-6.7 27.48l96 157.26a20 20 0 0 0 34.2 0l96-157.26a19.82 19.82 0 0 0 2.3-15.1ZM128 44a180.93 180.93 0 0 1 90.45 24.14L210 82a164.15 164.15 0 0 0-164 0l-8.45-13.86A180.93 180.93 0 0 1 128 44ZM83.88 144.06l-20.74-34A20 20 0 0 1 92 128a20.12 20.12 0 0 1-8.12 16.06Zm44.12 72.3-31.56-51.71a44 44 0 0 0-4.64-75.93 140.45 140.45 0 0 1 105.71 13.74l-8.95 14.66a44 44 0 0 0-45.75 75ZM148 156a20 20 0 0 1 28-18.32l-20.64 33.83A20 20 0 0 1 148 156Z"
  })]
});
var _default = exports.default = Pizza;
