// phosphor-react-native/lib/commonjs/bold/PlugCharging.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const PlugCharging = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "plug-charging-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-plug-charging-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M224 48h-44V16a12 12 0 0 0-24 0v32h-56V16a12 12 0 0 0-24 0v32H32.55C24.4 48 20 54.18 20 60a12 12 0 0 0 12 12h12v92a44.05 44.05 0 0 0 44 44h28v32a12 12 0 0 0 24 0v-32h28a44.05 44.05 0 0 0 44-44V72h12a12 12 0 0 0 0-24Zm-36 116a20 20 0 0 1-20 20H88a20 20 0 0 1-20-20V72h120Zm-85.86-29.17a12 12 0 0 1-1.38-11l12-32a12 12 0 1 1 22.48 8.42L129.32 116H144a12 12 0 0 1 11.24 16.21l-12 32a12 12 0 0 1-22.48-8.42l5.92-15.79H112a12 12 0 0 1-9.86-5.17Z"
  })]
});
var _default = exports.default = PlugCharging;
