// phosphor-react-native/lib/commonjs/bold/Radioactive.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Radioactive = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "radioactive-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-radioactive-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 112a16 16 0 1 1-16 16 16 16 0 0 1 16-16Zm-32 16a12 12 0 0 0-12-12H44.47a83.88 83.88 0 0 1 31.2-54.11L95.38 96a12 12 0 1 0 20.78-12l-22-38a20 20 0 0 0-28.87-6.34 107.85 107.85 0 0 0-45.26 78.4 20.22 20.22 0 0 0 5.21 15.43A20 20 0 0 0 40 140h44a12 12 0 0 0 12-12Zm139.92-9.94a107.81 107.81 0 0 0-45.26-78.39A20 20 0 0 0 161.79 46l-21.95 38a12 12 0 0 0 20.78 12l19.71-34.13a83.88 83.88 0 0 1 31.2 54.13H172a12 12 0 0 0 0 24h44a20 20 0 0 0 14.71-6.51 20.22 20.22 0 0 0 5.21-15.43Zm-75.52 42.05a12 12 0 0 0-20.79 12L159.19 206a84.33 84.33 0 0 1-62.38 0l19.58-33.91a12 12 0 0 0-20.79-12l-21.82 37.8a20 20 0 0 0 8.91 28.15 108.24 108.24 0 0 0 90.62 0 20 20 0 0 0 8.91-28.15Z"
  })]
});
var _default = exports.default = Radioactive;
