// phosphor-react-native/lib/commonjs/bold/Resize.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Resize = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "resize-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-resize-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M136 108H48a12 12 0 0 0-12 12v88a12 12 0 0 0 12 12h88a12 12 0 0 0 12-12v-88a12 12 0 0 0-12-12Zm-12 88H60v-64h64Zm96-8v12a20 20 0 0 1-20 20h-20a12 12 0 0 1 0-24h16v-8a12 12 0 0 1 24 0Zm0-72v24a12 12 0 0 1-24 0v-24a12 12 0 0 1 24 0Zm0-60v16a12 12 0 0 1-24 0V60h-12a12 12 0 0 1 0-24h16a20 20 0 0 1 20 20Zm-68-8a12 12 0 0 1-12 12h-24a12 12 0 0 1 0-24h24a12 12 0 0 1 12 12ZM36 76V56a20 20 0 0 1 20-20h12a12 12 0 0 1 0 24h-8v16a12 12 0 0 1-24 0Z"
  })]
});
var _default = exports.default = Resize;
