// phosphor-react-native/lib/commonjs/bold/SelectionForeground.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SelectionForeground = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "selection-foreground-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-selection-foreground-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M68 216a12 12 0 0 1-12 12h-8a20 20 0 0 1-20-20v-8a12 12 0 0 1 24 0v4h4a12 12 0 0 1 12 12Zm44-12H96a12 12 0 0 0 0 24h16a12 12 0 0 0 0-24Zm-72-32a12 12 0 0 0 12-12v-16a12 12 0 0 0-24 0v16a12 12 0 0 0 12 12Zm128 16a12 12 0 0 0-12 12v4h-4a12 12 0 0 0 0 24h8a20 20 0 0 0 20-20v-8a12 12 0 0 0-12-12Zm0-72a12 12 0 0 0 12-12v-8a20 20 0 0 0-20-20h-8a12 12 0 0 0 0 24h4v4a12 12 0 0 0 12 12ZM56 76h-8a20 20 0 0 0-20 20v8a12 12 0 0 0 24 0v-4h4a12 12 0 0 0 0-24Zm152-48H100a20 20 0 0 0-20 20v40a5.47 5.47 0 0 0 0 .6A12 12 0 0 0 92 100h20a12 12 0 0 0 0-24h-8V52h100v100h-24v-8a12 12 0 0 0-24 0v20a12 12 0 0 0 12 12h40a20 20 0 0 0 20-20V48a20 20 0 0 0-20-20Z"
  })]
});
var _default = exports.default = SelectionForeground;
