// phosphor-react-native/lib/commonjs/bold/ShippingContainer.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const ShippingContainer = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "shipping-container-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-shipping-container-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M237.5 66.8 131.3 36.46a12.07 12.07 0 0 0-5-.34l-105.13 15A20.1 20.1 0 0 0 4 70.94v114.12a20.1 20.1 0 0 0 17.17 19.8l105.13 15a12.15 12.15 0 0 0 1.7.12 12 12 0 0 0 3.3-.46l106.2-30.32A20.09 20.09 0 0 0 252 170V86a20.08 20.08 0 0 0-14.5-19.2ZM92 116h-8V66.41l32-4.57v132.32l-32-4.57V140h8a12 12 0 0 0 0-24ZM28 74.41l32-4.57V116h-8a12 12 0 0 0 0 24h8v46.16l-32-4.57ZM228 167l-88 25.14V63.91l88 25.14Z"
  })]
});
var _default = exports.default = ShippingContainer;
