// phosphor-react-native/lib/commonjs/bold/Shovel.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Shovel = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "shovel-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-shovel-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "m248.49 55.51-48-48a12 12 0 0 0-17 17L199 40l-65.37 65.37-31.49-31.51a20 20 0 0 0-28.28 0l-56 56A19.9 19.9 0 0 0 12 144v80a20 20 0 0 0 20 20h80a19.86 19.86 0 0 0 14.14-5.86l56-56a20 20 0 0 0 0-28.28l-31.51-31.52L216 57l15.51 15.52a12 12 0 1 0 17-17ZM162.34 168l-52 52H36v-74.34l52-52 28.69 28.68-37.18 37.17a12 12 0 0 0 17 17l37.17-37.18Z"
  })]
});
var _default = exports.default = Shovel;
