// phosphor-react-native/lib/commonjs/bold/SmileyNervous.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SmileyNervous = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "smiley-nervous-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-smiley-nervous-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 20a108 108 0 1 0 108 108A108.12 108.12 0 0 0 128 20Zm0 192a84 84 0 1 1 84-84 84.09 84.09 0 0 1-84 84ZM76 108a16 16 0 1 1 16 16 16 16 0 0 1-16-16Zm72 0a16 16 0 1 1 16 16 16 16 0 0 1-16-16Zm30.92 64.94a12 12 0 0 1-16.78 2.52 38.18 38.18 0 0 1-7.74-8.26 35.12 35.12 0 0 0-2.4-3 35.12 35.12 0 0 0-2.4 3C146 172 140 180 128 180s-18-8-21.6-12.8a35.12 35.12 0 0 0-2.4-3 35.12 35.12 0 0 0-2.4 3 38.18 38.18 0 0 1-7.74 8.26 12 12 0 0 1-14.27-19.3 22.82 22.82 0 0 0 2.81-3.36C86 148 92 140 104 140s18 8 21.6 12.8a35.12 35.12 0 0 0 2.4 3 35.12 35.12 0 0 0 2.4-3C134 148 140 140 152 140s18 8 21.6 12.8a22.82 22.82 0 0 0 2.81 3.36 12 12 0 0 1 2.51 16.78Z"
  })]
});
var _default = exports.default = SmileyNervous;
