// phosphor-react-native/lib/commonjs/bold/SolarPanel.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SolarPanel = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "solar-panel-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-solar-panel-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M32 100a12 12 0 0 1 12-12h8a12 12 0 0 1 0 24h-8a12 12 0 0 1-12-12Zm96-64a12 12 0 0 0 12-12v-8a12 12 0 0 0-24 0v8a12 12 0 0 0 12 12ZM65.77 54.74a12 12 0 1 0 17-17l-5.66-5.65a12 12 0 0 0-17 17Zm116 3.52a12 12 0 0 0 8.49-3.52l5.65-5.65a12 12 0 0 0-17-17l-5.66 5.65a12 12 0 0 0 8.49 20.49ZM192 100a12 12 0 0 0 12 12h8a12 12 0 0 0 0-24h-8a12 12 0 0 0-12 12ZM88 112a12 12 0 0 0 12-12 28 28 0 0 1 56 0 12 12 0 0 0 24 0 52 52 0 0 0-104 0 12 12 0 0 0 12 12Zm154.36 110.05A12 12 0 0 1 232 228H24a12 12 0 0 1-10.45-17.9l43-76A12 12 0 0 1 67 128h122a12 12 0 0 1 10.45 6.1l43 76a12 12 0 0 1-.09 11.95ZM182 152h-42v12h48.82Zm-108 0-6.78 12H116v-12Zm-29.43 52H116v-16H53.61Zm166.86 0-9-16H140v16Z"
  })]
});
var _default = exports.default = SolarPanel;
