// phosphor-react-native/lib/commonjs/bold/SprayBottle.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SprayBottle = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "spray-bottle-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-spray-bottle-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M200 84a12 12 0 0 0 12-12 60.07 60.07 0 0 0-60-60H80a20 20 0 0 0-20 20v48a20 20 0 0 1-20 20 12 12 0 0 0 0 24 44.06 44.06 0 0 0 43.82-40H108v20.62a19.92 19.92 0 0 1-7.51 15.62L84.51 133A43.8 43.8 0 0 0 68 167.38V224a20 20 0 0 0 20 20h104a20 20 0 0 0 20-20v-12.53A274.77 274.77 0 0 0 180.68 84ZM84 36h68a36 36 0 0 1 33.94 24H84Zm104 175.47V220H92v-52.62a19.92 19.92 0 0 1 7.51-15.62l16-12.78A43.8 43.8 0 0 0 132 104.62V84h21.24A250.93 250.93 0 0 1 188 211.47Z"
  })]
});
var _default = exports.default = SprayBottle;
