// phosphor-react-native/lib/commonjs/bold/SuitcaseRolling.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SuitcaseRolling = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "suitcase-rolling-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-suitcase-rolling-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M116 96v80a12 12 0 0 1-24 0V96a12 12 0 0 1 24 0Zm24 0v80a12 12 0 0 0 24 0V96a12 12 0 0 0-24 0Zm72-32v144a20 20 0 0 1-20 20h-12v12a12 12 0 0 1-24 0v-12h-56v12a12 12 0 0 1-24 0v-12H64a20 20 0 0 1-20-20V64a20 20 0 0 1 20-20h20V28a28 28 0 0 1 28-28h32a28 28 0 0 1 28 28v16h20a20 20 0 0 1 20 20ZM108 44h40V28a4 4 0 0 0-4-4h-32a4 4 0 0 0-4 4Zm80 24H68v136h120Z"
  })]
});
var _default = exports.default = SuitcaseRolling;
