// phosphor-react-native/lib/commonjs/bold/SwimmingPool.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const SwimmingPool = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "swimming-pool-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-swimming-pool-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M88 149.39a12 12 0 0 0 12-12V136h56v3.29a12 12 0 0 0 24 0V32a12 12 0 0 0-24 0v8h-56v-8a12 12 0 0 0-24 0v105.39a12 12 0 0 0 12 12ZM100 112v-12h56v12Zm56-48v12h-56V64ZM20 164a12 12 0 0 1 12-12c15.63 0 24 5.55 30.66 10 5.4 3.6 9 6 17.34 6s11.93-2.38 17.34-6c6.69-4.47 15-10 30.65-10s24 5.55 30.66 10c5.41 3.6 9 6 17.35 6s11.94-2.38 17.34-6c6.7-4.47 15-10 30.66-10a12 12 0 0 1 0 24c-8.37 0-11.94 2.38-17.35 6-6.69 4.47-15 10-30.65 10s-24-5.55-30.66-10c-5.41-3.6-9-6-17.35-6s-11.93 2.38-17.34 6c-6.69 4.47-15 10-30.65 10s-24-5.55-30.66-10c-5.4-3.6-9-6-17.34-6a12 12 0 0 1-12-12Zm216 44a12 12 0 0 1-12 12c-8.37 0-11.94 2.38-17.35 6-6.69 4.47-15 10-30.65 10s-24-5.55-30.66-10c-5.41-3.6-9-6-17.35-6s-11.93 2.38-17.34 6c-6.69 4.47-15 10-30.65 10s-24-5.55-30.66-10c-5.4-3.6-9-6-17.34-6a12 12 0 0 1 0-24c15.63 0 24 5.55 30.66 10 5.4 3.6 9 6 17.34 6s11.93-2.38 17.34-6c6.69-4.47 15-10 30.65-10s24 5.55 30.66 10c5.41 3.6 9 6 17.35 6s11.94-2.38 17.34-6c6.7-4.47 15-10 30.66-10a12 12 0 0 1 12 12Z"
  })]
});
var _default = exports.default = SwimmingPool;
