// phosphor-react-native/lib/commonjs/bold/TriangleDashed.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const TriangleDashed = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "triangle-dashed-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-triangle-dashed-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M164 216a12 12 0 0 1-12 12h-48a12 12 0 0 1 0-24h48a12 12 0 0 1 12 12Zm76.26-29.9-4.66-8.1a12 12 0 1 0-20.79 12l4.65 8.08a3.56 3.56 0 0 1 0 3.73 4.46 4.46 0 0 1-4 2.2H192a12 12 0 0 0 0 24h23.46a28.31 28.31 0 0 0 24.79-14.19 27.47 27.47 0 0 0 .01-27.72ZM64 204H40.55a4.46 4.46 0 0 1-4-2.2 3.56 3.56 0 0 1 0-3.73l4.64-8.07a12 12 0 1 0-20.79-12l-4.66 8.09a27.47 27.47 0 0 0 0 27.71A28.31 28.31 0 0 0 40.55 228H64a12 12 0 0 0 0-24Zm138.17-48a12 12 0 0 0 10.39-18l-23-40a12 12 0 0 0-20.8 12l23 40a12 12 0 0 0 10.42 6ZM166.5 58l-13.69-23.77a28.74 28.74 0 0 0-49.62 0L89.5 58a12 12 0 1 0 20.79 12L124 46.2a4.75 4.75 0 0 1 8 0L145.71 70a12 12 0 0 0 10.41 6 11.87 11.87 0 0 0 6-1.6A12 12 0 0 0 166.5 58ZM82.85 93.6A12 12 0 0 0 66.46 98l-23 40a12 12 0 0 0 20.8 12l23-40a12 12 0 0 0-4.41-16.4Z"
  })]
});
var _default = exports.default = TriangleDashed;
