// phosphor-react-native/lib/commonjs/bold/TrolleySuitcase.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const TrolleySuitcase = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "trolley-suitcase-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-trolley-suitcase-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M96 156h120a20 20 0 0 0 20-20V76a20 20 0 0 0-20-20h-24V40a20 20 0 0 0-20-20h-32a20 20 0 0 0-20 20v16H96a20 20 0 0 0-20 20v60a20 20 0 0 0 20 20Zm48-112h24v12h-24Zm-44 36h112v52H100Zm-8 144a20 20 0 1 1-20-20 20 20 0 0 1 20 20Zm144 0a20 20 0 1 1-20-20 20 20 0 0 1 20 20Zm16-44a12 12 0 0 1-12 12H32a12 12 0 0 1 0-24h4V77L15.51 56.49a12 12 0 0 1 17-17l21.63 21.68A19.86 19.86 0 0 1 60 75.31V168h180a12 12 0 0 1 12 12Z"
  })]
});
var _default = exports.default = TrolleySuitcase;
