// phosphor-react-native/lib/commonjs/bold/TumblrLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const TumblrLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "tumblr-logo-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-tumblr-logo-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M192 124a12 12 0 0 0 12-12V72a12 12 0 0 0-12-12h-36V24a12 12 0 0 0-12-12h-32a12 12 0 0 0-12 12 36 36 0 0 1-36 36 12 12 0 0 0-12 12v40a12 12 0 0 0 12 12h20v52a68.07 68.07 0 0 0 68 68h40a12 12 0 0 0 12-12v-48a12 12 0 0 0-12-12h-32a4 4 0 0 1-4-4v-44Zm-32 72h20v24h-28a44.05 44.05 0 0 1-44-44v-64a12 12 0 0 0-12-12H76V82.79A60.18 60.18 0 0 0 122.79 36H132v36a12 12 0 0 0 12 12h36v16h-36a12 12 0 0 0-12 12v56a28 28 0 0 0 28 28Z"
  })]
});
var _default = exports.default = TumblrLogo;
