// phosphor-react-native/lib/commonjs/bold/UsersThree.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const UsersThree = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "users-three-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-users-three-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M164.38 181.1a52 52 0 1 0-72.76 0 75.89 75.89 0 0 0-30 28.89 12 12 0 0 0 20.78 12 53 53 0 0 1 91.22 0 12 12 0 1 0 20.78-12 75.89 75.89 0 0 0-30.02-28.89ZM100 144a28 28 0 1 1 28 28 28 28 0 0 1-28-28Zm147.21 9.59a12 12 0 0 1-16.81-2.39c-8.33-11.09-19.85-19.59-29.33-21.64a12 12 0 0 1-1.82-22.91 20 20 0 1 0-24.78-28.3 12 12 0 1 1-21-11.6 44 44 0 1 1 73.28 48.35 92.18 92.18 0 0 1 22.85 21.69 12 12 0 0 1-2.39 16.8Zm-192.28-24c-9.48 2.05-21 10.55-29.33 21.65a12 12 0 0 1-19.19-14.45 92.37 92.37 0 0 1 22.85-21.69 44 44 0 1 1 73.28-48.35 12 12 0 1 1-21 11.6 20 20 0 1 0-24.78 28.3 12 12 0 0 1-1.82 22.91Z"
  })]
});
var _default = exports.default = UsersThree;
