// phosphor-react-native/lib/commonjs/bold/Virus.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Virus = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "virus-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-virus-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M108 128a20 20 0 1 1 20-20 20 20 0 0 1-20 20Zm60-20a20 20 0 1 0 20 20 20 20 0 0 0-20-20Zm-40 40a20 20 0 1 0 20 20 20 20 0 0 0-20-20Zm124-20a12 12 0 0 1-12 12h-12.72a99.62 99.62 0 0 1-20.62 49.69l9.83 9.82a12 12 0 0 1-17 17l-9.82-9.83a99.62 99.62 0 0 1-49.67 20.6V240a12 12 0 0 1-24 0v-12.72a99.62 99.62 0 0 1-49.69-20.62l-9.82 9.83a12 12 0 0 1-17-17l9.83-9.82A99.62 99.62 0 0 1 28.72 140H16a12 12 0 0 1 0-24h12.72a99.62 99.62 0 0 1 20.62-49.69l-9.83-9.82a12 12 0 0 1 17-17l9.82 9.83A99.62 99.62 0 0 1 116 28.72V16a12 12 0 0 1 24 0v12.72a99.62 99.62 0 0 1 49.69 20.62l9.82-9.83a12 12 0 0 1 17 17l-9.83 9.82a99.62 99.62 0 0 1 20.6 49.67H240a12 12 0 0 1 12 12Zm-48 0a76 76 0 1 0-76 76 76.08 76.08 0 0 0 76-76Z"
  })]
});
var _default = exports.default = Virus;
