// phosphor-react-native/lib/commonjs/bold/Visor.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Visor = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "visor-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-visor-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M188 104a12 12 0 0 1-12 12H80a12 12 0 0 1 0-24h96a12 12 0 0 1 12 12Zm67.91 27.63A76.29 76.29 0 0 1 182.43 204h-1.69c-14.44 0-29.37-5.78-45.58-17.64a12.08 12.08 0 0 0-14.32 0C104 198.68 88.56 204.44 73.57 204A76.29 76.29 0 0 1 .09 131.63 76 76 0 0 1 76 52h104a76 76 0 0 1 75.91 79.63ZM217.62 92.1A51.56 51.56 0 0 0 180 76H76a52 52 0 0 0-51.94 54.51A52.19 52.19 0 0 0 74.32 180c9.42.29 20-4 32.35-13a36 36 0 0 1 42.66 0c12.36 9 22.91 13.29 32.35 13a52.19 52.19 0 0 0 50.26-49.48 51.63 51.63 0 0 0-14.32-38.42Z"
  })]
});
var _default = exports.default = Visor;
