// phosphor-react-native/lib/commonjs/bold/Volleyball.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Volleyball = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "volleyball-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-volleyball-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M128 20a108 108 0 1 0 108 108A108.12 108.12 0 0 0 128 20ZM96.83 50a83.49 83.49 0 0 1 17.92-5 84 84 0 0 1 46.51 71h-26.33Zm38.1 90h76.2a83.37 83.37 0 0 1-4.69 18 84.07 84.07 0 0 1-84.68 4.79Zm76.2-24h-25.84a107.43 107.43 0 0 0-14.4-49.71 108.71 108.71 0 0 0-11.39-16.14A84.21 84.21 0 0 1 211.13 116ZM76.07 62.05 89 84.39a107.44 107.44 0 0 0-35.85 37.32 108.9 108.9 0 0 0-8.28 18 83.65 83.65 0 0 1 31.2-77.66ZM62.79 180.87A84 84 0 0 1 101 105.2l13.14 22.8-38.07 66a84.68 84.68 0 0 1-13.28-13.13Zm34 25.11 12.92-22.37A107.41 107.41 0 0 0 160 196a109 109 0 0 0 19.59-1.78A83.72 83.72 0 0 1 96.83 206Z"
  })]
});
var _default = exports.default = Volleyball;
