// phosphor-react-native/lib/commonjs/bold/Windmill.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var React = _interopRequireWildcard(require("react"));
var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));
var _jsxRuntime = require("react/jsx-runtime");
function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
/* GENERATED FILE */

const Windmill = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNativeSvg.default, {
  className: "windmill-bold__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-windmill-bold',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/(0, _jsxRuntime.jsx)("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeSvg.Path, {
    d: "M224 220h-37.59l-4.72-33a20 20 0 0 0 23.55-8.83l12-20.33a20 20 0 0 0-7.09-27.4l-31.57-18.58 38.66-65.68a20 20 0 0 0-7.08-27.39l-20.34-12a20 20 0 0 0-27.39 7.09l-18.59 31.55L78.15 6.77a20 20 0 0 0-27.39 7.09l-12 20.33a20 20 0 0 0 7.09 27.4l31.58 18.58-38.66 65.68a20 20 0 0 0 7.08 27.39l20.34 12a19.83 19.83 0 0 0 8 2.65L69.59 220H32a12 12 0 0 0 0 24h192a12 12 0 0 0 0-24Zm-37.41-57.47-62.26-36.64 7.93-13.46 62.26 36.65Zm-5.52-133 13.46 7.93-36.64 62.21-13.46-7.92Zm-111.66 0 62.26 36.64-7.93 13.46-62.26-36.7Zm-7.94 125.06 36.64-62.25 13.46 7.92-36.64 62.26Zm40.34 9.56 10.35-17.57L155.3 172l6.86 48H93.84Z"
  })]
});
var _default = exports.default = Windmill;
