// phosphor-react-native/lib/module/duotone/BracketsAngle.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const BracketsAngle = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "brackets-angle-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-brackets-angle-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "m232 128-56 88H80l-56-88 56-88h96Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/_jsx(Path, {
    d: "M86.75 44.3 33.48 128l53.27 83.7a8 8 0 0 1-2.46 11.05A7.91 7.91 0 0 1 80 224a8 8 0 0 1-6.76-3.71l-56-88a8 8 0 0 1 0-8.59l56-88a8 8 0 1 1 13.5 8.59Zm152 79.41-56-88a8 8 0 1 0-13.5 8.59l53.27 83.7-53.27 83.7a8 8 0 0 0 2.46 11.05A7.91 7.91 0 0 0 176 224a8 8 0 0 0 6.76-3.71l56-88a8 8 0 0 0-.01-8.58Z"
  })]
});
export default BracketsAngle;
