// phosphor-react-native/lib/module/duotone/Couch.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Couch = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "couch-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-couch-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M240 120v48a8 8 0 0 1-8 8H24a8 8 0 0 1-8-8v-48a8 8 0 0 1 8-8V72a8 8 0 0 1 8-8h192a8 8 0 0 1 8 8v40a8 8 0 0 1 8 8Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/_jsx(Path, {
    d: "M240 106.17V72a16 16 0 0 0-16-16H32a16 16 0 0 0-16 16v34.17A16 16 0 0 0 8 120v48a16 16 0 0 0 16 16h8v16a8 8 0 0 0 16 0v-16h160v16a8 8 0 0 0 16 0v-16h8a16 16 0 0 0 16-16v-48a16 16 0 0 0-8-13.83ZM224 104h-8a16 16 0 0 0-16 16v16h-64V72h88ZM120 72v64H56v-16a16 16 0 0 0-16-16h-8V72Zm112 96H24v-48h16v24a8 8 0 0 0 8 8h160a8 8 0 0 0 8-8v-24h16Z"
  })]
});
export default Couch;
