// phosphor-react-native/lib/module/duotone/DotsSix.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const DotsSix = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "dots-six-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-dots-six-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M240 64v128a16 16 0 0 1-16 16H32a16 16 0 0 1-16-16V64a16 16 0 0 1 16-16h192a16 16 0 0 1 16 16Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/_jsx(Path, {
    d: "M72 92a12 12 0 1 1-12-12 12 12 0 0 1 12 12Zm56-12a12 12 0 1 0 12 12 12 12 0 0 0-12-12Zm68 24a12 12 0 1 0-12-12 12 12 0 0 0 12 12ZM60 152a12 12 0 1 0 12 12 12 12 0 0 0-12-12Zm68 0a12 12 0 1 0 12 12 12 12 0 0 0-12-12Zm68 0a12 12 0 1 0 12 12 12 12 0 0 0-12-12Z"
  })]
});
export default DotsSix;
