// phosphor-react-native/lib/module/duotone/GreaterThanOrEqual.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const GreaterThanOrEqual = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "greater-than-or-equal-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-greater-than-or-equal-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M208 104 56 160V48Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/_jsx(Path, {
    d: "M53.24 152.49 184.86 104 53.24 55.51a8 8 0 1 1 5.53-15l152 56a8 8 0 0 1 0 15l-152 56A8.13 8.13 0 0 1 56 168a8 8 0 0 1-2.76-15.51ZM208 192H56a8 8 0 0 0 0 16h152a8 8 0 0 0 0-16Z"
  })]
});
export default GreaterThanOrEqual;
