// phosphor-react-native/lib/module/duotone/Hurricane.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Hurricane = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "hurricane-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-hurricane-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M148.62 50.68 160 8S48 32 48 128a80 80 0 0 0 59.38 77.32L96 248s112-24 112-120a80 80 0 0 0-59.38-77.32ZM128 152a24 24 0 1 1 24-24 24 24 0 0 1-24 24Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/_jsx(Path, {
    d: "M128 96a32 32 0 1 0 32 32 32 32 0 0 0-32-32Zm0 48a16 16 0 1 1 16-16 16 16 0 0 1-16 16Zm30.32-98.64 9.41-35.3a8 8 0 0 0-9.41-9.88 195.87 195.87 0 0 0-58.65 25.09C60.63 50.37 40 85.89 40 128a88.11 88.11 0 0 0 57.68 82.64l-9.41 35.3a8 8 0 0 0 9.41 9.88 195.87 195.87 0 0 0 58.65-25.09C195.37 205.63 216 170.11 216 128a88.1 88.1 0 0 0-57.68-82.64ZM148.06 217a184.14 184.14 0 0 1-40.68 19.37l7.73-29a8 8 0 0 0-5.67-9.79A72.06 72.06 0 0 1 56 128c0-36.77 17.48-66.72 51.94-89a184.14 184.14 0 0 1 40.68-19.37l-7.73 29a8 8 0 0 0 5.67 9.79A72.06 72.06 0 0 1 200 128c0 36.77-17.48 66.72-51.94 89Z"
  })]
});
export default Hurricane;
