// phosphor-react-native/lib/module/duotone/MapPinSimpleLine.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const MapPinSimpleLine = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "map-pin-simple-line-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-map-pin-simple-line-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M176 80a48 48 0 1 1-48-48 48 48 0 0 1 48 48Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/_jsx(Path, {
    d: "M216 208h-80v-72.58a56 56 0 1 0-16 0V208H40a8 8 0 0 0 0 16h176a8 8 0 0 0 0-16ZM88 80a40 40 0 1 1 40 40 40 40 0 0 1-40-40Z"
  })]
});
export default MapPinSimpleLine;
