// phosphor-react-native/lib/module/duotone/MusicNotesSimple.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const MusicNotesSimple = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "music-notes-simple-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-music-notes-simple-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M208 164a28 28 0 1 1-28-28 28 28 0 0 1 28 28Zm-156 4a28 28 0 1 0 28 28 28 28 0 0 0-28-28Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/_jsx(Path, {
    d: "M212.92 17.69a8 8 0 0 0-6.86-1.45l-128 32A8 8 0 0 0 72 56v110.08A36 36 0 1 0 88 196V62.25l112-28v99.83A36 36 0 1 0 216 164V24a8 8 0 0 0-3.08-6.31ZM52 216a20 20 0 1 1 20-20 20 20 0 0 1-20 20Zm128-32a20 20 0 1 1 20-20 20 20 0 0 1-20 20Z"
  })]
});
export default MusicNotesSimple;
