// phosphor-react-native/lib/module/duotone/NumberEight.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const NumberEight = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "number-eight-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-number-eight-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M216 40v176a16 16 0 0 1-16 16H56a16 16 0 0 1-16-16V40a16 16 0 0 1 16-16h144a16 16 0 0 1 16 16Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/_jsx(Path, {
    d: "M155.55 119.27a48 48 0 1 0-55.1 0 56 56 0 1 0 55.1 0ZM96 80a32 32 0 1 1 32 32 32 32 0 0 1-32-32Zm32 128a40 40 0 1 1 40-40 40 40 0 0 1-40 40Z"
  })]
});
export default NumberEight;
