// phosphor-react-native/lib/module/duotone/Percent.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Percent = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "percent-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-percent-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M95.8 56.2a28 28 0 1 1-39.6 0 28 28 0 0 1 39.6 0Zm104 104a28 28 0 1 0 0 39.6 28 28 0 0 0 0-39.6Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/_jsx(Path, {
    d: "m205.66 61.64-144 144a8 8 0 0 1-11.32-11.32l144-144a8 8 0 0 1 11.32 11.31Zm-155.12 39.8a36 36 0 0 1 50.92-50.91 36 36 0 0 1-50.92 50.91ZM56 76a20 20 0 1 0 34.14-14.16A20 20 0 0 0 56 76Zm160 104a36 36 0 1 1-10.54-25.46A35.76 35.76 0 0 1 216 180Zm-16 0a20 20 0 1 0-5.86 14.14A19.87 19.87 0 0 0 200 180Z"
  })]
});
export default Percent;
