// phosphor-react-native/lib/module/duotone/PhosphorLogo.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const PhosphorLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "phosphor-logo-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-phosphor-logo-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M216 104a64 64 0 0 1-64 64h-8V40h8a64 64 0 0 1 64 64ZM72 168h72L72 40Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/_jsx(Path, {
    d: "M152 32H72a8 8 0 0 0-8 8v128a80.09 80.09 0 0 0 80 80 8 8 0 0 0 8-8v-64a72 72 0 0 0 0-144ZM80 70.54 130.32 160H80Zm56 161A64.14 64.14 0 0 1 80.51 176H136Zm0-94L85.68 48H136ZM152 160V48a56 56 0 0 1 0 112Z"
  })]
});
export default PhosphorLogo;
