// phosphor-react-native/lib/module/duotone/SpotifyLogo.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const SpotifyLogo = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "spotify-logo-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-spotify-logo-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M224 128a96 96 0 1 1-96-96 96 96 0 0 1 96 96Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/_jsx(Path, {
    d: "M128 24a104 104 0 1 0 104 104A104.11 104.11 0 0 0 128 24Zm0 192a88 88 0 1 1 88-88 88.1 88.1 0 0 1-88 88Zm31.07-46.26a8 8 0 0 1-10.81 3.33 42.79 42.79 0 0 0-40.52 0 8 8 0 0 1-7.48-14.14 59.33 59.33 0 0 1 55.48 0 8 8 0 0 1 3.33 10.81Zm32-56a8 8 0 0 1-10.83 3.29 110.62 110.62 0 0 0-104.46 0 8 8 0 0 1-7.54-14.12 126.67 126.67 0 0 1 119.54 0 8 8 0 0 1 3.28 10.85Zm-16 28a8 8 0 0 1-10.82 3.3 77.07 77.07 0 0 0-72.48 0 8 8 0 0 1-7.52-14.12 93 93 0 0 1 87.52 0 8 8 0 0 1 3.29 10.84Z"
  })]
});
export default SpotifyLogo;
