// phosphor-react-native/lib/module/duotone/TextColumns.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const TextColumns = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "text-columns-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-text-columns-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M216 64v120H40V64Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/_jsx(Path, {
    d: "M120 64a8 8 0 0 1-8 8H40a8 8 0 0 1 0-16h72a8 8 0 0 1 8 8Zm-8 32H40a8 8 0 0 0 0 16h72a8 8 0 0 0 0-16Zm0 40H40a8 8 0 0 0 0 16h72a8 8 0 0 0 0-16Zm0 40H40a8 8 0 0 0 0 16h72a8 8 0 0 0 0-16Zm32-104h72a8 8 0 0 0 0-16h-72a8 8 0 0 0 0 16Zm72 24h-72a8 8 0 0 0 0 16h72a8 8 0 0 0 0-16Zm0 40h-72a8 8 0 0 0 0 16h72a8 8 0 0 0 0-16Zm0 40h-72a8 8 0 0 0 0 16h72a8 8 0 0 0 0-16Z"
  })]
});
export default TextColumns;
