// phosphor-react-native/lib/module/duotone/UserList.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const UserList = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "user-list-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-user-list-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M120 104a40 40 0 1 1-40-40 40 40 0 0 1 40 40Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/_jsx(Path, {
    d: "M152 80a8 8 0 0 1 8-8h88a8 8 0 0 1 0 16h-88a8 8 0 0 1-8-8Zm96 40h-88a8 8 0 0 0 0 16h88a8 8 0 0 0 0-16Zm0 48h-64a8 8 0 0 0 0 16h64a8 8 0 0 0 0-16Zm-96.25 22a8 8 0 0 1-5.76 9.74 7.55 7.55 0 0 1-2 .26 8 8 0 0 1-7.75-6c-6.16-23.94-30.34-42-56.25-42s-50.09 18.05-56.25 42a8 8 0 0 1-15.5-4c5.59-21.71 21.84-39.29 42.46-48a48 48 0 1 1 58.58 0c20.63 8.71 36.88 26.29 42.47 48ZM80 136a32 32 0 1 0-32-32 32 32 0 0 0 32 32Z"
  })]
});
export default UserList;
