// phosphor-react-native/lib/module/duotone/Voicemail.js
"use strict";

/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
const Voicemail = ({
  title,
  titleId,
  ...props
}) => /*#__PURE__*/_jsxs(Svg, {
  className: "voicemail-duotone__svg-icon-phosphor",
  testID: props.testID ?? 'phosphor-react-native-voicemail-duotone',
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size,
  "aria-labelledby": titleId,
  ...props,
  children: [title ? /*#__PURE__*/_jsx("title", {
    id: titleId,
    children: title
  }) : null, /*#__PURE__*/_jsx(Path, {
    d: "M104 128a48 48 0 1 1-48-48 48 48 0 0 1 48 48Zm96-48a48 48 0 1 0 48 48 48 48 0 0 0-48-48Z",
    opacity: props.duotoneOpacity ?? 0.2,
    fill: props.duotoneColor ?? "currentColor"
  }), /*#__PURE__*/_jsx(Path, {
    d: "M200 72a56 56 0 0 0-39.14 96H95.14A56 56 0 1 0 56 184h144a56 56 0 0 0 0-112ZM16 128a40 40 0 1 1 40 40 40 40 0 0 1-40-40Zm184 40a40 40 0 1 1 40-40 40 40 0 0 1-40 40Z"
  })]
});
export default Voicemail;
